<!DOCTYPE html>
<html lang="en-US" class="no-js no-svg">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<link rel="profile" href="https://gmpg.org/xfn/11">

<script>(function(html){html.className = html.className.replace(/\bno-js\b/,'js')})(document.documentElement);</script>
<title>Parallel Sessions &#8211; Conference on Statistical Learning and Data Science / Nonparametric Statistics</title>
<meta name='robots' content='max-image-preview:large' />
	<style>img:is([sizes="auto" i], [sizes^="auto," i]) { contain-intrinsic-size: 3000px 1500px }</style>
	<link rel="alternate" type="application/rss+xml" title="Conference on Statistical Learning and Data Science / Nonparametric Statistics &raquo; Feed" href="feed/index.html" />
<link rel="alternate" type="application/rss+xml" title="Conference on Statistical Learning and Data Science / Nonparametric Statistics &raquo; Comments Feed" href="comments/feed/index.html" />
<script>
window._wpemojiSettings = {"baseUrl":"https:\/\/s.w.org\/images\/core\/emoji\/16.0.1\/72x72\/","ext":".png","svgUrl":"https:\/\/s.w.org\/images\/core\/emoji\/16.0.1\/svg\/","svgExt":".svg","source":{"concatemoji":"https:\/\/publish.illinois.edu\/sldsc2018\/wp-includes\/js\/wp-emoji-release.min.js?ver=6.8.2"}};
/*! This file is auto-generated */
!function(s,n){var o,i,e;function c(e){try{var t={supportTests:e,timestamp:(new Date).valueOf()};sessionStorage.setItem(o,JSON.stringify(t))}catch(e){}}function p(e,t,n){e.clearRect(0,0,e.canvas.width,e.canvas.height),e.fillText(t,0,0);var t=new Uint32Array(e.getImageData(0,0,e.canvas.width,e.canvas.height).data),a=(e.clearRect(0,0,e.canvas.width,e.canvas.height),e.fillText(n,0,0),new Uint32Array(e.getImageData(0,0,e.canvas.width,e.canvas.height).data));return t.every(function(e,t){return e===a[t]})}function u(e,t){e.clearRect(0,0,e.canvas.width,e.canvas.height),e.fillText(t,0,0);for(var n=e.getImageData(16,16,1,1),a=0;a<n.data.length;a++)if(0!==n.data[a])return!1;return!0}function f(e,t,n,a){switch(t){case"flag":return n(e,"\ud83c\udff3\ufe0f\u200d\u26a7\ufe0f","\ud83c\udff3\ufe0f\u200b\u26a7\ufe0f")?!1:!n(e,"\ud83c\udde8\ud83c\uddf6","\ud83c\udde8\u200b\ud83c\uddf6")&&!n(e,"\ud83c\udff4\udb40\udc67\udb40\udc62\udb40\udc65\udb40\udc6e\udb40\udc67\udb40\udc7f","\ud83c\udff4\u200b\udb40\udc67\u200b\udb40\udc62\u200b\udb40\udc65\u200b\udb40\udc6e\u200b\udb40\udc67\u200b\udb40\udc7f");case"emoji":return!a(e,"\ud83e\udedf")}return!1}function g(e,t,n,a){var r="undefined"!=typeof WorkerGlobalScope&&self instanceof WorkerGlobalScope?new OffscreenCanvas(300,150):s.createElement("canvas"),o=r.getContext("2d",{willReadFrequently:!0}),i=(o.textBaseline="top",o.font="600 32px Arial",{});return e.forEach(function(e){i[e]=t(o,e,n,a)}),i}function t(e){var t=s.createElement("script");t.src=e,t.defer=!0,s.head.appendChild(t)}"undefined"!=typeof Promise&&(o="wpEmojiSettingsSupports",i=["flag","emoji"],n.supports={everything:!0,everythingExceptFlag:!0},e=new Promise(function(e){s.addEventListener("DOMContentLoaded",e,{once:!0})}),new Promise(function(t){var n=function(){try{var e=JSON.parse(sessionStorage.getItem(o));if("object"==typeof e&&"number"==typeof e.timestamp&&(new Date).valueOf()<e.timestamp+604800&&"object"==typeof e.supportTests)return e.supportTests}catch(e){}return null}();if(!n){if("undefined"!=typeof Worker&&"undefined"!=typeof OffscreenCanvas&&"undefined"!=typeof URL&&URL.createObjectURL&&"undefined"!=typeof Blob)try{var e="postMessage("+g.toString()+"("+[JSON.stringify(i),f.toString(),p.toString(),u.toString()].join(",")+"));",a=new Blob([e],{type:"text/javascript"}),r=new Worker(URL.createObjectURL(a),{name:"wpTestEmojiSupports"});return void(r.onmessage=function(e){c(n=e.data),r.terminate(),t(n)})}catch(e){}c(n=g(i,f,p,u))}t(n)}).then(function(e){for(var t in e)n.supports[t]=e[t],n.supports.everything=n.supports.everything&&n.supports[t],"flag"!==t&&(n.supports.everythingExceptFlag=n.supports.everythingExceptFlag&&n.supports[t]);n.supports.everythingExceptFlag=n.supports.everythingExceptFlag&&!n.supports.flag,n.DOMReady=!1,n.readyCallback=function(){n.DOMReady=!0}}).then(function(){return e}).then(function(){var e;n.supports.everything||(n.readyCallback(),(e=n.source||{}).concatemoji?t(e.concatemoji):e.wpemoji&&e.twemoji&&(t(e.twemoji),t(e.wpemoji)))}))}((window,document),window._wpemojiSettings);
</script>
<link rel='stylesheet' id='pt-cv-public-style-css' href='../wp-content/plugins/content-views-query-and-display-post-page/public/assets/css/cv.css%3Fver=4.2.css' media='all' />
<style id='wp-emoji-styles-inline-css'>

	img.wp-smiley, img.emoji {
		display: inline !important;
		border: none !important;
		box-shadow: none !important;
		height: 1em !important;
		width: 1em !important;
		margin: 0 0.07em !important;
		vertical-align: -0.1em !important;
		background: none !important;
		padding: 0 !important;
	}
</style>
<link rel='stylesheet' id='wp-block-library-css' href='wp-includes/css/dist/block-library/style.min.css%3Fver=6.8.2.css' media='all' />
<style id='wp-block-library-theme-inline-css'>
.wp-block-audio :where(figcaption){color:#555;font-size:13px;text-align:center}.is-dark-theme .wp-block-audio :where(figcaption){color:#ffffffa6}.wp-block-audio{margin:0 0 1em}.wp-block-code{border:1px solid #ccc;border-radius:4px;font-family:Menlo,Consolas,monaco,monospace;padding:.8em 1em}.wp-block-embed :where(figcaption){color:#555;font-size:13px;text-align:center}.is-dark-theme .wp-block-embed :where(figcaption){color:#ffffffa6}.wp-block-embed{margin:0 0 1em}.blocks-gallery-caption{color:#555;font-size:13px;text-align:center}.is-dark-theme .blocks-gallery-caption{color:#ffffffa6}:root :where(.wp-block-image figcaption){color:#555;font-size:13px;text-align:center}.is-dark-theme :root :where(.wp-block-image figcaption){color:#ffffffa6}.wp-block-image{margin:0 0 1em}.wp-block-pullquote{border-bottom:4px solid;border-top:4px solid;color:currentColor;margin-bottom:1.75em}.wp-block-pullquote cite,.wp-block-pullquote footer,.wp-block-pullquote__citation{color:currentColor;font-size:.8125em;font-style:normal;text-transform:uppercase}.wp-block-quote{border-left:.25em solid;margin:0 0 1.75em;padding-left:1em}.wp-block-quote cite,.wp-block-quote footer{color:currentColor;font-size:.8125em;font-style:normal;position:relative}.wp-block-quote:where(.has-text-align-right){border-left:none;border-right:.25em solid;padding-left:0;padding-right:1em}.wp-block-quote:where(.has-text-align-center){border:none;padding-left:0}.wp-block-quote.is-large,.wp-block-quote.is-style-large,.wp-block-quote:where(.is-style-plain){border:none}.wp-block-search .wp-block-search__label{font-weight:700}.wp-block-search__button{border:1px solid #ccc;padding:.375em .625em}:where(.wp-block-group.has-background){padding:1.25em 2.375em}.wp-block-separator.has-css-opacity{opacity:.4}.wp-block-separator{border:none;border-bottom:2px solid;margin-left:auto;margin-right:auto}.wp-block-separator.has-alpha-channel-opacity{opacity:1}.wp-block-separator:not(.is-style-wide):not(.is-style-dots){width:100px}.wp-block-separator.has-background:not(.is-style-dots){border-bottom:none;height:1px}.wp-block-separator.has-background:not(.is-style-wide):not(.is-style-dots){height:2px}.wp-block-table{margin:0 0 1em}.wp-block-table td,.wp-block-table th{word-break:normal}.wp-block-table :where(figcaption){color:#555;font-size:13px;text-align:center}.is-dark-theme .wp-block-table :where(figcaption){color:#ffffffa6}.wp-block-video :where(figcaption){color:#555;font-size:13px;text-align:center}.is-dark-theme .wp-block-video :where(figcaption){color:#ffffffa6}.wp-block-video{margin:0 0 1em}:root :where(.wp-block-template-part.has-background){margin-bottom:0;margin-top:0;padding:1.25em 2.375em}
</style>
<style id='classic-theme-styles-inline-css'>
/*! This file is auto-generated */
.wp-block-button__link{color:#fff;background-color:#32373c;border-radius:9999px;box-shadow:none;text-decoration:none;padding:calc(.667em + 2px) calc(1.333em + 2px);font-size:1.125em}.wp-block-file__button{background:#32373c;color:#fff;text-decoration:none}
</style>
<style id='global-styles-inline-css'>
:root{--wp--preset--aspect-ratio--square: 1;--wp--preset--aspect-ratio--4-3: 4/3;--wp--preset--aspect-ratio--3-4: 3/4;--wp--preset--aspect-ratio--3-2: 3/2;--wp--preset--aspect-ratio--2-3: 2/3;--wp--preset--aspect-ratio--16-9: 16/9;--wp--preset--aspect-ratio--9-16: 9/16;--wp--preset--color--black: #000000;--wp--preset--color--cyan-bluish-gray: #abb8c3;--wp--preset--color--white: #ffffff;--wp--preset--color--pale-pink: #f78da7;--wp--preset--color--vivid-red: #cf2e2e;--wp--preset--color--luminous-vivid-orange: #ff6900;--wp--preset--color--luminous-vivid-amber: #fcb900;--wp--preset--color--light-green-cyan: #7bdcb5;--wp--preset--color--vivid-green-cyan: #00d084;--wp--preset--color--pale-cyan-blue: #8ed1fc;--wp--preset--color--vivid-cyan-blue: #0693e3;--wp--preset--color--vivid-purple: #9b51e0;--wp--preset--gradient--vivid-cyan-blue-to-vivid-purple: linear-gradient(135deg,rgba(6,147,227,1) 0%,rgb(155,81,224) 100%);--wp--preset--gradient--light-green-cyan-to-vivid-green-cyan: linear-gradient(135deg,rgb(122,220,180) 0%,rgb(0,208,130) 100%);--wp--preset--gradient--luminous-vivid-amber-to-luminous-vivid-orange: linear-gradient(135deg,rgba(252,185,0,1) 0%,rgba(255,105,0,1) 100%);--wp--preset--gradient--luminous-vivid-orange-to-vivid-red: linear-gradient(135deg,rgba(255,105,0,1) 0%,rgb(207,46,46) 100%);--wp--preset--gradient--very-light-gray-to-cyan-bluish-gray: linear-gradient(135deg,rgb(238,238,238) 0%,rgb(169,184,195) 100%);--wp--preset--gradient--cool-to-warm-spectrum: linear-gradient(135deg,rgb(74,234,220) 0%,rgb(151,120,209) 20%,rgb(207,42,186) 40%,rgb(238,44,130) 60%,rgb(251,105,98) 80%,rgb(254,248,76) 100%);--wp--preset--gradient--blush-light-purple: linear-gradient(135deg,rgb(255,206,236) 0%,rgb(152,150,240) 100%);--wp--preset--gradient--blush-bordeaux: linear-gradient(135deg,rgb(254,205,165) 0%,rgb(254,45,45) 50%,rgb(107,0,62) 100%);--wp--preset--gradient--luminous-dusk: linear-gradient(135deg,rgb(255,203,112) 0%,rgb(199,81,192) 50%,rgb(65,88,208) 100%);--wp--preset--gradient--pale-ocean: linear-gradient(135deg,rgb(255,245,203) 0%,rgb(182,227,212) 50%,rgb(51,167,181) 100%);--wp--preset--gradient--electric-grass: linear-gradient(135deg,rgb(202,248,128) 0%,rgb(113,206,126) 100%);--wp--preset--gradient--midnight: linear-gradient(135deg,rgb(2,3,129) 0%,rgb(40,116,252) 100%);--wp--preset--font-size--small: 13px;--wp--preset--font-size--medium: 20px;--wp--preset--font-size--large: 36px;--wp--preset--font-size--x-large: 42px;--wp--preset--spacing--20: 0.44rem;--wp--preset--spacing--30: 0.67rem;--wp--preset--spacing--40: 1rem;--wp--preset--spacing--50: 1.5rem;--wp--preset--spacing--60: 2.25rem;--wp--preset--spacing--70: 3.38rem;--wp--preset--spacing--80: 5.06rem;--wp--preset--shadow--natural: 6px 6px 9px rgba(0, 0, 0, 0.2);--wp--preset--shadow--deep: 12px 12px 50px rgba(0, 0, 0, 0.4);--wp--preset--shadow--sharp: 6px 6px 0px rgba(0, 0, 0, 0.2);--wp--preset--shadow--outlined: 6px 6px 0px -3px rgba(255, 255, 255, 1), 6px 6px rgba(0, 0, 0, 1);--wp--preset--shadow--crisp: 6px 6px 0px rgba(0, 0, 0, 1);}:where(.is-layout-flex){gap: 0.5em;}:where(.is-layout-grid){gap: 0.5em;}body .is-layout-flex{display: flex;}.is-layout-flex{flex-wrap: wrap;align-items: center;}.is-layout-flex > :is(*, div){margin: 0;}body .is-layout-grid{display: grid;}.is-layout-grid > :is(*, div){margin: 0;}:where(.wp-block-columns.is-layout-flex){gap: 2em;}:where(.wp-block-columns.is-layout-grid){gap: 2em;}:where(.wp-block-post-template.is-layout-flex){gap: 1.25em;}:where(.wp-block-post-template.is-layout-grid){gap: 1.25em;}.has-black-color{color: var(--wp--preset--color--black) !important;}.has-cyan-bluish-gray-color{color: var(--wp--preset--color--cyan-bluish-gray) !important;}.has-white-color{color: var(--wp--preset--color--white) !important;}.has-pale-pink-color{color: var(--wp--preset--color--pale-pink) !important;}.has-vivid-red-color{color: var(--wp--preset--color--vivid-red) !important;}.has-luminous-vivid-orange-color{color: var(--wp--preset--color--luminous-vivid-orange) !important;}.has-luminous-vivid-amber-color{color: var(--wp--preset--color--luminous-vivid-amber) !important;}.has-light-green-cyan-color{color: var(--wp--preset--color--light-green-cyan) !important;}.has-vivid-green-cyan-color{color: var(--wp--preset--color--vivid-green-cyan) !important;}.has-pale-cyan-blue-color{color: var(--wp--preset--color--pale-cyan-blue) !important;}.has-vivid-cyan-blue-color{color: var(--wp--preset--color--vivid-cyan-blue) !important;}.has-vivid-purple-color{color: var(--wp--preset--color--vivid-purple) !important;}.has-black-background-color{background-color: var(--wp--preset--color--black) !important;}.has-cyan-bluish-gray-background-color{background-color: var(--wp--preset--color--cyan-bluish-gray) !important;}.has-white-background-color{background-color: var(--wp--preset--color--white) !important;}.has-pale-pink-background-color{background-color: var(--wp--preset--color--pale-pink) !important;}.has-vivid-red-background-color{background-color: var(--wp--preset--color--vivid-red) !important;}.has-luminous-vivid-orange-background-color{background-color: var(--wp--preset--color--luminous-vivid-orange) !important;}.has-luminous-vivid-amber-background-color{background-color: var(--wp--preset--color--luminous-vivid-amber) !important;}.has-light-green-cyan-background-color{background-color: var(--wp--preset--color--light-green-cyan) !important;}.has-vivid-green-cyan-background-color{background-color: var(--wp--preset--color--vivid-green-cyan) !important;}.has-pale-cyan-blue-background-color{background-color: var(--wp--preset--color--pale-cyan-blue) !important;}.has-vivid-cyan-blue-background-color{background-color: var(--wp--preset--color--vivid-cyan-blue) !important;}.has-vivid-purple-background-color{background-color: var(--wp--preset--color--vivid-purple) !important;}.has-black-border-color{border-color: var(--wp--preset--color--black) !important;}.has-cyan-bluish-gray-border-color{border-color: var(--wp--preset--color--cyan-bluish-gray) !important;}.has-white-border-color{border-color: var(--wp--preset--color--white) !important;}.has-pale-pink-border-color{border-color: var(--wp--preset--color--pale-pink) !important;}.has-vivid-red-border-color{border-color: var(--wp--preset--color--vivid-red) !important;}.has-luminous-vivid-orange-border-color{border-color: var(--wp--preset--color--luminous-vivid-orange) !important;}.has-luminous-vivid-amber-border-color{border-color: var(--wp--preset--color--luminous-vivid-amber) !important;}.has-light-green-cyan-border-color{border-color: var(--wp--preset--color--light-green-cyan) !important;}.has-vivid-green-cyan-border-color{border-color: var(--wp--preset--color--vivid-green-cyan) !important;}.has-pale-cyan-blue-border-color{border-color: var(--wp--preset--color--pale-cyan-blue) !important;}.has-vivid-cyan-blue-border-color{border-color: var(--wp--preset--color--vivid-cyan-blue) !important;}.has-vivid-purple-border-color{border-color: var(--wp--preset--color--vivid-purple) !important;}.has-vivid-cyan-blue-to-vivid-purple-gradient-background{background: var(--wp--preset--gradient--vivid-cyan-blue-to-vivid-purple) !important;}.has-light-green-cyan-to-vivid-green-cyan-gradient-background{background: var(--wp--preset--gradient--light-green-cyan-to-vivid-green-cyan) !important;}.has-luminous-vivid-amber-to-luminous-vivid-orange-gradient-background{background: var(--wp--preset--gradient--luminous-vivid-amber-to-luminous-vivid-orange) !important;}.has-luminous-vivid-orange-to-vivid-red-gradient-background{background: var(--wp--preset--gradient--luminous-vivid-orange-to-vivid-red) !important;}.has-very-light-gray-to-cyan-bluish-gray-gradient-background{background: var(--wp--preset--gradient--very-light-gray-to-cyan-bluish-gray) !important;}.has-cool-to-warm-spectrum-gradient-background{background: var(--wp--preset--gradient--cool-to-warm-spectrum) !important;}.has-blush-light-purple-gradient-background{background: var(--wp--preset--gradient--blush-light-purple) !important;}.has-blush-bordeaux-gradient-background{background: var(--wp--preset--gradient--blush-bordeaux) !important;}.has-luminous-dusk-gradient-background{background: var(--wp--preset--gradient--luminous-dusk) !important;}.has-pale-ocean-gradient-background{background: var(--wp--preset--gradient--pale-ocean) !important;}.has-electric-grass-gradient-background{background: var(--wp--preset--gradient--electric-grass) !important;}.has-midnight-gradient-background{background: var(--wp--preset--gradient--midnight) !important;}.has-small-font-size{font-size: var(--wp--preset--font-size--small) !important;}.has-medium-font-size{font-size: var(--wp--preset--font-size--medium) !important;}.has-large-font-size{font-size: var(--wp--preset--font-size--large) !important;}.has-x-large-font-size{font-size: var(--wp--preset--font-size--x-large) !important;}
:where(.wp-block-post-template.is-layout-flex){gap: 1.25em;}:where(.wp-block-post-template.is-layout-grid){gap: 1.25em;}
:where(.wp-block-columns.is-layout-flex){gap: 2em;}:where(.wp-block-columns.is-layout-grid){gap: 2em;}
:root :where(.wp-block-pullquote){font-size: 1.5em;line-height: 1.6;}
</style>
<link rel='stylesheet' id='fancybox-for-wp-css' href='../wp-content/plugins/fancybox-for-wordpress/assets/css/fancybox.css%3Fver=1.3.4.css' media='all' />
<link rel='stylesheet' id='twentyseventeen-fonts-css' href='../wp-content/themes/twentyseventeen/assets/fonts/font-libre-franklin.css%3Fver=20230328.css' media='all' />
<link rel='stylesheet' id='twentyseventeen-style-css' href='../wp-content/themes/twentyseventeen/style.css%3Fver=20250415.css' media='all' />
<link rel='stylesheet' id='twentyseventeen-block-style-css' href='../wp-content/themes/twentyseventeen/assets/css/blocks.css%3Fver=20240729.css' media='all' />
<link rel='stylesheet' id='tablepress-default-css' href='files/tablepress-combined.min.css%3Fver=15.css' media='all' />
<link rel='stylesheet' id='simcal-qtip-css' href='../wp-content/plugins/google-calendar-events/assets/generated/vendor/jquery.qtip.min.css%3Fver=3.5.5.css' media='all' />
<link rel='stylesheet' id='simcal-default-calendar-grid-css' href='../wp-content/plugins/google-calendar-events/assets/generated/default-calendar-grid.min.css%3Fver=3.5.5.css' media='all' />
<link rel='stylesheet' id='simcal-default-calendar-list-css' href='../wp-content/plugins/google-calendar-events/assets/generated/default-calendar-list.min.css%3Fver=3.5.5.css' media='all' />
<link rel='stylesheet' id='colorbox-css' href='../wp-content/plugins/lightbox-gallery/colorbox/example1/colorbox.css%3Fver=6.8.2.css' media='all' />
<script src="wp-includes/js/jquery/jquery.min.js%3Fver=3.7.1" id="jquery-core-js"></script>
<script src="wp-includes/js/jquery/jquery-migrate.min.js%3Fver=3.4.1" id="jquery-migrate-js"></script>
<script src="../wp-content/plugins/fancybox-for-wordpress/assets/js/purify.min.js%3Fver=1.3.4" id="purify-js"></script>
<script src="../wp-content/plugins/fancybox-for-wordpress/assets/js/jquery.fancybox.js%3Fver=1.3.4" id="fancybox-for-wp-js"></script>
<script id="twentyseventeen-global-js-extra">
var twentyseventeenScreenReaderText = {"quote":"<svg class=\"icon icon-quote-right\" aria-hidden=\"true\" role=\"img\"> <use href=\"#icon-quote-right\" xlink:href=\"#icon-quote-right\"><\/use> <\/svg>","expand":"Expand child menu","collapse":"Collapse child menu","icon":"<svg class=\"icon icon-angle-down\" aria-hidden=\"true\" role=\"img\"> <use href=\"#icon-angle-down\" xlink:href=\"#icon-angle-down\"><\/use> <span class=\"svg-fallback icon-angle-down\"><\/span><\/svg>"};
</script>
<script src="../wp-content/themes/twentyseventeen/assets/js/global.js%3Fver=20211130" id="twentyseventeen-global-js" defer data-wp-strategy="defer"></script>
<script src="../wp-content/themes/twentyseventeen/assets/js/navigation.js%3Fver=20210122" id="twentyseventeen-navigation-js" defer data-wp-strategy="defer"></script>
<link rel="EditURI" type="application/rsd+xml" title="RSD" href="https://publish.illinois.edu/sldsc2018/xmlrpc.php?rsd" />
<meta name="generator" content="WordPress 6.8.2" />
<link rel="canonical" href="index.html%3Fp=608.html" />
<link rel='shortlink' href='index.html%3Fp=608.html' />
<link rel="alternate" title="oEmbed (JSON)" type="application/json+oembed" href="https://publish.illinois.edu/sldsc2018/wp-json/oembed/1.0/embed?url=https%3A%2F%2Fpublish.illinois.edu%2Fsldsc2018%2Fparallel-sessions%2F" />
<link rel="alternate" title="oEmbed (XML)" type="text/xml+oembed" href="https://publish.illinois.edu/sldsc2018/wp-json/oembed/1.0/embed?url=https%3A%2F%2Fpublish.illinois.edu%2Fsldsc2018%2Fparallel-sessions%2F&#038;format=xml" />

<!-- Fancybox for WordPress v3.3.4 -->
<style type="text/css">
	.fancybox-slide--image .fancybox-content{background-color: #FFFFFF}div.fancybox-caption{display:none !important;}
	
	img.fancybox-image{border-width:10px;border-color:#FFFFFF;border-style:solid;}
	div.fancybox-bg{background-color:rgba(102,102,102,0.3);opacity:1 !important;}div.fancybox-content{border-color:#FFFFFF}
	div#fancybox-title{background-color:#FFFFFF}
	div.fancybox-content{background-color:#FFFFFF}
	div#fancybox-title-inside{color:#333333}
	
	
	
	div.fancybox-caption p.caption-title{display:inline-block}
	div.fancybox-caption p.caption-title{font-size:14px}
	div.fancybox-caption p.caption-title{color:#333333}
	div.fancybox-caption {color:#333333}div.fancybox-caption p.caption-title {background:#fff; width:auto;padding:10px 30px;}div.fancybox-content p.caption-title{color:#333333;margin: 0;padding: 5px 0;}body.fancybox-active .fancybox-container .fancybox-stage .fancybox-content .fancybox-close-small{display:block;}
</style><script type="text/javascript">
	jQuery(function () {

		var mobileOnly = false;
		
		if (mobileOnly) {
			return;
		}

		jQuery.fn.getTitle = function () { // Copy the title of every IMG tag and add it to its parent A so that fancybox can show titles
			var arr = jQuery("a[data-fancybox]");jQuery.each(arr, function() {var title = jQuery(this).children("img").attr("title") || '';var figCaptionHtml = jQuery(this).next("figcaption").html() || '';var processedCaption = figCaptionHtml;if (figCaptionHtml.length && typeof DOMPurify === 'function') {processedCaption = DOMPurify.sanitize(figCaptionHtml, {USE_PROFILES: {html: true}});} else if (figCaptionHtml.length) {processedCaption = jQuery("<div>").text(figCaptionHtml).html();}var newTitle = title;if (processedCaption.length) {newTitle = title.length ? title + " " + processedCaption : processedCaption;}if (newTitle.length) {jQuery(this).attr("title", newTitle);}});		}

		// Supported file extensions

				var thumbnails = jQuery("a:has(img)").not(".nolightbox").not('.envira-gallery-link').not('.ngg-simplelightbox').filter(function () {
			return /\.(jpe?g|png|gif|mp4|webp|bmp|pdf)(\?[^/]*)*$/i.test(jQuery(this).attr('href'))
		});
		

		// Add data-type iframe for links that are not images or videos.
		var iframeLinks = jQuery('.fancyboxforwp').filter(function () {
			return !/\.(jpe?g|png|gif|mp4|webp|bmp|pdf)(\?[^/]*)*$/i.test(jQuery(this).attr('href'))
		}).filter(function () {
			return !/vimeo|youtube/i.test(jQuery(this).attr('href'))
		});
		iframeLinks.attr({"data-type": "iframe"}).getTitle();

				// Gallery All
		thumbnails.addClass("fancyboxforwp").attr("data-fancybox", "gallery").getTitle();
		iframeLinks.attr({"data-fancybox": "gallery"}).getTitle();

		// Gallery type NONE
		
		// Call fancybox and apply it on any link with a rel atribute that starts with "fancybox", with the options set on the admin panel
		jQuery("a.fancyboxforwp").fancyboxforwp({
			loop: false,
			smallBtn: true,
			zoomOpacity: "auto",
			animationEffect: "fade",
			animationDuration: 500,
			transitionEffect: "fade",
			transitionDuration: "300",
			overlayShow: true,
			overlayOpacity: "0.3",
			titleShow: true,
			titlePosition: "inside",
			keyboard: true,
			showCloseButton: true,
			arrows: true,
			clickContent:false,
			clickSlide: "close",
			mobile: {
				clickContent: function (current, event) {
					return current.type === "image" ? "toggleControls" : false;
				},
				clickSlide: function (current, event) {
					return current.type === "image" ? "close" : "close";
				},
			},
			wheel: false,
			toolbar: true,
			preventCaptionOverlap: true,
			onInit: function() { },			onDeactivate
	: function() { },		beforeClose: function() { },			afterShow: function(instance) { jQuery( ".fancybox-image" ).on("click", function( ){ ( instance.isScaledDown() ) ? instance.scaleToActual() : instance.scaleToFit() }) },				afterClose: function() { },					caption : function( instance, item ) {var title = "";if("undefined" != typeof jQuery(this).context ){var title = jQuery(this).context.title;} else { var title = ("undefined" != typeof jQuery(this).attr("title")) ? jQuery(this).attr("title") : false;}var caption = jQuery(this).data('caption') || '';if ( item.type === 'image' && title.length ) {caption = (caption.length ? caption + '<br />' : '') + '<p class="caption-title">'+jQuery("<div>").text(title).html()+'</p>' ;}if (typeof DOMPurify === "function" && caption.length) { return DOMPurify.sanitize(caption, {USE_PROFILES: {html: true}}); } else { return jQuery("<div>").text(caption).html(); }},
		afterLoad : function( instance, current ) {var captionContent = current.opts.caption || '';var sanitizedCaptionString = '';if (typeof DOMPurify === 'function' && captionContent.length) {sanitizedCaptionString = DOMPurify.sanitize(captionContent, {USE_PROFILES: {html: true}});} else if (captionContent.length) { sanitizedCaptionString = jQuery("<div>").text(captionContent).html();}if (sanitizedCaptionString.length) { current.$content.append(jQuery('<div class=\"fancybox-custom-caption inside-caption\" style=\" position: absolute;left:0;right:0;color:#000;margin:0 auto;bottom:0;text-align:center;background-color:#FFFFFF \"></div>').html(sanitizedCaptionString)); }},
			})
		;

			})
</script>
<!-- END Fancybox for WordPress -->
		<style id="twentyseventeen-custom-header-styles" type="text/css">
				.site-title a,
		.colors-dark .site-title a,
		.colors-custom .site-title a,
		body.has-header-image .site-title a,
		body.has-header-video .site-title a,
		body.has-header-image.colors-dark .site-title a,
		body.has-header-video.colors-dark .site-title a,
		body.has-header-image.colors-custom .site-title a,
		body.has-header-video.colors-custom .site-title a,
		.site-description,
		.colors-dark .site-description,
		.colors-custom .site-description,
		body.has-header-image .site-description,
		body.has-header-video .site-description,
		body.has-header-image.colors-dark .site-description,
		body.has-header-video.colors-dark .site-description,
		body.has-header-image.colors-custom .site-description,
		body.has-header-video.colors-custom .site-description {
			color: #ffffff;
		}
		</style>
		<link rel="icon" href="files/2018/05/cropped-logo-32x32.jpeg" sizes="32x32" />
<link rel="icon" href="files/2018/05/cropped-logo-192x192.jpeg" sizes="192x192" />
<link rel="apple-touch-icon" href="files/2018/05/cropped-logo-180x180.jpeg" />
<meta name="msapplication-TileImage" content="https://publish.illinois.edu/sldsc2018/files/2018/05/cropped-logo-270x270.jpeg" />
<style id="sccss">/* Enter Your Custom CSS Here */
/*hide the category and tag info*/
.entry-footer {
    display: none;
}
.entry-meta .byline, .entry-meta .cat-links { display: none; }
.entry-meta .posted-on { display: none !important; }
.author-box {
  display: none;
}
.post-meta{
  display: none;
}
.below-entry-meta .byline {
  display: none;
}
.nocomments {
display: none !important;
}
.post-meta_box{
display:none !important;
}

/*Remove Image Borders */
.entry img, img.thumbnail {
     background: none;
     border: medium none;
     padding: 5px;
}

/*Remove borders of inserted pic 
#content .entry-content a img,
#content .entry-content img {
	border: 0;
}*/
/*Remove borders of inserted pic in a gallery */
.entry-content a img,
.entry-content img {
	border: 0 !important;
}

.gallery a img { border: 0 !important; }

.entry-content a,
.entry-content a:hover {
	-webkit-box-shadow: none;
	box-shadow: none;
}

/*show no underline for links*/
/*.entry-content a:link {
text-decoration: none;
border-bottom: none;
-webkit-box-shadow: none;
box-shadow: none;
}*/
.entry-content a {
    color: #2762a8;
    border: none;
    box-shadow: none;
}
.entry-content a:hover {
    color: #82b92a;
}
/*#logo .site-title a:hover {
    text-decoration: none;
}*/

/*a:link {
text-decoration: none;
border-bottom: none;
-webkit-box-shadow: none;
box-shadow: none;
}
.entry-content a:visited {
text-decoration: none;
border-bottom: none;
-webkit-box-shadow: none;
box-shadow: none;
}*/
/*add top margin of contents*/
/*.page.title { 
  margin-bottom: 60px !important; 
}*/</style></head>

<body class="wp-singular page-template-default page page-id-608 wp-embed-responsive wp-theme-twentyseventeen has-header-image page-one-column colors-light">
<div id="page" class="site">
	<a class="skip-link screen-reader-text" href="index.html%3Fp=608.html#content">
		Skip to content	</a>

	<header id="masthead" class="site-header">

		<div class="custom-header">

		<div class="custom-header-media">
			<div id="wp-custom-header" class="wp-custom-header"><img src="files/2018/05/cropped-jonathan-riley-118591-unsplash.jpg" width="2000" height="1200" alt="" srcset="files/2018/05/cropped-jonathan-riley-118591-unsplash.jpg 2000w, files/2018/05/cropped-jonathan-riley-118591-unsplash-300x180.jpg 300w, files/2018/05/cropped-jonathan-riley-118591-unsplash-768x461.jpg 768w, files/2018/05/cropped-jonathan-riley-118591-unsplash-1024x614.jpg 1024w" sizes="(max-width: 767px) 200vw, 100vw" decoding="async" fetchpriority="high" /></div>		</div>

	<div class="site-branding">
	<div class="wrap">

		
		<div class="site-branding-text">
										<p class="site-title"><a href="index.html" rel="home" >Conference on Statistical Learning and Data Science / Nonparametric Statistics</a></p>
			
							<p class="site-description">June 4th-6th, 2018, Columbia University, NY</p>
					</div><!-- .site-branding-text -->

		
	</div><!-- .wrap -->
</div><!-- .site-branding -->

</div><!-- .custom-header -->

					<div class="navigation-top">
				<div class="wrap">
					<nav id="site-navigation" class="main-navigation" aria-label="Top Menu">
	<button class="menu-toggle" aria-controls="top-menu" aria-expanded="false">
		<svg class="icon icon-bars" aria-hidden="true" role="img"> <use href="#icon-bars" xlink:href="#icon-bars"></use> </svg><svg class="icon icon-close" aria-hidden="true" role="img"> <use href="#icon-close" xlink:href="#icon-close"></use> </svg>Menu	</button>

	<div class="menu-home-container"><ul id="top-menu" class="menu"><li id="menu-item-23" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-home menu-item-23"><a href="index.html">Home</a></li>
<li id="menu-item-680" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-ancestor current-menu-parent current_page_parent current_page_ancestor menu-item-has-children menu-item-680"><a href="index.html%3Fp=653.html">Sessions<svg class="icon icon-angle-down" aria-hidden="true" role="img"> <use href="#icon-angle-down" xlink:href="#icon-angle-down"></use> </svg></a>
<ul class="sub-menu">
	<li id="menu-item-681" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-681"><a href="index.html%3Fp=653.html">Keynote Speech</a></li>
	<li id="menu-item-621" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-608 current_page_item menu-item-621"><a href="index.html%3Fp=608.html" aria-current="page">Parallel Sessions</a></li>
	<li id="menu-item-404" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-404"><a href="index.html%3Fp=400.html">Poster session</a></li>
</ul>
</li>
<li id="menu-item-152" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-152"><a href="index.html%3Fp=148.html">Reception<svg class="icon icon-angle-down" aria-hidden="true" role="img"> <use href="#icon-angle-down" xlink:href="#icon-angle-down"></use> </svg></a>
<ul class="sub-menu">
	<li id="menu-item-689" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-689"><a href="index.html%3Fp=148.html">Social Events</a></li>
</ul>
</li>
<li id="menu-item-58" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-58"><a href="index.html%3Fp=36.html">Schedule</a></li>
<li id="menu-item-59" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-59"><a href="index.html%3Fp=39.html">Key Dates</a></li>
<li id="menu-item-60" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-60"><a href="index.html%3Fp=43.html">Registration</a></li>
<li id="menu-item-61" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-61"><a href="index.html%3Fp=48.html">Local Information</a></li>
<li id="menu-item-62" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-62"><a href="index.html%3Fp=52.html">Conference Venue</a></li>
<li id="menu-item-63" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-63"><a href="index.html%3Fp=54.html">Committees</a></li>
<li id="menu-item-64" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-64"><a href="index.html%3Fp=56.html">Sponsors</a></li>
<li id="menu-item-800" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-800"><a href="index.html%3Fp=796.html">Gallery</a></li>
</ul></div>
	</nav><!-- #site-navigation -->
				</div><!-- .wrap -->
			</div><!-- .navigation-top -->
		
	</header><!-- #masthead -->

	
	<div class="site-content-contain">
		<div id="content" class="site-content">

<div class="wrap">
	<div id="primary" class="content-area">
		<main id="main" class="site-main">

			
<article id="post-608" class="post-608 page type-page status-publish hentry">
	<header class="entry-header">
		<h1 class="entry-title">Parallel Sessions</h1>			</header><!-- .entry-header -->
	<div class="entry-content">
		<p>The full list of abstracts can be downloaded <strong><a href="files/2018/06/sessions.pdf" target="_blank" rel="noopener">here</a></strong>.</p>
<p><strong>Click <a href="index.html%3Fp=640.html">here</a> to search</strong>.</p>
<div class="pt-cv-wrapper"><div class="pt-cv-view pt-cv-collapsible" id="pt-cv-view-89dee33ry6"><div data-id="pt-cv-page-1" class="pt-cv-page" data-cvc="1"><div class="panel-group" id="9db5e6bwfs"><div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#c97811ati3" href='index.html%3Fp=450.html' onclick='event.preventDefault()'>
		Session 1: High-dimensional Tensor Data Analysis	</a>
	</div>
<div id="c97811ati3" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: High-dimensional Tensor Data Analysis<br />
<strong>Organizer</strong>: Xin &#8220;Henry&#8221; Zhang (Florida State U)<br />
<strong>Chair:</strong> Xin &#8220;Henry&#8221; Zhang (Florida State U)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 9:00-10:30am<br />
<strong>Location:</strong> VEC 404/405</p>
<p><strong>Speech 1: Multilayer Tensor Factorization with Applications to Recommender Systems<br />
Speaker</strong>: Xuan Bi (Yale)<br />
<strong>abstract: </strong>Recommender systems have been widely adopted by electronic commerce and entertainment industries for individualized prediction and recommendation, which benefit consumers and improve business intelligence. In this talk, we propose an innovative method, namely the recommendation engine of multilayers (REM), for tensor recommender systems. The proposed method utilizes the structure of a tensor response to integrate information from multiple modes, and creates an additional layer of nested latent factors to accommodate between-subjects dependency. One major advantage is that the proposed method is able to address the “cold-start” issue in the absence of information from new customers, new products or new contexts. Specifically, it provides more effective recommendations through sub-group information. To achieve scalable computation, we develop a new algorithm for the proposed method, which incorporates a maximum block improvement strategy into the cyclic blockwise-coordinate-descent algorithm. In theory, we investigate algorithmic properties for convergence from an arbitrary initial point and local convergence, along with the asymptotic consistency of estimated pa- rameters. Finally, the proposed method is applied in simulations and IRI marketing data with 116 million observations of product sales. Numerical studies demonstrate that the proposed method outperforms existing competitors in the literature.</p>
<p><strong>Speech 2: Dynamic Tensor Clustering<br />
Speaker:</strong> Will Wei Sun (University of Miami)<br />
<strong>abstract: </strong>Dynamic tensor data are becoming prevalent in numerous applications. Existing tensor clustering methods either fail to account for the dynamic nature of the data, or are inapplicable to a general-order tensor. Also there is often a gap between statistical guarantee and computational efficiency for existing tensor clustering solutions. In this talk, I will introduce a new dynamic tensor clustering method, which takes into account both sparsity and fusion structures, and enjoys strong statistical guarantees as well as high computational efficiency. The efficacy of our approach will be illustrated via two real applications: brain dynamic functional connectivity analysis, and advertisement clustering for market segmentation. This is a joint work with Lexin Li.</p>
<p><strong>Speech 3: Covariate-adjusted tensor classification in high-dimensions<br />
Speaker:</strong> Qing Mai (Florida State U)<br />
<strong>abstract: </strong>In contemporary scientific research, it is of great interest to predict a categorical response based on a high-dimensional tensor and additional covariates. We introduce the CATCH model (in short for Covariate-Adjusted Tensor Classification in High-dimensions), that efficiently integrates the covariates and the tensor to predict the categorical outcome and jointly explains the relationships among the covariates, the tensor predictor, and the categorical response. To tackle the new computational and statistical challenges arising from the intimidating tensor dimensions, we propose a group penalized approach and an efficient algorithm. Theoretical results confirm that our method achieves variable selection consistency and optimal prediction, even when the tensor dimension is much larger than the sample size. The superior performance of our method over existing methods is demonstrated in extensive simulation studies, a colorimetric sensor array data, and two neuroimaging studies.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#c155c8cl2t" href='index.html%3Fp=452.html' onclick='event.preventDefault()'>
		Session 2: High-dimensional inference: assumption-lean or assumption-laden?	</a>
	</div>
<div id="c155c8cl2t" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: High-dimensional inference: assumption-lean or assumption-laden?<br />
<strong>Organizer</strong>: Ryan Tibshirani (CMU)<br />
<strong>Chair:</strong> Jelena Bradic (UCSD)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 9:00-10:30am<br />
<strong>Location:</strong> VEC 902/903<br />
<strong><br />
Speech 1: Inferential goals, targets, and principles in high-dimensional regression</strong><br />
<strong>Speaker: Todd Kuffner (Washington U)</strong><br />
<strong>Abstract: </strong>This talk will focus more on theory than methodology. We will analyze the inferential goals, the targets of inference, and the various principles employed to justify prominent methodologies. A new perspective, motivated by philosophy of science, will be presented on how to discern between competing inferential procedures for high-dimensional regression.<strong><br />
</strong><br />
<strong>Speech 2: Should We Model X in High-Dimensional Inference?</strong><br />
<strong>Speaker: Lucas Janson (Harvard)</strong><br />
<strong>Abstract:  </strong>For answering questions about the relationship between a response variable Y and a set of explanatory variables X, most statistical methods focus their assumptions on the conditional distribution of Y given X (or Y | X for short). I will describe some benefits of shifting those assumptions from the conditional distribution Y | X to the joint distribution of X, especially for high-dimensional data. First, modeling X can lead to assumptions that are more realistic and verifiable. Second, there are substantial methodological payoffs in terms of much greater flexibility in the tools an analyst can bring to bear on their data while also being guaranteed exact (non-asymptotic) inference. I will briefly mention some of my recent and ongoing work on methods for high-dimensional inference that model X instead of Y, as well as some challenges and interesting directions for the future.</p>
<p><strong>Speech 3: Towards a Better Understanding of  &#8220;High-Dimensional&#8221; Linear Least Squares Regression</strong><br />
<strong>Speaker: Andreas Buja (U Penn) </strong><br />
<strong>Abstract: </strong>High-dimensional regression is conventionally interpreted as an optimization problem of, most commonly, a penalized or constrained LS criterion. Instead, we propose to reinterpret high-dimensional regression as follows: Data is first explored either in a principled way (e.g., lasso or best subset selection) or in an exploratory/unprincipled way to select a manageable set of variables; subsequently the reduced data are subjected to linear regression. The final set of variables is often much smaller than the sample size and<br />
the total number of initial variables. We will treat the combination of both steps as forming high-dimensional linear regression. A first question we consider is to ask what the nature of the OLS estimator is if regressors have been subselected by some variable selection procedure. We answer this question in full generality by proving a deterministic uniform-in-model result about linear regression, and this provides an interpretation of what is being estimated irrespective of the data-dependent variable selection procedure.A second question we consider is how to perform statistical inference using the OLS estimator obtained from a variable selection procedure. This problem is exactly the problem of valid Post-Selection Inference (PoSI). This talk will focus on an approach to PoSI based on an asymptotic linear representation and a high-dimensional central limit theorem.All our results are proved without assuming any probability models, and they allow for non-identically distributed random vectors. In addition, they apply equally to independent and functionally dependent data. Finally, our results do not require any sparsity assumptions. Joint work with the Wharton Linear Models Group including Lawrence<br />
Brown, Edward George and Linda Zhao. Some of this talk is based on https://arxiv.org/abs/1802.05801.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#1bf0b95vf2" href='index.html%3Fp=454.html' onclick='event.preventDefault()'>
		Session 3: Modern nonparametric statistics	</a>
	</div>
<div id="1bf0b95vf2" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Modern nonparametric statistics<br />
<strong>Organizer</strong>: Richard Samworth (Cambridge)<br />
<strong>Chair: </strong>Zhengling Qi (UNC)<strong><br />
Time:</strong> June 4<sup>th</sup>, 9:00-10:30am<br />
<strong>Location: </strong>VEC 1202/1203</p>
<p><strong>Speech 1: Regimes of label-noise determine the benefits of Active Learning</strong><br />
<strong>Speaker:  Samory Kpotufe (Princeton)</strong><br />
<strong>Abstract: </strong>In active learning (for classification tasks), the learner has the ability to request the labels of carefully chosen points over space.  Intuitively, this might speedup the learning process — in terms of the number of labels required to achieve a fixed error— over the usual passive setting where the learner accesses i.i.d labeled data. Unfortunately, despite significant progress on the subject, the benefits of active over passive learning remain largely unclear: for example, in the usual PAC setting with VC classes, label requirements in active learning are of the same order as in passive learning outside of strong assumptions on label noise. However, a clearer picture of the benefits of active learning emerges under refined parameterization of label noise &#8211; this is considered e.g. in work by Hanneke and by Koltchinskii, however under the strong assumption of bounded ‘disagreement-coefficient’.In this talk, we aim to gain a better picture of the benefits of active learning over passive learning. In particular, we will consider parametrizations of label noise that help capture a continuum from easy to hard classification problems, and elicit a clearer picture of the benefits of active learning along this continuum. Such parametrizations draw on intuition from the so-called `cluster assumption’ in ML, and more generally on so-called ‘margin conditions’ common in both ML and Statistics. Our results reveal interesting phase transitions (in label requirements) driven by the interaction between noise parameters, marginal distribution, and data dimension. In particular, we manage to address a previous conjecture about the existence of some such transitions. Furthermore, our algorithmic strategies are adaptive, i.e., require no a priori knowledge of distributional parameters, yet are rate-optimal.The talk is based on recent collaborations with S. Ben-David, R. Urner, A. Locatelli, and A. Carpentier.</p>
<p><strong>Speech 2: Sampling design and stochastic gradient descent for relational data</strong><br />
<strong>Speaker: Peter Orbanz (Columbia)</strong><br />
<strong>Abstract: </strong>State-of-the art learning procedures for relational data typically involve several steps that randomly subsample a data set: (1) During data acquisition from the underlying population. (2) During data splitting <span class="il">or</span> cross validation. (3) During training, if learning involves stochastic gradient descent. There are many natural ways to subsample relational data, and in practice, it is not the exception but the rule that different sampling schemes are used in the different steps. That raises a number of problems: If the sampling schemes do not cohere in a suitable sense, the meaning of prediction becomes ambiguous, error estimates are biased, etc. I will discuss what conditions are required to avoid such problems, and describe a new method for learning from relational data that incorporates the sampling scheme as an explicit model design choice.</p>
<p>Joint work with Victor Veitch, Wenda Zhou, Morgane Austern and David Blei.</p>
<p><strong>Speech 3: Statistical Properties of Maximum Mean Discrepancy with Gaussian Kernels</strong><br />
<strong>Speaker: Tong Li(Columbia)</strong><br />
<strong>Abstract: </strong>Despite the popularity of reproducing kernel based techniques for nonparametric hypothesis testing, the choice of kernel in these approaches is usually ad hoc and how to do so in a more principled way remains one of the most critical challenges in practice. To overcome this difficulty, we provide here justifications for one of the most common and successful choices, Gaussian kernels with a flexible shape parameter. More specifically, we study the statistical properties of maximum mean discrepancy (MMD) based testing procedures with Gaussian kernels. We show that they arise naturally when maximizing MMD over a general class of radial basis function kernels. Moreover, we show that when the underlying distributions are sufficiently smooth, MMD with Gaussian kernels gives rise to a test adaptive over different levels of smoothness in that it attains the minimax optimal detection rates, up to a logarithmic factor, for any given smoothness index.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#d861e2314d" href='index.html%3Fp=459.html' onclick='event.preventDefault()'>
		Session 4: New methods for directed acyclic Gaussian graph and adaptive data analysis	</a>
	</div>
<div id="d861e2314d" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: <span style="color: black;font-family: arial, helvetica, sans-serif">New methods for directed acyclic Gaussian graph and </span><span style="font-family: arial, helvetica, sans-serif">adaptive data analysis</span><br />
<strong>Organizer</strong>: Yichao Wu (UIC)<br />
<strong>Chair:</strong> Weibin Mo (UNC)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 9:00-10:30am<br />
<strong>Location: </strong>VEC 1302/1303</p>
<p><strong>Speech 1: Sublinear-Time Adaptive Data Analysis</strong><br />
<strong>Speaker: Lev Reyzin (UIC)<br />
</strong><strong>Abstract: </strong>The topic of this talk lies in the burgeoning area of adaptive data analysis, where the goal is to design mechanisms that can give statistically valid answers to adaptively generated queries.  In this talk, I will discuss sublinear-time mechanisms for answering adaptive queries into datasets. These mechanisms provide a polynomial speed-up per query over previous approaches, without increasing the total amount of data needed. I will also discuss a new method for achieving statistically-meaningful responses even when the mechanism is only allowed to see a constant number of samples from the data per query.  Finally, I will show how these techniques also yield improved bounds for adaptively optimizing convex and strongly convex functions over a dataset.  This is based on work joint with Benjamin Fish and Benjamin I. P. Rubinstein.</p>
<p><strong>Speech 2:  Reconstruction of a directed acyclic Gaussian graph  for observational and interventional data<br />
</strong><strong>Speaker: Xiaotong Shen (U Minnesota)</strong><br />
<strong>Abstract: </strong>Directed acyclic graphs are widely used to describe, among interacting units, causal relations. Causal relations are estimated by reconstructing a directed acyclic graph&#8217;s structure, presenting a great challenge when the unknown total ordering of a DAG needs to be estimated.In such a situation, it remains unclear if a graph&#8217;s structure is reconstructable in the absence of an identifiable likelihood with regard to graphs, and in facing super-exponentially many candidate graphs in the number of nodes. In this talk, I will introduce a global approach to process observational data and interventional data, to identify all estimable causal directions and estimate model parameters. This approach uses constrained maximum likelihood with nonconvex constraints reinforcing the non-loop equirement to yield an estimated directed acyclic graph, where super-exponentially many constraints characterize the major challenge. Computational issues will be discussed in addition to some theoretical aspects. This work is joint with Y. Yuan, W. Pan, Z. Wang and S. Peng.</p>
<p><span style="color: black;font-family: Menlo;font-size: xx-small">.</span><strong>Speech 3: Convex clustering over an undirected graph</strong><br />
<strong>Speaker: Yunzhang Zhu (OSU)</strong><br />
<strong>Abstract: </strong>Cluster analysis is a fundamental problem in statistics. It aims at categorizing the observations into different groups, called clusters, such that observations in the same cluster tend to be more similar to each other than those from different clusters. In this talk, I will introduce a new optimization-based clustering method called convex clustering over (weighted) undirected graph. The choice of both the graph and its weights is crucial to clustering performance as well as the algorithm&#8217;s computational efficiency. Specifically, we consider two types of graphs: a minimum spanning tree and a so-called K-means bipartite graph. Computationally, both graphs make the associated optimization problems easier to solve compared to that with a complete graph; and statistically, both lead to better clustering results. Further numerical comparisons with the K-means clustering algorithm and a density-based algorithm demonstrate the superior performance of the proposed algorithms.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#caf6934pki" href='index.html%3Fp=461.html' onclick='event.preventDefault()'>
		Session 5: Statistical Inference in Clustering Problems	</a>
	</div>
<div id="caf6934pki" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Statistical Inference in Clustering Problems<br />
<strong>Organizer</strong>: Jacob Bien (Cornell)<br />
<strong>Chair:</strong> Jacob Bien (Cornell)<strong><br />
Time:</strong> June 4<sup>th</sup>, 9:00-10:30am<br />
<strong>Location: </strong>VEC 1402<br />
<strong><br />
Speech 1: Inference for variable clustering under correlation-like similarities</strong><br />
<strong>Speaker:  Max G&#8217;Sell (CMU)</strong><br />
<strong>Abstract: </strong>Clustering is often applied to detect dependence structure among the variables in large data sets.  However, it is typically difficult to determine the appropriate amount of clustering to carry out in a given application.  We will take a selective inference approach to testing of hierarchical clustering of variables based on measures of their correlation.  We will see that this yields reasonable goodness-of-fit stopping rules for selecting the number of clusters.  We will consider weakening the required assumptions and generalizing the measure of correlation, and the computational issues that arise in this pursuit.<strong><br />
</strong><strong><br />
Speech 2: Large scale cluster analysis via L1 fusion penalization</strong><br />
<strong>Speaker: Gourab Mukherjee (USC)</strong><br />
<strong>Abstract: </strong>We study the large sample behavior of a convex clustering framework, which minimizes the sample within cluster sum of squares under an L_1 fusion constraint on the cluster centroids. We establish that the sample procedure consistently estimates its population analog. We derive the corresponding rates of convergence and develop a novel methodology for feature screening in the clustering of massive datasets. We demonstrate empirically the applicability of our method to cluster analysis of big datasets arising in single-cell gene expression studies.</p>
<div></div>
<p><strong>Speech 3: Density Tree and Density Ranking in Singular Measures<br />
</strong> <strong>Speaker: Yen-Chi Chen (UW)</strong><br />
<strong>Abstract: </strong>A density tree (also known as a cluster tree of a probability density function) is a tool in topological data analysis that uses a tree structure to represent the shape of a density function. Even if the density function is multivariate, a density tree can always be displayed on a two-dimensional plane, making it an ideal tool for visualizing the shape of a multivariate dataset. However, in complex datasets such as GPS data, the underlying distribution function is singular so the usual density function and density tree no longer exist. To analyze this type of data and generalize the density tree, we introduce the concept of density ranking and ranking tree (also called an $\alpha$-tree). We then show that one can consistently estimate the density ranking and the ranking tree using a kernel density estimator. Based on the density ranking, we introduce several geometric and topological summary curves for analyzing GPS datasets.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#8ed8a33l5t" href='index.html%3Fp=464.html' onclick='event.preventDefault()'>
		Session 6: Statistical methods of integrating -omics data	</a>
	</div>
<div id="8ed8a33l5t" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Statistical methods of integrating -omics data<br />
<strong>Organizer</strong>: Wei, Ying (Columbia U)<br />
<strong>Chair:</strong> X. Song (Mount Sinai)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 9:00-10:30am<br />
<strong>Location:</strong> VEC 1403</p>
<p><strong>Speech 1: A Statistical Framework for Leveraging Information across Multiple Traits in Genetic Studies</strong><br />
<strong>Speaker:  Gen Li (Columbia)<br />
</strong><strong>Abstract: </strong>In genetic studies, pleiotropy occurs when a genetic variant affects multiple traits simultaneously. The true effect sizes for different traits usually have significant correlations. Most existing genome-wide association studies only focus on one trait at a time and fail to leverage the relationships between different traits. Motivated by the multi-tissue expression quantitative trait loci (eQTL) analysis in the Genotype-Tissue Expression (GTEx) project, we develop a two-stage method to address this limitation. It effectively borrows strength across multiple traits to identify genetic variants regulating a target trait. The method is based on summary statistics and allows genotype data to partially overlap between traits. We apply the proposed method to the GTEx data and identify more eQTLs with potential functionality.<strong><br />
</strong><strong><br />
</strong><strong>Speech 2: <b>A new method to study the change of miRNA–mRNA interactions due to environmental exposures</b></strong><br />
<strong>Speaker: Pei Wang (Icahn School of Medicine at Mount Sinai)</strong><br />
<strong>Abstract: </strong>ntegrative approaches characterizing the interactions among different types of biological molecules have been demonstrated to be useful for revealing informative biological mechanisms. One such example is the interaction between microRNA (miRNA) and messenger RNA (mRNA), whose deregulation may be sensitive to environmental insult leading to altered phenotypes. In this work, <u></u><u></u>we introduce a new network approach—integrative Joint Random Forest (iJRF), which characterizes the regulatory system between miRNAs and mRNAs using a network model. iJRF is designed to work under the high-dimension low-sample-size regime, and can borrow information across different treatment conditions to achieve more accurate network inference. It also effectively takes into account prior information of miRNA–mRNA regulatory relationships from existing databases. We then apply iJRF to data from an animal experiment designed to investigate the effect of low-dose environmental chemical exposure on normal mammary gland development. We detected a few important miRNAs that regulated a large number of mRNAs in the control group but not in the exposed groups, suggesting the disruption of miRNA activity due to chemical exposure. Effects of chemical exposure on two affected miRNAs were further validated using breast cancer human cell lines.</p>
<p>&nbsp;</p>
<p><strong>Speech 3: smFARM: sparse multivariate Factor Analysis Regression Model in integrative genomics analysis</strong><br />
<strong>Speaker: Peter Song(Umich)</strong><br />
<strong>Abstract:<br />
</strong>The multivariate regression model is a useful tool to explore complex associations between multiple response variables (e.g. gene expressions) and multiple predictors (e.g. SNPs). When the multiple responses are correlated, ignoring such dependency will impair statistical power in the data analysis. Motivated by an integrative genomic data, we propose a new methodology – sparse multivariate factor analysis regression model (smFARM), in which the covariance of the response variables is modeled by a factor analysis model with latent factors. This proposed method not only allows us to address the challenge that the number of genetic predictors is larger than the sample size, but also to adjust for unobserved genetic and/or non-genetic factors that potentially conceal the underlying real response-predictor associations. The proposed smFARM is implemented efficiently by utilizing the strength of the EM algorithm and the group-wise coordinate descend algorithm. In addition, the identified latent factors are explained by the means of gene enrichment analysis. The proposed methodology is evaluated and compared to the existing methods through extensive simulation studies. We apply smFARM in an integrative genomics analysis of a breast cancer dataset on the relationship between DNA copy numbers and gene expression arrays to derive genetic regulatory patterns relevant to breast cancer.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#31f57ebij1" href='index.html%3Fp=467.html' onclick='event.preventDefault()'>
		Session 7: Recent Advances in Statistical Learning	</a>
	</div>
<div id="31f57ebij1" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Recent Advances in Statistical Learning<br />
<strong>Organizer</strong>: Ming Yuan (Columbia)<br />
<strong>Chair:</strong> Dong Xia (UCSD)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 11:00am-12:30pm<br />
<strong>Location: </strong>VEC 404</p>
<p><strong>Speech 1: Geometry of Optimization Landscapes and Implicit Regularization of Optimization Algorithms.</strong><br />
<strong>Speaker: Jason Lee (USC)</strong><br />
<strong>Abstract:</strong></p>
<p dir="ltr">We first study the problem of learning a Gaussian input two-layer ReLU network with positive output layer and  and the symmetric matrix completion problem. Despite the non-convexity of both problems, we prove that every local minimizer is a global minimizer. Since gradient descent converges to local minimizers, this shows that simple gradient-based methods can find the global optimum of these non-convex problems.</p>
<p dir="ltr">In the second part, we analyze the implicit regularization effects of various optimization algorithms. In particular we prove that for least squares with mirror descent, the algorithm converges to the closest solution in terms of the bregman divergence. For linearly separable classification problems, we prove that the steepest descent with respect to a norm solves SVM with respect to the same norm. For over-parametrized non-convex problems such as matrix sensing or neural net with quadratic activation, we prove that gradient descent converges to the minimum nuclear norm solution, which allows for both meaningful optimization and generalization guarantees.</p>
<p dir="ltr">This is joint work with Rong Ge, Suriya Gunasekar, Tengyu Ma, Mor Shpigel, Daniel Soudry, and Nati Srebro.</p>
<p><strong>Speech 2: M-estimation with the Trimmed L1 penalty<br />
Speaker: Aurelie Lozano (IBM)<br />
Abstract:</strong></p>
<p>We study high-dimensional M-estimators with the trimmed L1 penalty. While standard L1 penalty incurs bias (shrinkage),  trimmed L1 leaves the h largest entries penalty-free. This family of estimators include the Trimmed Lasso for sparse linear regression and its counterpart for sparse graphical model estimation. The trimmed L1 penalty is non-convex, but unlike other non-convex regularizers such as SCAD and MCP, it is not amenable and therefore prior analyzes cannot be applied.</p>
<p>We characterize the support recovery of the estimates as a function of the trimming parameter h. Under certain conditions, we show that for any local optimum, (i) if the trimming parameter h is smaller than the true support size, all zero entries of the true parameter vector are successfully estimated as zero, and (ii) if h is larger than the true support size,  the non-relevant parameters of the local optimum have smaller absolute values than relevant parameters and hence relevant parameters are not penalized. We then bound the L2 error of any local optimum. These bounds are asymptotically comparable to those for non-convex amenable penalties such as SCAD or MCP, but enjoy better constants. We specialize our main results to linear regression and graphical model estimation.</p>
<p>Finally, we develop a fast provably convergent optimization algorithm for the trimmed regularizer problem.  The algorithm has the same rate of convergence as difference of convex (DC)-based approaches, but is faster in practice and finds better objective values than recently proposed algorithms for DC optimization. Empirical results further demonstrate the value of L1 trimming.</p>
<p><strong>Speech 3:  Methods of network comparison</strong><br />
<strong>Speaker: Sofia Olhede (UCL)</strong><br />
<strong>Abstract: </strong>The topology of any complex system is key to understanding its structure and function. Fundamentally, algebraic topology guarantees that any system represented by a network can be understood through its closed paths. The length of each path provides a notion of scale, which is vitally important in characterizing dominant modes of system behavior. Here, by combining topology with scale, we prove the existence of universal features which reveal the dominant scales of any network. We use these features to compare several canonical network types in the context of a social media discussion which evolves through the sharing of rumors, leaks and other news. Our analysis enables for the first time a universal understanding of the balance between loops and tree-like structure across network scales, and an assessment of how this balance interacts with the spreading of information online. Crucially, our results allow networks to be quantified and compared in a purely model-free way that is theoretically sound, fully automated, and inherently scalable. This work is joint with Patrick Wolfe.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#5b848d6dxi" href='index.html%3Fp=469.html' onclick='event.preventDefault()'>
		Session 8: Supervised and unsupervised learning of complex data	</a>
	</div>
<div id="5b848d6dxi" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: <strong>Supervised and unsupervised learning of complex data</strong><br />
<strong>Organizer</strong>: Junhui Wang (Citi U of HK)<br />
<strong>Chair:</strong> Junhui Wang (Citi U of HK)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 11:00am-12:30pm<br />
<strong>Location: </strong>VEC 405</p>
<p><strong>Speech 1: <b>Systems of partially linear models with gradient boosting</b><br />
</strong> <strong>Speaker: Yongzhao Shao (NYU)</strong><br />
<strong>Abstract: </strong>We develop systems partially linear models with gradient boosting for prediction in multicenter studies or regression-based clustering in large scale data. Simultaneous variable selection and effect estimation are achieved using LASSO type penalty functions and ADMM. Simulation studies and real data examples are used to illustrate effectiveness of the proposed methods.<br />
<strong><br />
</strong><strong>Speech 2: Supervised Dimensionality Reduction for Exponential Family Data</strong><br />
<strong>Speaker: </strong>Yoonkyung Lee (OSU)<br />
<strong>Abstract: </strong><span class="m_2907465332869453706SubmittedValue">Supervised dimensionality reduction techniques, such as partial least squares and supervised principal components, are powerful tools for making predictions with a large number of variables. The implicit squared error terms in the objectives, however, make it less attractive to non-Gaussian data, either in the covariates or the responses. Drawing on a connection between partial least squares and the Gaussian distribution, we show how partial least squares can be extended to other members of the exponential family &#8211; similar to the generalized linear model &#8211; for both the covariates and the responses. Unlike previous attempts, our extension gives latent variables which are easily interpretable as linear functions of the data and is computationally efficient. In particular, it does not require additional optimization for the scores of new observations and therefore predictions can be made in real time. </span>This is joint work with Andrew Landgraf at Battelle Memorial Institute.</p>
<div><strong>Speech 3:  Transform-based unsupervised point registration and unseeded low-rank graph matching</strong><br />
<strong>Speaker: Yuan Zhang (OSU)</strong><br />
<strong>Abstract: </strong>
<div>Unsupervised estimation of the correspondence between two point sets has long been an attractive topic to CS and EE researchers.  In this paper, we focus on the vanilla form of the problem: matching two point sets that are identical over a linear transformation.  The problem is well-studied and many classical algorithms exist, yet, many of them suffer one or several of the following shortcomings:  slow computation on large data sets, limited applicable distribution families and lack of theoretical analysis.  Arguably, the bottleneck of computation lies in the need of many methods to evaluate n^2 many pairwise similarity or distance measures in each iteration.  Also, few results exist in bounding the error of some specific point matching algorithm, where dependence might be a main obstacle.</div>
<div></div>
<div>In this paper, we propose a novel method using Laplace transformation to directly match the underlying distributions of the two point sets.  Our method is fast because it avoids the n^2 many pairwise evaluations in iterations.  On the theory side, we propose a new error bound on the Wasserstein distance between two distributions in terms of the integrated difference between their Laplace transforms.  Based on this, we can establish consistency of our method.   Our method is also distinct by its versatility in handling a wide range of distribution families, while most existing methods typically require the data generating distributions to be continuous.</div>
<div></div>
<div>We then show how our method applies to the problem of match up nodes in two low-rank networks such that the aligned networks &#8220;look similar&#8221;.  Numerical comparisons illustrate our method&#8217;s significant advantages in both speed and accuracy over existing methods.</div>
</div>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#7278fdbstm" href='index.html%3Fp=471.html' onclick='event.preventDefault()'>
		Session 9: Advances in estimation and prediction for understanding complex disorders	</a>
	</div>
<div id="7278fdbstm" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: <strong> Advances in estimation and prediction for understanding complex disorders</strong><br />
<strong>Organizer</strong>: Heping Zhang (Yale)<br />
<strong>Chair:</strong> Naveen Narisetty (UIUC)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 11:00am-12:30pm<br />
<strong>Location: </strong>VEC 902</p>
<p><strong>Speech 1: Uncertainty Quantification of Treatment Regime in Precision Medicine by Confidence Distributions</strong><br />
<strong>Speaker: Min-ge Xie (Rutgers)</strong><br />
<strong>Abstract: </strong>Personalized decision rule in precision medicine can be viewed as a “discrete parameter”, for which theoretical development for statistical inference is lacking. In this talk, we propose a new way to quantify the estimation uncertainty in a personalized decision based on recent developments of confidence distribution (CD). Specifically, in a parametric regression model setup, suppose the decision for treatment versus control for an individual x<sub>a</sub> is determined by a linear decision rule D<sub>a</sub> = I(x<sub>a</sub><span style="font-family: Symbol;font-size: small">b</span>&gt; x<sub>a</sub><span style="font-family: Symbol;font-size: small">g</span>), where <span style="font-family: Symbol;font-size: small">b</span>  and  <span style="font-family: Symbol;font-size: small">g</span> are unknown regression coefficients in models for potential outcomes of treatment and control, respectively. The data-driven decision hat-D<sub>a</sub> relies on the estimates of b and g, which in turn introduces uncertainty on the decision. In this work, we propose to find a CD for  <span style="font-family: Symbol;font-size: small">h</span><sub>a </sub>=x<sub>a</sub><span style="font-family: Symbol;font-size: small">b</span> &#8211; x<sub>a</sub><span style="font-family: Symbol;font-size: small">g</span> and compute a “confidence measure” of the decision {Da = 1} = {ha  &gt; 0}. This measure has a value between 0 and 1, and provides a frequency-based assessment on how reliable our decision is. For example, if the confidence measure of the decision {Da = 1} is 63%, then we know that, out of 100 patients who are the same as patient xa, 63 will benefit to have the treatment and 38 will be better off to be in the control group. Numerical study suggests that this new measurement is inline with classical assessments (such as sensitivity, specificity, etc.), but different from the classical assessments, this measurement can be directly computed from the observed data without the need to know the truth of {Da = 1} or {Da = 0}. Utility of this new measure will also be demonstrated in an application of an adaptive-design clinical trial. (Joint work with Yilei Zhan and Sijian Wang)Personalized decision rule in precision medicine can be viewed as a “discrete parameter”, for which theoretical development for statistical inference is lacking. In this talk, we propose a new way to quantify the estimation uncertainty in a personalized decision based on recent developments of confidence distribution (CD). Specifically, in a parametric regression model setup, suppose the decision for treatment versus control for an individual x<sub>a</sub>is determined by a linear decision rule <span style="font-size: small"> D</span><sub>a</sub><span style="font-size: small"> = I(x</span><sub>a</sub><span style="font-family: Symbol;font-size: small">b</span><span style="font-size: small">&gt; </span><span style="font-size: small">x</span><sub>a</sub><span style="font-family: Symbol;font-size: small">g</span><span style="font-size: small">), where </span><span style="font-family: Symbol;font-size: small">b</span><span style="font-size: small"> and </span><span style="font-family: Symbol;font-size: small">g</span><span style="font-size: small"> are unknown regression coefficients in models for potential outcomes of treatment and control, respectively. The data-driven decision hat-D</span><sub>a</sub><span style="font-size: small"> </span>relies on the estimates of  andg, which in turn introduces uncertainty on the decision. In this work, we propose to find a CD for <span style="font-family: Symbol;font-size: small">h<sub>a </sub>=x</span><sub>a</sub><span style="font-family: Symbol;font-size: small">b &#8211; x</span><sub>a</sub><span style="font-family: Symbol;font-size: small">g</span> and compute a “confidence measure” of the decision D<sub>a</sub> = 1} = {<span style="font-family: Symbol;font-size: small">h</span><sub>a</sub><sub>  </sub>&gt; 0}. This measure has a value between 0 and 1, and provides a frequency-based assessment on how reliable our decision is. For example, if the confidence measure of the decision D<sub>a</sub> = 1} is 63%, then we know that, out of 100 patients who are the same as patient x<sub>a</sub>, 63 will benefit to have the treatment and 38 will be better off to be in the control group. Numerical study suggests that this new measurement is inline with classical assessments (such as sensitivity, specificity, etc.), but different from the classical assessments, this measurement can be directly computed from the observed data without the need to know the truth of D<sub>a</sub> = 1} or D<sub>a</sub> = 0}. Utility of this new measure will also be demonstrated in an application of an adaptive-design clinical trial. (Joint work with Yilei Zhan and Sijian Wang)</p>
<p><strong><br />
</strong><strong>Speech 2: Semiparametric Estimation in the Secondary Analysis of Case-Control Studies</strong><br />
<strong>Speaker: Yanyuan Ma (Penn State)</strong><br />
<strong>Abstract: </strong></p>
<div>We study the regression relationship among covariates in case-control data, an area known as the secondary analysis of case-control studies. The context is such that only the form of the regression mean is specified, so that we allow an arbitrary regression error distribution, which can depend on the covariates and thus can be heteroscedastic. Under mild regularity conditions we establish the theoretical identifiability of such models. Previous work in this context has either (a) specified a fully parametric distribution for the regression errors, (b) specified a homoscedastic distribution for the regression errors, (c) has specified the rate of disease in the population (we refer this as true population), or (d) has made a rare disease approximation. We construct a class of semiparametric estimation procedures that rely on none of these. The estimators differ from the usual semiparametric ones in that they draw conclusions about the true population, while technically operating in a hypothetic superpopulation. We also construct estimators with a unique feature, in that they are robust against the misspecification of the regression error distribution in terms of variance structure, while all other nonparametric effects are estimated despite of the biased samples. We establish the asymptotic properties of the estimators and illustrate their finite sample performance through simulation studies, as well as through an empirical example on the relation between red meat consumption and heterocyclic amines. Our analysis verified the positive relationship between red meat consumption and two forms of HCA, indicating that increased red meat consumption leads to increased levels of MeIQA and PhiP, both being risk factors for colorectal cancer.</div>
<p>&nbsp;</p>
<p><strong>Speech 3: Quantile Decision Trees and Forest with its application for predicting the risk (Post-Traumatic Stress Disorder) PTSD after experienced an acute coronary syndrome</strong><br />
<strong>Speaker: Ying Wei (Columbia)</strong><br />
<strong>Abstract:  </strong>Classification and regression trees (CART) are a classic statistical learning method that efficiently partitions the sample space into mutually exclusive subspaces with the distinctive means of an outcome of interest. It is a powerful tool for efficient subgroup analysis and allows for complex associations and interactions to achieve high prediction accuracy and stability. Hence, they are appealing tools for precision health applications that deal with large amounts of data from EMRs, genomics, and mobile data and aim to provide a transparent decision mechanism. Although there is a vast literature on decision trees and random forests, most algorithms identify subspaces with distinctive outcome means. The most vulnerable or high-risk groups for certain diseases are often patients with extremely high (or low) biomarker and phenotype values. However, means-based partitioning may not be effective for identifying patients with extreme phenotype values. We propose a new regression tree framework based on quantile regression \cite{KoenkerBassett1978} that partitions the sample space and predicts the outcome of interest based on conditional quantiles of the outcome variable. We implemented and evaluated the performance of the conditional quantile trees/forests to predict the risk of developing PTSD after experiencing an acute coronary syndrome (ACS), using an observational cohort data from the REactions to Acute Care and Hospitalization (REACH) study\cite{onge2017depressive} at New York Presbyterian Hospital. The results show that the conditional quantile based trees/forest have better discrimination power to identify patients with severe PTSD symptoms, in comparison to the classical mean based CART.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#3911ccc7xs" href='index.html%3Fp=515.html' onclick='event.preventDefault()'>
		Session 12: Modern Multivariate Statistics: Tensors and Networks	</a>
	</div>
<div id="3911ccc7xs" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Modern Multivariate Statistics: Tensors and Networks<br />
<strong>Organizer</strong>: Jacob Bien (Cornell)<br />
<strong>Chair:</strong> Jacob Bien (Cornell)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 1:45pm &#8211; 3:15pm<br />
<strong>Location: </strong>VEC 404<br />
<strong><br />
Speech 1: Computationally Efficient Tensor Completion with Statistical Optimality</strong><br />
<strong>Speaker: Dong Xia (columbia)</strong><br />
<strong>Abstract:  </strong>We develop methods for estimating a low rank tensor from noisy observations on a subset of its entries to achieve both statistical and computational efficiencies. There have been a lot of recent interests in this problem of noisy tensor completion. Much of the attention has been focused on the fundamental computational challenges often associated with problems involving higher order tensors, yet very little is known about their statistical performance. To fill in this void, in this article, we characterize the fundamental statistical limits of noisy tensor completion by establishing minimax optimal rates of convergence for estimating a k-th order low rank tensor which suggest significant room for improvement over the existing approaches. Furthermore, we propose a polynomial-time computable estimating procedure based upon power iteration and a second-order spectral initialization that achieves the optimal rates of convergence. Our method is fairly easy to implement and numerical experiments are presented to further demonstrate the practical merits of our estimator.<strong><br />
</strong><br />
<strong>Speech 2: Structured shrinkage of tensor parameters</strong><br />
<strong>Speaker: </strong>Peter Hoff (Duke)<br />
<strong>Abstract:<br />
</strong>Tensor-valued parameters arise in many multivariate statistical models, such as network autoregression where the relationship between a pair of nodes is potentially dependent on that of any other pair. Parameters in such models are likely to be near, but not necessarily in, low-dimensional subspaces of the parameter space. In this talk we discuss some adaptive empirical Bayes methods for shrinking parameter estimates towards an appropriately chosen subspace.</p>
<p><strong>Speech 3: Global Spectral Clustering for Dynamic Networks</strong><br />
<strong>Speaker: </strong>Patrick Perry (NYU)<br />
<strong>Abstract: </strong>In this talk, we present a new method (PisCES) for finding time-varying community structure in dynamic networks. The method implements spectral clustering, with a smoothing penalty to promote similarity across time periods. We prove that this method converges to the global solution of a nonconvex optimization problem, which can be interpreted as the spectral relaxation of a smoothed K-means clustering objective. We also show that smoothing is applied in a time-varying and data-dependent manner; for example, when a drastic change point exists in the data, smoothing is automatically suppressed at the time of the change point. Finally, we show that the detected time-varying communities can be effectively visualized through the use of sankey plots.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#1cda278q9u" href='index.html%3Fp=518.html' onclick='event.preventDefault()'>
		Session 10: Survival analysis with high-dimensional data	</a>
	</div>
<div id="1cda278q9u" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: <strong>Survival analysis with high-dimensional data</strong><br />
<strong>Organizer</strong>: Ingrid Van Keilegom (KU Leuven)<br />
<strong>Chair:</strong> Ricardo Cao (Universidade da Coruña)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 11:00am-12:30am<br />
<strong>Location:</strong> VEC 903<strong><br />
</strong><br />
<strong>Speech 1: Robust optimal treatment regime estimation with survival outcome<br />
Speaker: Lan Wang (U Minnesota)<br />
Abstract:  </strong>We consider estimating the single-stage quantile-optimal treatment regime from data with right-censored outcome. The proposed method is directly applicable to individualized treatment decision making in medicine when survival time or time to event is used as the primary end point, and the produced estimated rule is easy to interpret since the quantile-type statistics is widely used in such context. We proposed a nonparametric estimator belonging to value search category in literature, which directly estimates the optimal rule from a class of practically useful treatment regimes without posing constraints on the way treatment interact with covariates. We studied the nonstandard asymptotics for the estimated parameter of the optimal rule using semiparametric<br />
M-estimation theories, which reveals how censoring is influencing the uncertainty in the learned rule. (Joint work with Yu Zhou and Rui Song).</p>
<p><strong>Speech 2: Fine-Gray Competing Risks Model with High-Dimensional Covariates: Estimation and Inference </strong><br />
<strong>Speaker: Jelena Bradic (University of California, San Diego)<br />
Abstract: </strong>The purpose of this paper is to construct confidence intervals for the regression coefficients in the Fine-Gray model for competing risks data with random censoring, where the number of covariates can be larger than the sample size. Despite strong motivation from biostatistics applications, highdimensional Fine-Gray model has attracted relatively little attention among the methodological or theoretical literatures. We fill in this blank by proposing first a consistent regularized estimator and then the confidence intervals based on the one-step bias-correcting estimator. We are able to generalize the partial likelihood approach for the Fine-Gray model under random censoring despite many technical difficulties. We lay down a methodological and theoretical framework for the one-step bias-correcting estimator with the partial likelihood, which does not have independent and identically distributed entries. We also handle for our theory the approximation error from the inverse probability weighting (IPW), proposing novel concentration results for time dependent processes. In addition to the theoretical results and algorithms, we present extensive numerical experiments and an application to a study of non-cancer mortality among prostate cancer patients using the linked Medicare-SEER data.</p>
<p><strong>Speech 3: Envelopes for censored quantile regression<br />
Speaker: Yue Zhao (KU Leuven)<br />
Abstract: </strong>Quantile regression has emerged as a powerful tool for survival analysis with censored data.  We propose an efficient estimator for the coefficients in quantile regression with censored data using the envelope model.  The envelope model uses dimension reduction techniques to identify material and immaterial components in the data, and forms the estimator of the regression coefficient based only on the material component, thus reducing the variability of the estimation.  We will derive asymptotic properties of the proposed estimator and demonstrate its efficiency gains compared to the traditional estimator for the quantile regression with censored data.  The starting point of our technical analysis is the Z-estimation approach with local weighing (e.g., Wang &amp; Wang 2009) that in particular involves the conditional Kaplan-Meier estimator.  Traditionally, the Kaplan-Meier estimator is treated as an infinite dimensional nuisance parameter.  We will instead invoke the i.i.d. representation of the Kaplan-Meier estimator, which leads to a re-writing of our objective function as a U-process indexed by only the Euclidean parameter in the envelope model.  The modified Z-estimation problem then becomes much more amenable to analysis.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#ee46cd0kce" href='index.html%3Fp=531.html' onclick='event.preventDefault()'>
		Session 11: Recent advances of high-dimensional statistical learning.	</a>
	</div>
<div id="ee46cd0kce" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Recent advances of high-dimensional statistical learning.<br />
<strong>Organizer</strong>: Xiaotong Shen (U of Minnesota)<br />
<strong>Chair: </strong>Xiaotong Shen (U of Minnesota)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 11:00pm &#8211; 12:30pm<br />
<strong>Location: </strong>Hammer LL109 A/B</p>
<p><strong>Speech 1: Multiclass Probability Estimation with Support Vector Machines</strong><br />
<strong>Speaker: Helen Zhang (Arizona State University)</strong><br />
<strong>Abstract: </strong>Multiclass classification and probability estimation have important applications in data analytics. We propose a simple and scalable estimation framework for multiclass probabilities based on kernel SVMs. The new estimator does not rely on any parametric assumption on the data distribution, and hence it is flexible and robust. Theoretically, we can show that the proposed estimator is asymptotically consistent. Computationally, the new procedure can be conveniently implemented using standard SVM softwares. Our numerical studies demonstrate competitive performance of the new estimator when compared with existing methods such as multiple logistic regression, linear discrimination analysis (LDA), tree-based methods, and random forest (RF), under various settings.</p>
<p><strong>Speech 2: Minimizing Sum of Truncated Convex Functions and Its Applications</strong><br />
<strong>Speaker: Hui Jiang (UMich)</strong><br />
<strong>Abstract: </strong>We study a class of problems where the sum of truncated convex functions is minimized. In statistical applications, they are commonly encountered when L0-penalized models are fitted and usually lead to NP-Hard non-convex optimization problems. We propose a general algorithm for the global minimizer in low-dimensional settings. We also extend the algorithm to high-dimensional settings, where an approximate solution can be found efficiently. We introduce several applications where the sum of truncated convex functions is used, compare our proposed algorithm with other existing algorithms in simulation studies, and show its utility in edge-preserving image restoration on real data.</p>
<p><strong>Speech 3: Uncertainty and Inference for High-Dimensional Models Using the Solution Paths</strong><br />
<strong>Speaker: Peng Wang (University of Cincinnati)</strong><br />
<strong>Abstract: </strong>Bootstrap based model inference has been well studied. However, such approaches will almost certainly fail for high-dimensional models due to the fact that the selection results are highly sensitive to the choice of tuning parameter and therefore are extremely unstable. We proposed to utilize the information of the entire solution paths to overcome this obstacle. In particular, we select the best model based on the entire solution paths for each bootstrap sample, and make inference about both model and parameters using the results from all the bootstrap samples. Moreover, we also develop tools that visualize and quantify model selection uncertainty. These tools would allow practitioners evaluate the validity of the estimated models in high-dimensional settings.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#6f2c28bp12" href='index.html%3Fp=534.html' onclick='event.preventDefault()'>
		Session 13: Flexible Statistical Learning and Inference	</a>
	</div>
<div id="6f2c28bp12" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: <strong>Flexible Statistical Learning and Inference</strong><br />
<strong>Organizer</strong>: Yufeng Liu (UNC)<br />
<strong>Chair:</strong> Siliang Gong (UNC)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 1:45pm &#8211; 3:15pm<br />
<strong>Location: </strong>VEC 405</p>
<p><strong>Speech 1: Multi-layered Graphical Models</strong><br />
<strong>Speaker: Min Jin Ha (MD Anderson)</strong><br />
<strong>Abstract:</strong><br />
Simultaneous modeling of data arising from multiple ordered layers provides insight into the holistic picture of the interactive system and the flow of information. Chain graphs have been used to model the layered architecture of networks where the vertices can be naturally partitioned into ordered layers that exhibit undirected and directed acyclic relations within and between the layers. We develop a multi-layered Gaussian graphical model (mlGGM) to investigate conditional independence structures in probabilistic chain graphs. Our proposed model uses a Bayesian node-wise selection framework that coherently accounts for dependencies in the mlGGM. Using Bayesian variable selection strategies for each of the node-wise regressions allows for flexible modeling, sparsity and incorporation of edge-specific prior knowledge. Through simulated data generated from various scenarios, we demonstrate that our node-wise regression method outperforms other related multivariate regression-based methodologies. We apply mlGGM to identify integrative networks for key signaling pathways in kidney cancer and dynamic signaling networks using longitudinal proteomics data in breast cancer.</p>
<p><strong>Speech 2: Variable Selection for Highly Correlated Predictors</strong><br />
<strong>Speaker: Fei Xue (UIUC)<br />
Abstract: </strong>Penalty-based variable selection methods are powerful in selecting relevant covariates and estimating coefficients simultaneously. However, variable selection could fail to be consistent when covariates are highly correlated. The partial correlation approach has been adopted to solve the problem with correlated covariates. Nevertheless, the restrictive range of partial correlation is not effective for capturing signal strength for relevant covariates. In this paper, we propose a new Semi-standard PArtial Covariance (SPAC) which is able to reduce correlation effects from other predictors while incorporating the magnitude of coefficients. The proposed SPAC variable selection facilitates choosing covariates which have direct association with the response variable, via utilizing dependency among covariates. We show that the proposed method with the Lasso penalty (SPAC-Lasso) enjoys strong sign consistency in both finite-dimensional and high-dimensional settings under regularity conditions. Simulation studies and the ‘HapMap’ gene data application demonstrate that the proposed method outperforms the traditional Lasso, adaptive Lasso, SCAD, and Peter–Clark-simple (PC-simple) methods for highly correlated predictors.</p>
<p><strong>Speech 3: Support Vector Machine with Confidence</strong><br />
<strong>Speaker: Xingye Qiao (SUNY Binghamton)</strong><br />
<strong>Abstract: </strong>Classification with confidence (Lei, 2014) is a new type of problem in statistical learning. Its goal, in the binary case, is to identify two regions with a specific coverage probability for each class. We propose a support vector classifier to achieve classification with confidence. The classifier has two boundaries. An observation outside of the two boundaries is deemed to be from one of the two classes (with certain confidence), while the region between the boundaries is an ambiguity region which could belong to either class. In the theoretical study, we show a Fisher consistency result and that, with high probability, the resulting classifier can control the non-coverage rates and minimize the ambiguity. Efficient algorithms are developed and numerical studies are conducted to illustrate the effectiveness of the proposed method. This is a joint work with Wenbo Wang.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#f80e1325mp" href='index.html%3Fp=536.html' onclick='event.preventDefault()'>
		Session 14: Philosophy of Science and the New Paradigm of Data-Driven Science	</a>
	</div>
<div id="f80e1325mp" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Philosophy of Science and the New Paradigm of Data-Driven Science<br />
<strong>Organizer</strong>: Todd Kuffner (Washington U)<br />
<strong>Chair:</strong> Todd Kuffner (Washington U)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 1:45pm &#8211; 3:15pm<br />
<strong>Location: </strong>VEC 902/903</p>
<p><strong>Speech 1: Your Data-Driven Claims Must Still be Probed Severely</strong><br />
<strong>Speaker: Deborah Mayo (Virginia Tech)</strong><br />
<strong>Abstract: </strong>I analyse some of the philosophical claims from leaders of the “new paradigm” of data-driven science regarding the end of theory and the obsoleteness of scientific method. Science may be data intensive, but data are theory laden, so conceptual assumptions and biases must still be examined.  Science may be question driven, but good science can&#8217;t opt out of probing flaws in any potential answers to those questions. <strong><br />
</strong><br />
<strong>Speech 2: On the replicability of scientific studies</strong><br />
<strong>Speaker: Ian McKeague (Columbia)</strong><br />
<strong>Abstract: </strong></p>
<p>I will discuss a number of issues, both statistical and philosophical, related to the replicability and verification of scientific results.  In particular, I will discuss a recent proposal of Munafo and Davey Smith (Nature, 2018) that such verification requires disparate lines of evidence, a technique that they call triangulation.</p>
<p>&nbsp;</p>
<p><strong>Speech 3: Conducting Highly Principled Data Science: A Statistician&#8217;s Job and Joy</strong><br />
<strong>Speaker: Xiao-Li Meng (Harvard)</strong><br />
<strong>Abstract: </strong></p>
<p>This talk is based on a contribution to the special issue on &#8220;The Role of Statistics in the Era of Big Data&#8221; organized by <em>Statistics and Probability Letters,</em></p>
<p>with the title above and the following abstract:  &#8220;Highly Principled Data Science insists on methodologies that are: (1) scientifically justified; (2) statistically principled; and (3) computationally efficient. An astrostatistics collaboration, together with some reminiscences, illustrates the increased roles statisticians can and should play to ensure this trio, and to advance the science of data along the way. &#8220;</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#5e35cbdhe9" href='index.html%3Fp=538.html' onclick='event.preventDefault()'>
		Session 15: Advances in Bayesian methods for high-dimensional data	</a>
	</div>
<div id="5e35cbdhe9" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: <strong>Advances in Bayesian methods for high-dimensional data</strong><br />
<strong>Organizer</strong>: Howard Bondell (U. of Melbourne)<br />
<strong>Chair:</strong> Xuan Bi (Yale)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 1:45pm &#8211; 3:15pm<br />
<strong>Location: </strong>VEC 1202/1203</p>
<p><strong>Speech 1: The Graphical Horseshoe Estimator for Inverse Covariance Matrices</strong><br />
<strong>Speaker:  Anindya Bhadra (Purdue)</strong><br />
<strong>Abstract:</strong>We develop a new estimator of the inverse covariance matrix for high-dimensional multivariate normal data using the horseshoe prior. The proposed graphical horseshoe estimator has attractive properties compared to other popular estimators, such as the graphical lasso and the graphical smoothly clipped absolute deviation. The most prominent benefit is that when the true inverse covariance matrix is sparse, the graphical horseshoe provides estimates with small information divergence from the sampling model. The posterior mean under the graphical horseshoe prior can also be almost unbiased under certain conditions. In addition to these theoretical results, we also provide a full Gibbs sampler for implementing our estimator. MATLAB code is available for download from github at <a href="http://github.com/liyf1988/GHS"><strong>http://github.com/liyf1988/GHS</strong></a><strong>. </strong>The graphical horseshoe estimator compares favorably to existing techniques in simulations and in a human gene network data analysis. This is joint work with Yunfan Li and Bruce Craig at Purdue.<strong><br />
</strong><br />
<strong>Speech 2: Scalable MCMC for Bayes shrinkage priors<br />
Speaker: Anirban Bhattacharya (Texas A &amp; M)</strong><br />
<strong>Abstract: </strong>Gaussian scale mixture priors are common in high-dimensional Bayesian analysis. While optimization algorithms for the extremely popular Lasso and elastic net scale to dimension in the hundreds of thousands, Bayesian computation by Markov chain Monte Carlo (MCMC) is limited to problems an order of magnitude smaller. This is due to high computational cost per step and growth of the variance of time-averages as a function of dimension. We propose an MCMC algorithm for computation in these models that combines block updating and approximations of the Markov kernel to combat both of these factors. Our algorithm gives orders of magnitude speedup over the best existing alternatives in high-dimensional applications. We give theoretical guarantees for the accuracy of the approximation. Scalability of the algorithm is illustrated in an application to a genome wide association study with $N=2,267$ observations and $p=98,385$ predictors. The empirical results show that the new algorithm yields estimates with lower mean squared error, intervals with better coverage, and elucidates features of the posterior often missed by previous algorithms, including bimodality of marginals indicating uncertainty about which covariates belong in the model. This latter feature is an important motivation for a Bayesian approach to testing and selection in high dimensions.(joint work with James Johndrow &amp; Paulo Orenstein)<br />
<strong><br />
Speech 3: Clustering on the Sphere: State-of-the-art and a Poisson Kernel-Based Model<br />
Speaker: Marianthi Markatou (U. at Buffalo)</strong><br />
<strong>Abstract: </strong>Many applications of interest involve data that can be analyzed as unit vectors on a d-dimensional sphere. Specific examples include text mining, biology, astronomy and medicine. We present a clustering method based on mixtures of Poisson-kernel based densities on the high-dimensional sphere. We study connections of the Poisson kernel-based densities with other distributions appropriate for the analysis of directional data, prove identifiability of mixtures of the Poisson kernel-based densities model, convergence of the associated EM-type algorithm, and study its operational characteristics. We further propose an <em>empirical densities distance plot</em> for estimating the number of clusters in a Poisson kernel-based densities model. Finally, we propose a method to simulated data from Poisson kernel-based densities and exemplify our methods via application on real data sets and simulation experiments. Our experimental results show that the newly introduced model exhibits higher macro-precision and macro-recall than competing methods based on von Mises Fisher and Watson distributions. <em>This is joint work with Mojgan Golzy, Ph.D.</em></p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#500aa68xt1" href='index.html%3Fp=540.html' onclick='event.preventDefault()'>
		Session 16: High-dimensional machine learning methods	</a>
	</div>
<div id="500aa68xt1" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: High-dimensional machine learning methods<br />
<strong>Organizer</strong>: Annie Qu (UIUC)<br />
<strong>Chair:</strong> Annie Qu (UIUC)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 1:45pm &#8211; 3:15pm<br />
<strong>Location: </strong>VEC 1302/1303</p>
<p><strong>Speech 1: Latent Space Approaches to Community Detection in Dynamic Networks</strong><br />
<strong>Speaker: Yuguo Chen  (UIUC)</strong><br />
<strong>Abstract: </strong>Embedding dyadic data into a latent space has long been a popular approach to modeling networks of all kinds. While clustering has been done using this approach for static networks, we give two methods of community detection within dynamic network data, building upon the distance and projection models previously proposed in the literature. Our proposed approaches capture the time-varying aspect of the data, can model directed or undirected edges, inherently incorporate transitivity and account for each actor&#8217;s individual propensity to form edges. We provide Bayesian estimation algorithms, and apply these methods to a ranked dynamic friendship network and world export/import data.</p>
<p><strong>Speech 2: Classification for High-Dimensional Functional Data</strong><br />
<strong>Speaker: Taps Maiti (MSU)</strong><br />
<strong>Abstract: </strong>The functional magnetic resonance imaging (fMRI) records signals coming from different areas in human brains, which show activities and states of brains. This measurements result in high-dimensional time series, and each dimension represents a region in brains. In this work, we propose a classification technique of this kind of high-dimensional time series data while we discuss several competitive approaches. In addition, we present two classification applications to demonstrate the performance of your method. One is an alcoholic condition detection with Electroencephalography (EEG) data collected from electrodes placed on subject&#8217;s scalps, and the other is a resting state detection using resting state fMRI data from the OpenfMRI database. In both applications, we compare the performance of our method with some other classification methods.</p>
<p><strong>Speech 3: A unified approach for censored quantile regression</strong><br />
<strong>Speaker: Naveen Narisetty (UIUC)</strong><br />
<strong>Abstract:  </strong>In this talk, I will present a new and unified approach for estimation of quantile regression under censoring of arbitrary types. The proposed method is based on a variation of the data augmentation algorithm and easily adapts to different forms of censoring including doubly censored and interval censored data unlike existing methods which are mostly limited to single censoring. Moreover, the proposed estimators improve on the performance of the best known estimators with singly censored data. Empirical results demonstrating the fine performance of the proposed approach will be presented. This is joint work with Xiaorong Yang and Xuming He.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#f945abe56l" href='index.html%3Fp=542.html' onclick='event.preventDefault()'>
		Session 17: Recent development of Statistical Neuroimaging Analysis	</a>
	</div>
<div id="f945abe56l" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Recent development of Statistical Neuroimaging Analysis<br />
<strong>Organizer</strong>: Lexin Li (UC Berkeley)<br />
<strong>Chair:</strong> Jason Lee (USC)<br />
<strong>Time:</strong> June 4<sup>th</sup>, 1:45pm &#8211; 3:15pm<br />
<strong>Location: </strong>VEC 1402/1403</p>
<p><strong>Speech 1: Analyzing Non-Stationary High-Dimensional Time Series: Structural Break Detection and Parameter Estimation</strong><br />
<strong>Speaker: Ali Shojaie (U of Washington)</strong><br />
<strong>Abstract: </strong>Assuming stationarity is unrealistic in many time series applications, including neuroscience. A more realistic alternative is to assume piecewise stationarity, where the model is allowed to change at potentially many time points. We propose a three-stage procedure for consistent estimation of both structural change points and parameters of high-dimensional piecewise vector autoregressive (VAR) models. In the first step, we reformulate the change point detection problem as a high-dimensional variable selection one, and solve it using a penalized least square estimator with a total variation penalty. We show that the proposed penalized estimation method over-estimates the number of change points. We then propose a backward selection criterion to identify the change points. In the last step of our procedure, we estimate the VAR parameters in each of the segments. We show that the proposed procedure consistently detects the number of change points and their locations. We also show that the procedure consistently estimates the VAR parameters. The performance of the method is illustrated through several simulation studies, as well as an analysis of EEG data.</p>
<p><strong>Speech 2: Tensor-on-tensor regression</strong><br />
<strong>Speaker: Eric Lock (U of Minnesota)</strong><br />
<strong>Abstract: </strong>In neuroimaging analysis and other fields, both predictors and outcomes can take the form of a multi-way array (i.e., a tensor).  We propose a framework for the linear prediction of a multi-way array from another multi-way array of arbitrary dimension, using the contracted tensor product. This framework generalizes several existing approaches, including methods to predict a scalar outcome from a tensor, a matrix from a matrix, or a tensor from a scalar. We describe an approach that exploits the multiway structure of both the predictors and the outcomes by restricting the coefficients to have reduced CP-rank. We propose a general and efficient algorithm for penalized least-squares estimation, which allows for a ridge ($L_2$) penalty on the coefficients. The objective is shown to give the mode of a Bayesian posterior, which motivates a Gibbs sampling algorithm for inference. We illustrate the approach with an application to facial image data.</p>
<p><strong>Speech 3: A Joint Modeling Approach for Baseline Matrix-valued Imaging Data and Treatment Outcome</strong><br />
<strong>Speaker: Bei Jiang (University of Alberta)</strong><br />
<strong>Abstract: </strong>In this talk we propose a unified Bayesian joint modeling framework for studying association between a binary treatment outcome and a baseline matrix-valued predictor, such as imaging data. Under this framework, a theoretically implied relationship can be established between the treatment outcome and the matrix-valued imaging data, although the imaging data is not directly considered in the model.  The proposed joint modeling approach provides a promising framework for both association estimation and prediction. Properties of this method are examined using simulated datasets. Finally, a detailed illustration of the proposed modeling approach is provided using a motivating depression study that aims to explore the association between the baseline EEG data and the probability of a favorable response to an antidepressant treatment.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#7fcfcccbds" href='index.html%3Fp=544.html' onclick='event.preventDefault()'>
		Session 18: Nonparameteric and Robust Statistical Methods for Imaging	</a>
	</div>
<div id="7fcfcccbds" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Nonparameteric and Robust Statistical Methods for Imaging<br />
<strong>Organizer</strong>: Hernando Ombao (KAUST)<br />
<strong>Chair:</strong> Wei Pan(UMN)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 8:30am – 10:00am<br />
<strong>Location: </strong>VEC 405</p>
<p><strong>Speech 1: Nonparametric Collective Spectral Density Estimation and Clustering with Application to Brian Activities</strong><br />
<strong>Speaker: Mehdi Maadooliat (Marquette University)</strong><br />
<strong>Abstract: </strong>In this paper, we develop a method for the simultaneous estimation of spectral density functions (SDFs) for a collection of stationary time series that share some common features. Due to the similarities among the SDFs, the log-SDF can be represented using a common set of basis functions. The basis shared by the collection of the log-SDFs is estimated as a low-dimensional manifold of a large space spanned by a pre-specified rich basis. A collective estimation approach pools information and borrows strength across the SDFs to achieve better estimation efficiency. Also, each estimated spectral density has a concise representation using the coefficients of the basis expansion, and these coefficients can be used for visualization, clustering, and classification purposes. The Whittle pseudo-maximum likelihood approach is used to fit the model and an alternating blockwise Newton-type algorithm is developed for the computation. A web-based <a href="https://webmail.illinois.edu/owa/redir.aspx?C=voqRJ0NUY1dsJ3X0enoFXJoFzmmPq7OT6xNm1krWeNVZ26paopzVCA..&amp;URL=https%3a%2f%2furldefense.proofpoint.com%2fv2%2furl%3fu%3dhttps-3A__ncsde.shinyapps.io_NCSDE%26d%3dDwMFaQ%26c%3dOCIEmEwdEq_aNlsP4fF3gFqSN-E3mlr2t9JcDdfOZag%26r%3d8w4JPlCiB2g_eHkCaWvCdFR4JNn8fq_ZY8WrysjTMZ0%26m%3dgYY_QoeBrwx6BccM1JD_oGqTvdh2zfvJ-_jfdZ5CY38%26s%3dgyLnB3QP-NweFXFakDlsQNm1zRPP5nUe0uYC_dFzX3c%26e%3d">shiny App</a>found at <a href="https://webmail.illinois.edu/owa/redir.aspx?C=voqRJ0NUY1dsJ3X0enoFXJoFzmmPq7OT6xNm1krWeNVZ26paopzVCA..&amp;URL=https%3a%2f%2furldefense.proofpoint.com%2fv2%2furl%3fu%3dhttps-3A__ncsde.shinyapps.io_NCSDE%26d%3dDwMFaQ%26c%3dOCIEmEwdEq_aNlsP4fF3gFqSN-E3mlr2t9JcDdfOZag%26r%3d8w4JPlCiB2g_eHkCaWvCdFR4JNn8fq_ZY8WrysjTMZ0%26m%3dgYY_QoeBrwx6BccM1JD_oGqTvdh2zfvJ-_jfdZ5CY38%26s%3dgyLnB3QP-NweFXFakDlsQNm1zRPP5nUe0uYC_dFzX3c%26e%3d">https://ncsde.shinyapps.io/NCSDE</a> is developed for visualization, training and learning the SDFs collectively using the proposed technique. Finally, we apply our method to cluster similar brain signals recorded by the electroencephalogram for identifying synchronized brain regions according to their spectral densities.</p>
<p><strong>Speech 2: A Flexible Non-parametric Framework for Imaging Genetics</strong><br />
<strong>Speaker: Zhaoxia Yu (UC Irvine) </strong><br />
<strong>Abstract: </strong> Data collected in many scientific areas are inherently high-dimensional and multi-way. While such data provides an excellent opportunity for us to conduct an integrative analysis of multiple data modalities, it is challenging to model associations between sets of massive, complexly structured, and high-dimensional data. Here we propose a flexible, easy-to-implement, and non-parametric framework to assessing the overall association between high-dimensional modalities. We illustrate how the methods are connected to classical regression-based methods.  To take some important structure of brain imaging data into consideration, we also extend our method to non-Euclidean space. The principles that we propose are applicable to various types of high dimensional data, such as genetic variants and brain connectivity.</p>
<p><strong>Speech 3: Hybrid Principal Components Analysis For Region-Referenced Longitudinal Functional EEG Data</strong><br />
<strong>Speaker: Damla Senturk (UCLA)</strong><br />
<strong>Abstract: </strong>Electroencephalography (EEG) data possess a complex structure that includes regional, functional, and longitudinal dimensions. Our motivating example is a word segmentation paradigm in which typically developing (TD) children and children with Autism Spectrum Disorder (ASD) were exposed to a continuous speech stream. For each subject, continuous EEG signals recorded at each electrode were divided into one-second segments and projected into the frequency domain via Fast Fourier Transform. Following a spectral principal components analysis, the resulting data consist of region-referenced principal power indexed regionally by scalp location, functionally across frequencies and longitudinally by one-second segments. Standard EEG power analyses often collapse information across the longitudinal and functional dimensions by averaging power across segments and concentrating on specific frequency bands. We propose a hybrid principal components analysis (HPCA) for region-referenced longitudinal functional EEG data which utilizes both vector and functional principal components analyses and does not collapse information along any of the three dimensions of the data. The proposed decomposition only assumes weak separability of the higher-dimensional covariance process and utilizes a product of one dimensional eigenvectors and eigenfunctions, obtained from the regional, functional, and longitudinal marginal covariances, to represent the observed data, providing a computationally feasible non-parametric approach. A mixed effects framework is proposed to estimate the model components coupled with a bootstrap test for group level inference, both geared towards sparse data applications. Analysis of the data from the word segmentation paradigm leads to valuable insights about group-region differences among the TD and verbal and minimally verbal children with ASD. Finite sample properties of the proposed estimation framework and bootstrap inference procedure are further studied via extensive simulations.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#e4cd03dpei" href='index.html%3Fp=546.html' onclick='event.preventDefault()'>
		Session 19: Big Data of different forms and different challenges	</a>
	</div>
<div id="e4cd03dpei" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Big Data of different forms and different challenges<br />
<strong>Organizer</strong>: Regina Liu (Rutgers)<br />
<strong>Chair:</strong> Xuan Bi(Yale)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 8:30am – 10:00am<br />
<strong>Location: </strong>VEC 1202 /1203</p>
<p><strong>Speech 1: Individualized Multilayer Learning with An Application in Breast Cancer Imaging</strong><br />
<strong>Speaker: Annie Qu (UIUC)</strong><br />
<strong>Abstract: </strong>This work is motivated by breast cancer imaging data produced by a multimodal multiphoton optical imaging technique. One unique aspect of breast cancer imaging is that different individuals might have breast imaging at different locations, which also creates a technical difficulty in that the imaging background could vary for different individuals. We develop a multilayer tensor learning method to predict disease status effectively through utilizing subject-wise imaging information. In particular, we construct an individualized multilayer model which leverages an additional layer of individual structure of imaging in addition to employing a high-order tensor decomposition shared by populations. In addition, to incorporate multimodal imaging data for different profiling of tissue, cellular and molecular levels, we propose a higher order tensor representation to combine multiple sources of information at different modalities, so important features associated with disease status and clinical outcomes can be extracted effectively. One major advantage of our approach is that we are able to capture the spatial information of microvesicles observed in certain modalities of optical imaging through combining multimodal imaging data. This has medical and clinical significance since microvesicles are more frequently observed among cancer patients than healthy ones, and identification of microvesicles enables us to provide an effective diagnostic tool for early-stage cancer detection. This is joint work with Xiwei Tang and Xuan Bi.</p>
<p><strong>Speech 2:  Efficient estimation and fast algorithms for genetic microarray data with survival outcomes</strong><br />
<strong>Speaker: Catherine Chunling Liu (Polytech U of HK) </strong><br />
<strong>Abstract: </strong>In gene expression microarray studies,   genetic and genomic data tend to be  high- or ultrahigh- dimensional and are accompanied with random censored survival outcomes.  To search out and evaluate influence features that will impact on the disease, it  is imperative to develop  new modeling, efficient estimation methodology, and feasible algorithms within such data setting. In this talk, we will discuss in three aspects. First of all, for ultrahigh dimensional data modeled by the proportional hazard model, we present a non-monotone proximal gradient algorithm with lasso-type initial value to do feature screening and variable selection; Next, we recommend a single index hazard model without specifying the functional form. Efficient estimation procedures for index coefficients will facilitate detection of significance of individual effects. Finally we consider jointly modeling the mean and intensity function  involving multiple index structure and develop a unified methodology to conduct dimension reduction. A normal acute myeloid leukemia data is analyzed to demonstrate our approaches.</p>
<p><strong>Speech 3: Nonparametric mean estimation for big-but-biased data</strong><br />
<strong>Speaker: Ricardo Cao (Universidade da Coruña)</strong><br />
<strong>Abstract: </strong>Some authors have recently warned about the risks of the sentence &#8220;with enough data, the numbers speak for themselves&#8221;. Some of the problems coming from ignoring sampling bias in big data statistical analysis have been recently reported. The problem of nonparametric statistical inference in big data under the presence of sampling bias is considered in this work. The mean estimation problem is studied in this setup, in a nonparametric framework, when the biasing weight function is known (unrealistic) as well as for unknown weight functions (realistic). Two different scenarios are considered to remedy the problem of ignoring the weight function: (i) having a small sized simple random sample of the real population and (ii) having observed a sample from a doubly biased distribution. In both cases the problem is related to nonparametric density estimation. Asymptotic expressions for the mean squared error of the estimators proposed for scenario (i) are considered. This leads to asymptotic formulas for the optimal smoothing parameters. Some simulations illustrate the performance of the nonparametric methods proposed in this work.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#e586a67bou" href='index.html%3Fp=548.html' onclick='event.preventDefault()'>
		Session 20: OODA: Manifold Data Integration	</a>
	</div>
<div id="e586a67bou" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: OODA: Manifold Data Integration<br />
<strong>Organizer</strong>: Marron, James Stephen (UNC)<br />
<strong>Chair:</strong> Anna Smith (Columbia)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 8:30am – 10:00am<br />
<strong>Location: </strong>VEC 1302</p>
<p><strong>Speech 1: Random Domain Decomposition for Kriging Riemannian Data</strong><br />
<strong>Speaker: Piercesare Secchi (Politecnico di Milano)</strong><br />
<strong>Abstract: </strong></p>
<p>Investigation of object data distributed over complex domains gives rise to new challenges for spatial statistics. The linear methods of geostatistics cannot be directly applied when the embedding space for the observed data is non Hilbert. Moreover global assumptions about the  stationarity of the random field generating the data are often unsuitable when the spatial domain is large, textured or convoluted, with holes or barriers. We here examine the Random Domain Decomposition computational approach proposed by Menafoglio et al. (2018) applied to the analysis and prediction of tensor data observed over a complex spatial domain. As an illustrative case study, we will analyse the covariances between dissolved oxygen and water temperature in the Chesapeake Bay.</p>
<p>&nbsp;</p>
<p>This is joint work with Alessandra Menafoglio (Dept. of Mathematics, Politecnico di Milano) and Davide Pigoli (Dept. of Mathematics, King’s College London)</p>
<p>&nbsp;</p>
<p><strong>Speech 2: Nonparametric K-Sample Test on Riemannian Manifolds with Applications to Analyzing Mitochondrial Shapes</strong><br />
<strong>Speaker: Ruiyi Zhang (Florida State)</strong><br />
<strong>Abstract: </strong>This paper develops a nonparametric approach for a k-sample test involving data lying on a Riemannian manifold, such as a shape manifold. The specific problem is to test the hypothesis that a factor  (such as the subject, cell, or  living conditions) significantly affects mitochondrial morphology  as observed in images of skeletal muscles of mice. The fact that a shape space is non-Euclidean and infinite-dimensional rules out standard ANOVA decomposition and requires new ideas. In this work, we extend a metric-based approach, developed for Euclidean spaces previously and termed DISCO analysis, to the several shape representations of planar closed curves. This adaptation leads to a statistic for testing equality of distributions of across groups. We provide the underlying theory for one shape representation, but apply the test to several other shape metrics also. Since the data have a nested structure, we also develop a procedure to test a factor while it includes another significant factor. We analyze and present results for a mitochondria shape dataset, including an interesting result that a change in lifestyle alters shapes of some type of mitochondria.</p>
<p>&nbsp;</p>
<p><strong>Speech 3: High-Dimensional Manifold Data Clustering on Symmetric Spaces </strong><br />
<strong>Speaker: Chao Huang (UNC)</strong><br />
<strong>Abstract: </strong>Clustering is one of the fundamental tools in manifold learning, and it has been extensively studied in many applications. However, in many image analysis problems (e.g., directional data analysis, shape analysis), most existing clustering methods established in Euclidian space face several challenges including a symmetric space, a high dimensional feature space, and manifold data variation associated with some covariates. In order to address such challenges, a penalized model-based clustering framework is developed to cluster high dimensional manifold data in symmetric spaces. Specifically, a mixture of geodesic factor analyzers (MGFA) is proposed with mixing proportions de- fined through a logistic model and Riemannian normal distribution in each component for data in symmetric spaces. A geodesic factor analyzer is established to explicitly model the high dimensional features. Penalized likelihood approaches are used to real- ize variable selection procedures. Simulation studies are performed on data generated from unit sphere, and real data analysis are performed on the corpus callosum (CC) shape data from the ADNI study.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#dfce56cslw" href='index.html%3Fp=550.html' onclick='event.preventDefault()'>
		Session 21: Advances in high-dimensional statistics	</a>
	</div>
<div id="dfce56cslw" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Advances in high-dimensional statistics<br />
<strong>Organizer</strong>: Genevera Allen (Rice)<br />
<strong>Chair:</strong> Genevera Allen (Rice)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 8:30am – 10:00am<br />
<strong>Location: </strong>VEC 1303</p>
<p><strong>Speech 1: Adaptive local estimation for high dimensional linear models<br />
Speaker: Yufeng Liu (UNC)</strong><br />
<strong>Abstract: </strong>High dimensional linear models are commonly used in practice. In many applications, one is interested in linear transformations of regression coefficients such as prediction of the response. One common approach is the global technique which first estimates the coefficients, then plugs the estimator in the linear transformation for prediction. Despite its popularity, regression estimation can be difficult for high dimensional problems. Commonly used assumptions in the literature include that the signal of coefficients is sparse and predictors are weakly correlated. These assumptions, however, may not be easily verified, and can be violated in practice. When the coefficients are not sparse or predictors are strongly correlated, estimation of  coefficients can be very difficult. In this talk, I will present a new adaptive local estimator for linear transformations of the coefficients. This new estimator greatly relaxes the common assumptions for high dimensional problems. Simulation and theoretical results demonstrate the competitive advantages of the proposed method for a wide range of problems.<br />
<strong><br />
Speech 2:  Are Clusterings of Multiple Data Views Independent?</strong><br />
<strong>Speaker: Jacob Bien (Cornell)</strong><br />
<strong>Abstract: </strong>It has become increasingly common for scientists to collect more than one type of measurement on a single set of observations.  For instance, a medical researcher might gather both clinical measurements and DNA sequences on a single set of individuals.  Many &#8220;multi-view clustering&#8221; methods have been developed, which use the information from the different data views to determine a clustering of the observations.  In this talk, however, we explore a more basic question: is the clustering structure from each data view related or independent?  We develop a hypothesis test for investigating this question on the basis of a set of data.  This is joint work with Lucy Gao and Daniela Witten.</p>
<p><strong>Speech 3: Regularized Robust Buckley-James method for AFT Model with General Loss Function</strong><br />
<strong>Speaker: Sijian Wang (Rutgers)</strong><br />
<strong>Abstract:  </strong>In the last decade of genome research, one of the most popular topics is to relate large numbers of gene information to clinical survival phenotypes. As an alternative to the popular Cox’s proportional hazard model, the Accelerated Failure Time (AFT) model specifies an association between survival time and covariates directly. Consequently, it has a simpler and possible more intuitive interpretation than Cox’s model which is based on the hazard function. The Buckley-James method is a popular method to get the estimation for the AFT model. It iterates between the imputation of failure time for censored subjects and the estimation of regression coefficients. In the estimation step, the least square criterion with quadratic loss function is used to get the estimation. It is well known that, for regression with uncensored data, the traditional estimation resulting from the quadratic loss function may not be robust when the variability of response is high and/or there are outliers. In this talk, we proposed a regularized Robust Buckley-James method which can incorporate general loss functions including the absolute loss function, quantile loss function, Huber’s loss function and Tukey’s bisquare loss function. The proposed methods are demonstrated using simulation studies and analysis of a TCGA ovarian cancer dataset.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#792616d5zy" href='index.html%3Fp=552.html' onclick='event.preventDefault()'>
		Session 22: Causal Inference and Machine Learning	</a>
	</div>
<div id="792616d5zy" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Causal Inference and Machine Learning<br />
<strong>Organizer</strong>: Ryan Tibshirani (CMU)<br />
<strong>Chair: </strong>Vincent Joseph Dorie (Columbia)  <strong>     </strong><br />
<strong>Time:</strong> June 5<sup>th</sup>, 8:30am – 10:00am<br />
<strong>Location: </strong>VEC 1402<br />
<strong><br />
Speech 1: Nonparametric causal effects based on incremental propensity score interventions</strong><br />
<strong>Speaker: Edward Kennedy (CMU)</strong><br />
<strong>Abstract: </strong></p>
<p>Most work in causal inference considers deterministic interventions that set each unit&#8217;s treatment to some fixed value. However, under positivity violations these interventions can lead to non-identification, inefficiency, and effects with little practical relevance. Further, corresponding effects in longitudinal studies are highly sensitive to the curse of dimensionality, resulting in widespread use of unrealistic parametric models. We propose a novel solution to these problems: incremental interventions that shift propensity score values rather than set treatments to fixed values. Incremental interventions have several crucial advantages. First, they avoid positivity assumptions entirely. Second, they require no parametric assumptions and yet still admit a simple characterization of longitudinal effects, independent of the number of timepoints. For example, they allow longitudinal effects to be visualized with a single curve instead of lists of coefficients. After characterizing these incremental interventions and giving identifying conditions for corresponding effects, we also develop general efficiency theory, propose efficient nonparametric estimators that can attain fast convergence rates even when incorporating flexible machine learning, and propose a bootstrap-based confidence band and simultaneous test of no treatment effect. Finally we explore finite-sample performance via simulation, and apply the methods to study time-varying sociological effects of incarceration on entry into marriage.</p>
<p>&nbsp;</p>
<p><strong>Speech 2: Quasi-Oracle Estimation of Heterogeneous Causal Effects</strong><br />
<strong>Speaker: Stefan Wager (Stanford)</strong><br />
<strong>Abstract: </strong>Many scientific and engineering challenges, ranging from personalized medicine to customized marketing recommendations, require an understanding of treatment effect heterogeneity. In this paper, we develop a class of two-step algorithms for heterogeneous treatment effect estimation in observational studies. We first estimate marginal effects and treatment propensities to form an objective function that isolates the heterogeneous treatment effects, and then optimize the learned objective. This approach has several advantages over existing methods. From a practical perspective, our method is very flexible and easy to use: In both steps, we can use any method of our choice, e.g., penalized regression, a deep net, or boosting; moreover, these methods can be fine-tuned by cross-validating on the learned objective. Meanwhile, in the case of penalized kernel regression, we show that our method has a quasi-oracle property, whereby even if our pilot estimates for marginal effects and treatment propensities are not particularly accurate, we achieve the same regret bounds as an oracle who has a-priori knowledge of these nuisance components. We implement variants of our method based on both penalized regression and convolutional neural networks, and find promising performance relative to existing baselines.</p>
<p><strong>Speech 3: Off-policy Learning in Theory and in the Wild</strong><br />
<strong>Speaker: Yu-Xiang Wang (Amazon/UCSB)</strong><br />
<strong>Abstract: </strong>The talk considers the problem of offline policy learning for automated decision systems under the contextual bandits model, where we aim at evaluating the performance of a given policy (a decision algorithm) and also learning a better policy using logged historical data consisting of context, actions, rewards and probabilities of the actions taken. This is a generalization of the Average Treatment Effect (ATE) estimation problem and has some interesting new set of desiderata to consider.</p>
<p>In the first part of the talk, I will compare and contrast off-policy evaluation and ATE estimation and clarify how different assumptions change the corresponding minimax risk in estimating the &#8220;causal effect&#8221;. In addition, I will talk about how one can achieve significantly better finite sample performance than asymptotically optimal estimators through the SWITCH estimator.</p>
<p>In the second part of the talk, I will talk about off-policy evaluation and learning in a real industry environment. I will highlight several intersting challenges there including partially logged probilities, unobserved decision variables (Simpson&#8217;s paradox), effect of model bias and so on. We then propose and recommend practical ways to deal with these challenges under different circumstances.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#33fac29jvg" href='index.html%3Fp=554.html' onclick='event.preventDefault()'>
		Session 23: Decision making, operations research and statistical learning	</a>
	</div>
<div id="33fac29jvg" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Decision making, operations research and statistical learning<br />
<strong>Organizer</strong>: Cynthia Rudin (Duke)<br />
<strong>Chair:</strong> Cynthia Rudin (Duke)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 8:30am – 10:00am<br />
<strong>Location: </strong>VEC 1403</p>
<p><strong>Speech 1: </strong><strong>Online Learning of Buyer Behavior under Realistic Pricing Restrictions</strong><br />
<strong>Speaker:  Theja Tulabandhula (UIC)</strong><br />
<strong>Abstract: </strong>We propose a new efficient online algorithm to learn the parameters governing the purchasing behavior of a utility maximizing buyer, who responds to prices, in a repeated interaction setting. The key feature of our algorithm is that it can learn even non-linear buyer utility while working with arbitrary price constraints that the seller may impose. This overcomes a major shortcoming of previous approaches, which use unrealistic prices to learn these parameters making them unsuitable in practice.</p>
<p><strong>Speech 2: Smart &#8220;Predict, then Optimize&#8221;</strong><br />
<strong>Speaker: Adam Elmachtoub (Columbia)</strong><br />
<strong>Abstract:  </strong>We consider a class of optimization problems where the objective function is not explicitly provided, but contextual information can be used to predict the objective based on historical data. A traditional approach would be to simply predict the objective based on minimizing prediction error, and then solve the corresponding optimization problem. Instead, we propose a prediction framework that leverages the structure of the optimization problem that will be solved given the prediction. We provide theoretical, algorithmic, and computational results to show the validity and practicality of our framework. This is joint work with Paul Grigas (UC Berkeley).</p>
<p><strong>Speech 3: </strong><strong>P-splines with an l1 penalty for repeated measures</strong><br />
<strong>Speaker: Brian Segal (Flatiron Health)</strong><br />
<strong>Abstract: </strong>P-splines are penalized B-splines, in which finite order differences in coefficients are typically penalized with an l2 norm. P-splines can be used for semiparametric regression and can include random effects to account for within-subject variability. In addition to l2 penalties, l1-type penalties have been used in nonparametric and semiparametric regression to achieve greater flexibility, such as in locally adaptive regression splines, l1 trend filtering, and the fused lasso additive model. However, there has been less focus on using l1 penalties in P-splines, particularly for estimating conditional means. We demonstrate the potential benefits of using an l1 penalty in P-splines with an emphasis on fitting non-smooth functions. We propose an estimation procedure using the alternating direction method of multipliers and cross validation, and provide degrees of freedom and approximate confidence bands based on a ridge approximation to the l1 penalized fit. We also demonstrate potential uses through simulations and an application to electrodermal activity data collected as part of a stress study.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#b44def0swr" href='index.html%3Fp=556.html' onclick='event.preventDefault()'>
		Session 24: Novel inference approaches for complex data setting	</a>
	</div>
<div id="b44def0swr" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Novel inference approaches for complex data setting<br />
<strong>Organizer</strong>: Regina Liu (Rutgers)<br />
<strong>Chair:</strong> Junhui Wang (City U. of Hong Kong)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 1:15pm &#8211; 2:45pm<br />
<strong>Location: </strong>VEC 1202/1203</p>
<p><strong>Speech 2: Stein Discrepancy Methods for Robust Estimation and Regression</strong><br />
<strong>Speaker: Emre Barut (George Washington University)   </strong><br />
<strong>Abstract: </strong>All statistical procedures highly depend on the modeling assumptions and how close these assumptions are to reality. This dependence is critical: Even the slightest deviation from assumptions can cause major instabilities during statistical estimation.</p>
<p>In order to mitigate issues arising from model mismatch, numerous methods have been developed in the area of robust statistics. However, these approaches are aimed at specific problems, such as heavy tailed or correlated errors. The lack of a holistic framework in robust regression results in a major problem for the data practitioner. That is, in order to build a robust statistical model, possible issues in the data have to be found and understood before conducting the analysis. In addition, the practitioner needs to have an understanding of which robust models can be applied in which situations.</p>
<p>In this talk, we propose a new framework for robust parameter estimation to address these issues. The new method relies on the Stein Discrepancy Measure, and the estimate is given as the empirical minimizer of a second order U-statistic. The approach provides a “silver bullet” that can be used in a range of problems. When estimating parameters in the exponential family, the estimate can be obtained by solving a convex problem. For parameter estimation, our approach significantly improves upon MLE when outliers are present, or when the model is misspecified. Furthermore, we show how the new estimator can be used for robust high dimensional covariance estimation. Extensions of the method for regression problems and its efficient computation by subsampling are also discussed.</p>
<p><strong>Speech 2: Toward a sampling theory for statistical network analysis</strong><br />
<strong>Speaker: Harry Crane (Rutgers)</strong><br />
<strong>Abstract: </strong>Many classical network models (e.g., stochastic blockmodels, graphons, exponential random graph models) are ill-suited for modern applications because they implicitly assume that the data is obtained by an unrealistic sampling scheme, such as vertex selection or simple random vertex sampling.   More recent approaches (completely random measures and edge exchangeable models) improve somewhat upon these limitations, but leave plenty of room for further exploration of the role played by sampling in network analysis.  I present here a framework that is intended to overcome theoretical and practical issues arising from the use of ill-specified network models.  Within this framework I discuss how to incorporate the sampling scheme into statistical models in a way that is both flexible and insightful for modern network science applications.</p>
<p><strong>Speech 3: Estimating a covariance function from fragments of functional data</strong><br />
<strong>Speaker: Aurore Delaigle (U of Melbourne)</strong><br />
<strong>Abstract: </strong>Functional data are often observed only partially, in the form of fragments. In that case, the standard approaches for estimating the covariance function do not work because entire parts of the domain are completely unobserved. In previous work, Delaigle and Hall (2013, 2016) have suggested ways of estimating the covariance function, based for example on Markov assumptions. In this work we take a completely different approach which does not rely on such assumptions. We show that, using a tensor product approach, it is possible to reconstruct the covariance function using observations located only on the diagonal of its domain.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#4335458gst" href='index.html%3Fp=558.html' onclick='event.preventDefault()'>
		Session 25: New development for analyzing biomedical complex data	</a>
	</div>
<div id="4335458gst" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: New development for analyzing biomedical complex data<br />
<strong>Organizer</strong>: Zhezhen Jin (Columbia)<br />
<strong>Chair:</strong> Peng Wang (University of Cincinnati)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 1:15pm &#8211; 2:45pm<br />
<strong>Location: </strong>VEC 1302</p>
<p><strong>Speech 1: New methods for estimating follow-up rates in cohort studies</strong><br />
<strong>Speaker: Xiaonan Xue (Albert Einstein College of Medicine)</strong><br />
<strong>Abstract:<br />
</strong>Background: The follow-up rate, a standard index of the completeness of follow-up, is important for assessing the validity of a cohort study.   A common method for estimating the follow-up rate, the “Percentage Method”, defined as the fraction of all enrollees who developed the event of interest or had complete follow-up, can severely underestimate the degree of follow-up. Alternatively, the median follow-up time does not indicate the completeness of follow-up, and the reverse Kaplan-Meier based method and Clark’s Completeness Index (CCI) also have limitations.<br />
Methods:  We propose a new definition for the follow-up rate, the Person-Time Follow-up Rate (PTFR), which is the observed person-time divided by total person-time assuming no dropouts.  The PTFR cannot be calculated directly since the event times for dropouts are not observed.  Therefore, two estimation methods are proposed:  a formal person-time method (FPT) in which the expected total follow-up time is calculated using the event rate estimated from the observed data, and a simplified person-time method (SPT) that avoids estimation of the event rate by assigning full follow-up time to all events.  Simulations were conducted to measure the accuracy of each method, and each method was applied to a prostate cancer recurrence study dataset.<br />
Results:  Simulation results showed that the FPT has the highest accuracy overall. In most situations, the computationally simpler SPT and CCI methods are only slightly biased.  When applied to a retrospective cohort study of cancer recurrence, the FPT, CCI and SPT showed substantially greater 5-year follow-up than the Percentage Method (92%, 92% and 93% vs 68%).<br />
Conclusions: The Person-time methods correct a systematic error in the standard Percentage Method for calculating follow-up rates.   The easy to use SPT and CCI methods can be used in tandem to obtain an accurate and tight interval for PTFR. However, the FPT is recommended when event rates and dropout rates are high.<br />
<strong><br />
Speech 2: Mediation analysis with time-to-event mediator</strong><br />
<strong>Speaker: Mengling Liu (New York University)</strong><br />
<strong>Abstract: </strong>Mediation analysis is often employed in social and biomedical sciences to facilitate understanding of the effects that an intervention exerts over an outcome either directly or through a mediator. A motivating question is to study the pathway from circulating level of anti-Mullerian hormone (AMH, the exposure) to age at menopause (the mediator) and then to post-menopausal breast cancer (the outcome). However, challenge in the AMH mediation analysis is that the mediator variable, age at menopause, is a time-to-event variable and subject to censoring. Statistical methods to handle censored time-to-event mediator, or incompletely observed mediator in general, are lacking. We propose a series of statistical inference approaches for mediation analysis with a censored time-to-event mediator. Specifically, an estimating equation-based method is proposed for continuous outcomes and incorporates censored mediator through mean residual life (MRL) modeling; and a likelihood-based approach is proposed for categorical outcomes and estimates mediation effects through Monte Carlo methods to handle censored mediator.</p>
<p><strong>Speech 3: Adjustment for covariates in genome-wide association study</strong><br />
<strong>Speaker: Tao Wang (Albert Einstein College of Medicine)</strong> <strong>Abstract: </strong>Genome-wide association study (GWAS) has become a popular approach for identifying common genetic variants associated with complex diseases and quantitative traits. For continuous traits, linear regression model is a standard approach widely used in GWAS. Adjustment of covariates is often used to identify the direct effects of genetic variants or to improve statistical power by reducing variability of the trait. However, it is problematic to adjust for heritable covariates.  Here, we propose a new method for adjusting covariates by incorporating prior GWAS summary statistics for inferring the direct biological influence on a given trait and improve statistical power. Using simulation studies, the proposed methodology remains a good control of type I error rate under various situations and can achieve high power than a simple linear regression. The method is illustrated by an application to a GWAS analysis.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#a3fbb6ft71" href='index.html%3Fp=560.html' onclick='event.preventDefault()'>
		Session 26: New Statistical Machine Learning Tools	</a>
	</div>
<div id="a3fbb6ft71" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: New Statistical Machine Learning Tools<br />
<strong>Organizer</strong>: Liu, Yufeng (UNC)<br />
<strong>Chair:</strong> Liu, Yufeng (UNC)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 1:15pm &#8211; 2:45pm<br />
<strong>Location: </strong>VEC 201</p>
<p><strong>Speech 1: Inference, Computation, and Visualization for Convex Clustering and Biclustering</strong><br />
<strong>Speaker: Genevera Allen (Rice)</strong><br />
<strong>Abstract: </strong>Hierarchical clustering enjoys wide popularity because of its fast computation, ease of interpretation, and appealing visualizations via the dendogram and cluster heatmap.  Recently, several have proposed and studied convex clustering and biclustering which, similar in spirit to hierarchical clustering, achieve cluster merges via convex fusion penalties.  While these techniques enjoy superior statistical performance, they suffer from slower computation and are not generally conducive to representation as a dendogram.  In the first part of the talk, we present new convex (bi)clustering methods and fast algorithms that inherit all of the advantages of hierarchical clustering. Specifically, we develop a new fast approximation and variation of the convex (bi)clustering solution path that can be represented as a dendogram or cluster heatmap.  Also, as one tuning parameter indexes the sequence of convex (bi)clustering solutions, we can use these to develop interactive and dynamic visualization strategies that allow one to watch data form groups as the tuning parameter varies. In the second part of this talk, we consider how to conduct inference for convex clustering solutions that addresses questions like: Are there clusters in my data set? Or, should two clustersbe merged into one?  To achieve this, we develop a new geometric representation of Hotelling&#8217;s T^2-test that allows us to use the selective inference paradigm to test multivariate hypotheses for the first time.  We can use this approach to test hypotheses and calculate confidence ellipsoids on the cluster means resulting from convex clustering.  We apply these techniques to examples from text mining and cancer genomics. This is joint work with John Nagorski, Michael Weylandt, and Frederick Campbell.</p>
<p><strong>Speech 2: High-dimensional Cost-constrained Regression via Non-convex Optimization</strong><br />
<strong>Speaker: Guan Yu (SUNY Buffalo)</strong><br />
<strong>Abstract: </strong>In modern predictive modeling process, budget constraints become a very important consideration due to the high cost of collecting data using new techniques such as brain imaging and DNA sequencing. This motivates us to develop new and efficient high-dimensional costconstrained predictive modeling methods. In this paper, to address this challenge, we first study a new non-convex high-dimensional cost-constrained linear regression problem, that is, we aim to find the cost-constrained regression model with the smallest expected prediction error among all models satisfying a budget constraint. The non-convex budget constraint makes this problem NP-hard. In order to estimate the regression coefficient vector of the cost-constrained regression model, we propose a new discrete extension of recent first-order continuous optimization methods. In particular, our method delivers a series of estimates of the regression coefficient vector by solving a sequence of 0-1 knapsack problems that can be addressed by many existing algorithms such as dynamic programming efficiently. Next, we show some extensions of our proposed method for statistical learning problems using loss functions with Lipschitz continuous gradient. It can be also extended to problems with groups of variables or multiple constraints. Theoretically, we prove that the series of the estimates generated by our iterative algorithm converge to a first-order stationary point, which can be a globally optimal solution to the nonconvex high-dimensional cost-constrained regression problem. Computationally, our numerical studies show that the proposed method can solve problems of fairly high dimensions and has promising estimation, prediction, and model selection performance.</p>
<p><strong>Speech 3: </strong><strong>Modeling Hybrid Traits for Comorbidity and Genetic Studies of Alcohol and Nicotine Co-Dependence<br />
Speaker: Heping Zhang (Yale)</strong><br />
<strong>Abstract: </strong>I will present a novel multivariate model for analyzing hybrid traits and identifying genetic factors for comorbid conditions. Comorbidity is a common phenomenon in mental health in which an individual suffers from multiple disorders simultaneously. For example, in the Study of Addiction: Genetics and Environment (SAGE), alcohol and nicotine addiction were recorded through multiple assessments that we refer to as hybrid traits. Statistical inference for studying the genetic basis of hybrid traits has not been well-developed. Recent rank-based methods have been utilized for conducting association analyses of hybrid traits but do not inform the strength or direction of effects. To overcome this limitation, a parametric modeling framework is imperative. Although such parametric frameworks have been proposed in theory, they are neither well-developed nor extensively used in practice due to their reliance on complicated likelihood functions that have high computational complexity. Many existing parametric frameworks tend to instead use pseudo-likelihoods to reduce computational burdens. Here, we develop a model fitting algorithm for the full likelihood. Our extensive simulation studies demonstrate that inference based on the full likelihood can control the type-I error rate, and gains power and improves the effect size estimation when compared with several existing methods for hybrid models. These advantages remain even if the distribution of the latent variables is misspecified. After analyzing the SAGE data, we identify three genetic variants (rs7672861, rs958331, rs879330) that are significantly associated with the comorbidity of alcohol and nicotine addiction at the chromosome-wide level. Moreover, our approach has greater power in this analysis than several existing methods for hybrid traits. Although the analysis of the SAGE data motivated us to develop the model, it can be broadly applied to analyze any hybrid responses.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#3a97dc6oeh" href='index.html%3Fp=562.html' onclick='event.preventDefault()'>
		Session 27: Functional Data Analysis in Action	</a>
	</div>
<div id="3a97dc6oeh" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: <strong>Functional Data Analysis in Action</strong><br />
<strong>Organizer</strong>: <strong>Kehui Chen (U of Pitt)<br />
Chair:</strong> Kehui Chen (U of Pitt)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 1:15pm &#8211; 2:45pm<br />
<strong>Location: </strong>VEC 1402</p>
<p><strong>Speech 1: Brain Functional Connectivity &#8212; The FDA Approach</strong><br />
<strong>Speaker: Jane-Ling Wang (UC Davis)</strong><br />
<strong>Abstract: </strong>Functional connectivity refers to the connectivity between brain regions that share functional properties. It can be defined through statistical association or dependency among two or more anatomically distinct  brain regions.  In functional magnetic resonance imaging (fMRI), a standard way to measure brain functional connectivity is to assess the similarity of fMRI time courses for anatomically separated brain regions.  Due to the temporal nature of fMRI data, tools of functional data analysis (FDA) are intrinsically applicable to such data.  However, standard functional data techniques need to be modified when the goal is to study functional connectivity. We discuss two examples, where a new functional data approach is employed to study brain functional connectivity.</p>
<p>&nbsp;</p>
<p><strong>Speech 2: Functional Data Methods for Replicated Point Processes</strong><br />
<strong>Speaker: Daniel Gervini (U of Wisconsin at Milwaukee)</strong><br />
<strong>Abstract: </strong>Functional Data Analysis has traditionally focused on samples of smooth functions. However, many functional data methods can be extended to discrete point processes which are driven by smooth intensity functions. We will review some models that can be used for principal component analysis, joint modelling of discrete and continuous processes, and clustering of spatio-temporal point processes. We will apply these approaches to the analysis of spatio-temporal patterns in the distribution of crime and in the use of the shared-bicycle system in the city of Chicago.</p>
<p><strong>Speech 3:</strong> <strong>Frechet Regression for Time-Varying Covariance Matrices: Assessing Regional Co-Evolution in the Developing Brain</strong><br />
<strong>Speaker: Hans Mueller (UC Davis)<br />
Abstract: </strong>Frechet Regression provides an extension of Frechet means to the case of conditional Frechet means and is of interest for samples of random objects in a metric space (Petersen &amp; Müller 2018).  A specific application is encountered in cross-sectional studies where one observes $p$-dimensional vectors at one or a few random time points per subject and is interested in the p x p covariance or correlation matrix as a function of time. A challenge is that at each observation time one observes only a $p$-vector of measurements but not a covariance or correlation matrix. For a given metric on the space of covariance matrices, Frechet regression then generates a matrix function where at each fixed time the matrix is a non-negative definite covariance matrix. We demonstrate how this approach can be applied to MRI-extracted measurements of the myelin contents of various brain regions in small infants, aiming to quantify the regional co-evolution of myelination in the developing brain. Based on joint work with Alex Petersen and Sean Deoni.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#aba44cc9qx" href='index.html%3Fp=564.html' onclick='event.preventDefault()'>
		Session 28: Statistical Learning and Genomics	</a>
	</div>
<div id="aba44cc9qx" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Statistical Learning and Genomics<br />
<strong>Organizer</strong>: Ji Zhu (Umich)<br />
<strong>Chair:</strong> Bing Li (Penn State)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 1:15pm &#8211; 2:45pm<br />
<strong>Location: </strong>VEC 1403</p>
<p><strong>Speech 1: Proteomics and Genomics Integration for Translational Cancer Research</strong><br />
<strong>Speaker:  Umut Ozbek (Mount Sinai)</strong><br />
<strong>Abstract: </strong>Advances in biomedical research bring the opportunity to gather data in various platforms. Integrating those diverse data to understand complex biological<br />
systems has been a big challenge for statisticians. We propose a novel statistical tool, spaceMap; a conditional graphical model, which learns the conditional dependency relationships between two types of high dimensional omic profiles through a penalized multivariate regression framework. spaceMap infers an undirected graph among response variables in tandem with a directed graph encoding perturbations from predictor variables on the response network. In addition, it utilizes cross-validation and model aggregation to reduce the false<br />
discovery rate and consequently to improve reproducibility. We applied spaceMap to the copy number alterations, gene expression and proteomics datasets from CPTAC-TCGA ovarian cancer study. The results help to pinpoint crucial cancer genes and provide insights on the functional consequences of important CNA in the disease.<br />
<strong><br />
Speech 2: What can we gain from proteogenomics prediction? The downstream analysis of NCI-CPTAC Proteogenomics DREAM Challenge</strong><br />
<strong>Speaker: Xiaoyu Song (MSSM)</strong><br />
<strong>Abstract: </strong>Background: Proteins are complex macromolecules responsible for nearly every task of cellular life, and thus play an essential role in the formation, progression and metastasis of cancer. A community-based collaborative competition, NCI-CPTAC DREAM Proteogenomics Challenge, is developing computational tools to answer “Can one predict abundance of any given protein from mRNA and genetic data?” in sub-challenge 2 and “Can one predict phosphoprotein abundances from protein abundance?” and in sub-challenge 3. Methods and Results: In pairwise Pearson correlation analyses, we found the correlation between true protein/phosphoprotein abundances and their predicted scores from the top performing models varies dramatically from protein to protein. Therefore, we investigated the biological factors that influence the performance of predictions. We also applied the well-predicted proteins/phosphosites to 317 independent samples of ovarian cancer in TCGA for protein prediction and to 105 independent samples of ovarian cancer in CPTAC for phosphoprotein prediction. We found that the most significant overall survival associated pathways were repeatedly identified in the top performing models and both in training samples of the models and the independent samples. The utility of these prediction models in drug sensitivity analyses, cell lines and trans-cancer models have also been investigated. Conclusion: Proteogenomics prediction is promising to improve our understanding of molecular mechanisms of human cancer.<strong><br />
</strong><br />
<strong>Speech 3: An empirical comparison of deep neural networks and other<br />
supervised learning methods</strong><br />
<strong>Speaker: Wei Pan (U of Minnesota)</strong><br />
<strong>Abstract: </strong>Deep convolutional neural networks (CNN) have been proposed for supervised classification of high-throughput microscopy images to predict protein subcellular localization. We consider several CNN architectures in addition to a few traditional supervised learning methods such as random forest and gradient boosting. We compare their empirical performance when applied to a large dataset.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#29f1501qxd" href='index.html%3Fp=566.html' onclick='event.preventDefault()'>
		Session 29: Recent advances in high-dimensional data	</a>
	</div>
<div id="29f1501qxd" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Recent advances in high-dimensional data<br />
<strong>Organizer</strong>: Cunhui Zhang (Rutgers)<br />
<strong>Chair: </strong>Sijian Wang (Rutgers)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 3:15pm &#8211; 4:45pm<br />
<strong>Location: </strong>VEC 1202</p>
<p><strong>Speech 1: The noise barrier and the large signal bias of the Lasso and other convex estimators</strong><br />
<strong>Speaker: Pierre Bellec (Rutgers)</strong><br />
<strong>Abstract: </strong>Convex estimators such as the Lasso, the matrix Lasso and the group Lasso have been studied extensively in the last two decades, demonstrating great success in both theory and practice.  This paper introduces two quantities, the noise barrier and the large scale bias, that provides novel insights on the performance of these convex regularized estimators. In sparse linear regression, it is now well understood that the Lasso achieves fast prediction rates, provided that the correlations of the design satisfy some Restricted Eigenvalue or Compatibility condition, and provided that the tuning parameter is at least larger than some universal threshold.  Using the two quantities introduced in the paper, we show that the compatibility condition on the design matrix is actually unavoidable to achieve fast prediction rates with the Lasso.  In other words, the $\ell_1$-regularized Lasso must incur a loss due to the correlations of the design matrix, measured in terms of the compatibility constant.  This results holds for any design matrix, any active subset of covariates, and any positive tuning parameter.It is now well known that the Lasso enjoys a dimension reduction property: if the target vector is $s$-sparse, the prediction rate of the Lasso with tuning parameter $\lambda$ is of order \lambda\sqrt s$, even if the ambient dimension $p$ is much larger than $p$.  Such results require that the tuning parameters is greater than some universal threshold.  We characterize sharp phase transitions for the tuning parameter of the Lasso around a critical threshold dependent on $s$.  If $\lambda$ is equal or larger than this critical threshold, the Lasso is minimax over $s$-sparse target vectors.  If $\lambda$ is equal or smaller than critical threshold, the Lasso incurs a loss of order $\sigma\sqrt s$ &#8211;which corresponds to a model of size $s$&#8211; even if the target vector is more sparse than $s$.Remarkably, the lower bounds obtained in the paper also apply to random, data-driven tuning parameters.  Additionally, the results extend to convex penalties beyond the Lasso.<br />
<strong><br />
Speech 2: Factor-Driven Two-Regime Regression</strong><br />
<strong>Speaker: Yuan Liao (Rutgers)</strong><br />
<strong>Abstract: </strong>We propose a novel two-regime regression model, where the switching between the regimes is driven by a vector of possibly unobservable factors that are estimated from a  much larger panel data set. Estimating this model brings new challenges in terms of both computationand asymptotic theory. We show that our optimization problem can be reformulated as Mixed Integer Optimization and present two alternative computational algorithms. We also derive the asymptotic distribution of the resulting estimator and find that the effectof estimating the factors results in a phase transition on the rates of convergence and asymptotic distributions.  (Joint with Lee S, Seo M, and Shin Y)</p>
<p><strong>Speech 3: Network Analysis by SCORE</strong><br />
<strong>Speaker: Jiashun Jin (CMU)</strong><br />
<strong>Abstract: </strong>We have collected a data set for the networks of statisticians, consisting of titles, authors, abstracts, MSC numbers, keywords, and citation counts of papers published in representative journals in statistics and related fields. In Phase I of our study, the data set covers all published papers from 2003 to 2012 in Annals of Statistics, Biometrika, JASA, and JRSS-B. In Phase II of our study, the data set covers all published papers in 36 journals in statistics and related fields, spanning 40 years. The data sets motivate an array of interesting problems, and for the talk, I will focus on two closely related problems: network community detection, and network membership estimation. We tackle these problems with the recent approach of Spectral Clustering On Ratioed Eigenvectors (SCORE), reveal a surprising simplex structure underlying the networks, and explain why SCORE is the right approach. We use the methods to investigate the Phase I data and report some of the results.We also report some Exploratory Data Analysis (EDA) results including productivity, journal-journal citations, and citation patterns. This part of result is based on Phase II of our data set (ready for use not very long ago).</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#b0b1e0fi4d" href='index.html%3Fp=569.html' onclick='event.preventDefault()'>
		Session 30: Interpretable modeling and understanding variables	</a>
	</div>
<div id="b0b1e0fi4d" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Interpretable modeling and understanding variables<br />
<strong>Organizer</strong>: Cynthia Rudin (Duke)<br />
<strong>Chair:</strong> Cynthia Rudin (Duke)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 3:15pm &#8211; 4:45pm<br />
<strong>Location: </strong>VEC 1203</p>
<p><strong>Speech 1: Model Class Reliance: Variable Importance when all Models are Wrong, but *Many* are Useful.</strong><br />
<strong>Speaker</strong>: Aaron Fisher (Harvard)<br />
<strong>Abstract: </strong>Variable importance (VI) tools are typically used to examine the inner workings of prediction models. However, many existing VI measures are not comparable across model types, can obscure implicit assumptions about the data generating distribution, or can give seemingly incoherent results when multiple prediction models fit the data well. In this paper we propose a framework of VI measures for describing how much any model class (e.g. all linear models of dimension p), any model-fitting algorithm (e.g. Ridge regression with fixed regularization parameter), or any individual prediction model (e.g. a single linear model with fixed coefficient vector), relies on covariate(s) of interest. The building block of our approach, Model Reliance (MR), compares a prediction model&#8217;s expected loss with that model&#8217;s expected loss on a pair of observations in which the value of the covariate of interest has been switched. Expanding on MR, we propose Model Class Reliance (MCR) as the upper and lower bounds on the degree to which any well-performing prediction model within a class may rely on a variable of interest, or set of variables of interest. Thus, MCR describes reliance on a variable while accounting for the fact that many prediction models, possibly of different parametric forms, may fit the data well. We give probabilistic bounds for MR and MCR, using existing results for U-statistics. These bounds can be generalized to create finite-sample confidence regions for the best-performing models from any class. We also illustrate connections between MR, conditional causal effects, and linear regression coefficients. We then apply MR &amp; MCR in a public dataset of Broward County criminal records to study the reliance of recidivism prediction models on sex and race.</p>
<p><strong>Speech 2: Feature-Efficient Multi-value Rule Sets for Interpretable Classification</strong><br />
<strong>Speaker</strong>: Tong Wang (U Iowa)<br />
<strong>Abstract:</strong></p>
<p>We present Multi-vAlue Rule Set (MARS) models for interpretable classification with feature efficient presentations. Compared to rule sets built from single-valued rules, MARS introduces a more generalized form of association rules that allows multiple values in a condition. Rules of this form are more concise than classical single-valued rules in capturing and describing patterns in data. Our formulation also pursues a higher efficiency of feature utilization, which reduces possible cost in data collection and storage. We propose a Bayesian framework for formulating a MARS model and propose an efficient inference method for learning a maximum a posteriori, incorporating theoretically grounded bounds to iteratively reduce the search space and improve the search efficiency. Experiments on synthetic and real-world data demonstrate that MARS models have significantly smaller complexity and fewer features than baseline models while being competitive in predictive accuracy. We conducted a usability study with human subjects and the results show that MARS is the easiest to understand compared with other competing rule-based models. We apply MARS to a real-world application to predict in-hospital mortality rate.</p>
<p><strong>Speech 3: Recent Work on Interpretable Machine Learning Models</strong><br />
<strong>Speaker</strong>: Cynthia Rudin (Duke)<br />
<strong>Abstract: </strong> I will overview some work on interpretable machine learning models including: (i) one-sided decision trees (rule lists) that provably minimize accuracy and sparsity, (ii) falling rule lists, which are constrained one-sided decision trees, (iii) deep neural networks with an interpretable prototype layer, (iv) matching methods for causal inference that use machine learning to gain both interpretability and accuracy.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#e5ceaf5ra3" href='index.html%3Fp=571.html' onclick='event.preventDefault()'>
		Session 31: Statistical Inference for High-Dimensional Data	</a>
	</div>
<div id="e5ceaf5ra3" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Statistical Inference for High-Dimensional Data<br />
<strong>Organizer</strong>: Jeff Simonoff (NYU)<br />
<strong>Chair:</strong> Jeff Simonoff (NYU)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 3:15pm &#8211; 4:45pm<br />
<strong>Location: </strong>VEC 1302<br />
<strong><br />
Speech 1: Quantile Regression for big data with small memory</strong><br />
<strong>Speaker: Xi Chen (NYU)</strong><br />
<strong>Abstract:  </strong>In this talk, we discuss the inference problem of quantile regression for a large sample size $n$ (and a diverging dimensionality p) but under a limited memory constraint, where the memory can only store a small batch of data of size $m$. A popular approach, the naive divide-and-conquer method, only works when $n=o(m^2)$ and is computationally expensive. This talk proposes a novel inference approach and establishes the asymptotic normality result that achieves the same efficiency as the quantile regression estimator computed on all the data. Essentially, our method can allow arbitrarily large sample size $n$ as compared to the memory size $m$. Our method can also be applied to address the quantile regression under distributed computing environment (e.g., in a large-scale sensor network) or for real-time streaming data.</p>
<p><strong>Speech 2: Inference after cross-validation</strong><br />
<strong>Speaker: Joshua Loftus (NYU)</strong><br />
<strong>Abstract: </strong>We described a method for performing inference on models chosen by cross-validation. When the test error being minimized in cross-validation is a residual sum of squares it can be written as a quadratic form. This allows us to apply the inference framework in Loftus et al. (2016) for models determined by quadratic constraints to the model that minimizes CV test error. Our only requirement on the model training procedure is that its selection events are regions satisfying linear or quadratic constraints. This includes both Lasso and forward stepwise, which serve as our main examples throughout. We do not require knowledge of the error variance. The overall procedure is a computationally intensive method of selecting models adaptively and performing inference for the selected model.</p>
<p><strong>Speech 3: Optimal estimation of Gaussian mixtures via denoised method of moments</strong><br />
<strong>Speaker: Yihong Wu (Yale)</strong><br />
<strong>Abstract: </strong>The Method of Moments is one of the most widely used methods in statistics for parameter estimation, obtained by solving the system of equations that match the population and estimated moments. However, in practice and especially for the important case of mixture models, one frequently needs to contend with the difficulties of non-existence or non-uniqueness of statistically meaningful solutions, as well as the high computational cost of solving large polynomial systems. Moreover, theoretical analysis of method of moments are mainly confined to asymptotic normality style of results established under strong assumptions.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#2dacccdczd" href='index.html%3Fp=573.html' onclick='event.preventDefault()'>
		Session 32: New Development on Neuroimage Data Analysis	</a>
	</div>
<div id="2dacccdczd" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: New Development on Neuroimage Data Analysis<br />
<strong>Organizer</strong>: Zhu, Hongtu (MD Anderson)<br />
<strong>Chair:</strong> Xuan Bi(Yale)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 3:15pm &#8211; 4:45pm<br />
<strong>Location: </strong>VEC 1303</p>
<p><strong>Speech 1: </strong><strong>A Low-Rank Multivariate General Linear Model forMulti-Subject fMRI Data and a Non-Convex Optimization Algorithm for Brain Response Comparison</strong><br />
<strong>Speaker: Tingting Zhang (UVA)</strong><br />
<strong>Abstract:<br />
</strong>The focus of this paper is on evaluating brain responses to different stimuli and identifying brain regions with different responses using multi-subject, stimulus-evoked functional magnetic resonance imaging (fMRI) data. To jointly model many brain voxels’ responses to designed stimuli, we present a new low-rank multivariate general linear model (LRMGLM) for stimulus-evoked fMRI data. The new model not only is flexible to characterize variation in hemodynamic response functions (HRFs) across different regions and stimulus types, but also enables information “borrowing” across voxels and uses much fewer parameters than typical nonparametric models for HRFs. To estimate the proposed LRMGLM, we introduce a new penalized optimization function, which leads to temporally and spatially smooth HRF estimates. We develop an efficient optimization algorithm to minimize the optimization function and identify the voxels with different responses to stimuli. We show that the proposed method can outperform several existing voxel-wise methods by achieving both high sensitivity and specificity. We apply the proposed method to the fMRI data collected in an emotion study, and identify anterior dACC to have different responses to a designed threat and control stimuli.</p>
<p><strong>Speech 2: Nonparametric Bayes Models of Fiber Curves Connecting Brain Regions</strong><br />
<strong>Speaker: Zhengwu Zhang (Rochester)</strong><br />
<strong>Abstract: </strong></p>
<p>In studying structural inter-connections in the human brain, it is common to first estimate fiber bundles connecting different regions relying on diffusion MRI.  These fiber bundles act as highways for neural activity.  Current statistical methods reduce the rich information into an adjacency matrix, with the elements containing a count of fibers or a mean diffusion feature along the fibers.  The goal of this article is to avoid discarding the rich geometric information of fibers, developing flexible models for characterizing the population distribution of fibers between brain regions of interest within and across different individuals.  We start by decomposing each fiber into a rotation matrix, shape and translation from a global reference curve.  These components are viewed as data lying on a product space composed of different Euclidean spaces and manifolds.  To non-parametrically model the distribution within and across individuals, we rely on a hierarchical mixture of product kernels specific to the component spaces.  Taking a Bayesian approach to inference, we develop efficient methods for posterior sampling.  The approach automatically produces clusters of fibers within and across individuals. Applying the method to Human Connectome Project data, we find an interesting relationship between brain fiber geometry and reading ability.</p>
<p><strong><br />
Speech 3: Supervised Principal Component Regression for Functional Data with High Dimensional Predictors<br />
Speaker: Dehan Kong (U Toronto)</strong><br />
<strong>Abstract: </strong>Motived by functional magnetic resonance imaging studies, we study the effect of clinical/demographic variables on the dynamic functional structures, which plays a key role in understanding brain functionality. To this end, we propose the supervised principal component regression for functional data with possibly high dimensional clinical variables. Compared with its classical counterpart, the principal component regression, the proposed methodology relies on a newly proposed integrated residual sum of squares for functional data and makes use of the association information directly. It can be formulated as a sequence of nonconvex generalized Rayleigh quotient optimization problems, which turn out to be NP-hard and thus computational intractable. Utilizing the invariance property of linear subspaces under rotations, we then reformulate the problem into a simultaneous sparse linear regression problem. Formally, we show that the reformulated problem can recover the same subspace as if the original sequence of nonconvex problems were solved. Statistically, the rate of convergence for the obtained estimators is established. Numerical studies and an application to the Human Connectome Project lend further support to the proposed methodology. (Joint work with Xinyi Zhang and Qiang Sun)</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#8703de6pw1" href='index.html%3Fp=575.html' onclick='event.preventDefault()'>
		Session 33: Spectral Clustering, Functional Graphical Models, and Hierarchical Interactions	</a>
	</div>
<div id="8703de6pw1" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: <strong>Spectral Clustering, Functional Graphical Models, and Hierarchical Interactions</strong><br />
<strong>Organizer</strong>: Lingzhou Xue (Penn State)<br />
<strong>Chair:</strong> Kuang-Yao Lee (Temple)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 3:15pm &#8211; 4:45pm<br />
<strong>Location: </strong>VEC 1402</p>
<p><strong>Speech 1: Spectral clustering based on learning similarity matrix </strong><br />
<strong>Speaker:</strong> Hongyu Zhao (Yale)<br />
<strong>Abstract:  </strong>Single-cell RNA-sequencing (scRNA-seq) technology can generate genome-wide expression data at the single-cell levels. One important objective in scRNA-seq analysis is to cluster cells where each cluster consists of cells belonging to the same cell type based on gene expression patterns. In this presentation, we will introduce a novel spectral clustering framework that imposes sparse structures on a target matrix. Specifically, we utilize multiple doubly stochastic similarity matrices to learn a similarity matrix, motivated by the observation that each similarity matrix can be a different informative representation of the data. We impose a sparse structure on the target matrix followed by shrinking pairwise differences of the rows in the target matrix, motivated by the fact that the target matrix should have these structures in the ideal case. We solve the proposed non-convex problem iteratively using the ADMM algorithm and show the convergence of the algorithm. We evaluate the performance of the proposed clustering method on various simulated as well as real scRNA-seq data, and show that it can identify clusters accurately and robustly. This is joint work with Seyoung Park.</p>
<p><strong>Speech 2: Copula Gaussian Graphical Models for Functional Data </strong><br />
<strong>Speaker</strong>: Bing Li (Penn State)<br />
<strong>Abstract: </strong>We consider the problem of constructing statistical graphical models for functional data; that is, the observations on the vertices are random functions. This types of data are common in medical applications such as EEG and fMRI. Recently published functional graphical models rely on the assumption that the random functions are Hilbert-space-valued Gaussian random elements. We relax this assumption by introducing a copula Gaussian random elements Hilbert spaces, leading to what we call the Functional Copula Gaussian Graphical Model (FCGGM). This model removes the marginal Gaussian assumption but retains the simplicity of the Gaussian dependence structure, which is particularly attractive for large data. We develop four estimators, together with their implementation algorithms, for the FCGGM. We establish the consistency and the convergence rates of one of the estimators under different sets of sufficient conditions with varying strengths. We compare our FCGGM with the existing functional Gaussian graphical model by simulation, under both non-Gaussian and Gaussian graphical models, and apply our method to an EEG data set to construct brain networks. This is a joint work with Eftychia Solea.</p>
<p><strong>Speech 3:</strong> <strong>Learning Nonconvex Hierarchical Interactions</strong><br />
<strong>Speaker:</strong> Lingzhou Xue (Penn State)<br />
<strong>Abstract: </strong>In this talk, we will focus on learning nonconvex hierarchical interactions in high-dimensional statistical models. We first use the affine sparsity constraints to provide a precise characterization of both strong and weak hierarchical interactions. However, these affine sparsity constraints do not lead to a closed feasible region. To address this issue, we derive the explicit closure of the affine sparsity constraint for learning nonconvex hierarchical interactions. We prove that the global solution can be found by solving a sequence of folded concave penalized estimation and the desired strong or weak hierarchy holds with probability one. Furthermore, we study the local convergence theory for learning hierarchical interactions using the folded concave penalized estimation. Numerical studies are used to demonstrate the power of our proposed methods.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#b7a0c3206d" href='index.html%3Fp=577.html' onclick='event.preventDefault()'>
		Session 34: Data Science in IT Industries	</a>
	</div>
<div id="b7a0c3206d" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Data Science in IT Industries<br />
<strong>Organizer</strong>: David Banks (Duke)<br />
<strong>Chair:</strong> Genevera Allen(Rice)<br />
<strong>Time:</strong> June 5<sup>th</sup>, 3:15pm &#8211; 4:45pm<br />
<strong>Location: </strong>VEC 1403</p>
<p><strong>Speech 1 : Using Data Science to Improve Streaming Quality at Netflix<br />
Speaker:  </strong>Julie Novak (Netflix)<strong><br />
Abstract: </strong>In this talk, I will begin by giving an overview of the data science challenges involved in providing an optimal streaming service at Netflix. There are many dimensions to this problem, including selecting best picture quality based on network speed, determining proper content to cache on Netflix’s Content Delivery Networks (CDN), and improving each customer’s Quality of Experience (QoE). The talk will then dive deeper into the notion of QoE by explaining how to use statistical tools to measure and gain deeper understanding of it in the context of A/B testing.<br />
<strong><br />
Speech 2: Random Forests, Decision Trees, and Categorical Predictors: The &#8220;Absent Levels&#8221; Problem<br />
Speaker:  </strong>Tim Au (Google)<br />
<strong>Abstract:</strong> One advantage of decision tree based methods like random forests is their ability to natively handle categorical predictors without having to first transform them (e.g., by using one-hot encoding). However, in this talk, we show how this capability can lead to an inherent “absent levels” problem for decision tree based methods that has never been thoroughly discussed, and whose consequences have never been carefully explored. This problem occurs whenever there is an indeterminacy over how to handle an observation that has reached a categorical split which was determined when the observation in question’s level was absent during training. Although these incidents may appear to be innocuous, by using Leo Breiman and Adele Cutler’s random forests FORTRAN code and the randomForest R package (Liaw and Wiener, 2002) as motivating case studies, we study how overlooking the absent levels problem can systematically bias a model. Furthermore, by using three real data examples, we illustrate how absent levels can dramatically alter a model’s performance in practice, and we empirically demonstrate how some simple heuristics can be used to help mitigate the effects of the absent levels problem until a more robust theoretical solution is found.</p>
<p><strong>Speech 3: </strong><strong>The Challenge of Educating Data Scientists for Industry</strong><br />
<strong>Speaker:</strong> David Banks (Duke University and SAMSI)<br />
<strong>Abstract: </strong>The statistical world is changing quickly, and our graduate programs are (generally) not keeping pace.  This talk reviews some of the structural and cultural barriers that we need to overcome.  Besides proposing a model curriculum, it also addresses ways in which our publication processes now longer serve the interests of our profession, and it discusses the commodification of analysis.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#cf280840u3" href='index.html%3Fp=579.html' onclick='event.preventDefault()'>
		Session 35: Machine Learning and Precision Medicine	</a>
	</div>
<div id="cf280840u3" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Machine Learning and Precision Medicine<br />
<strong>Organizer</strong>: Haoda Fu (Eli Lilly)<br />
<strong>Chair:</strong> Genevera Allen(Rice)<br />
<strong>Time:</strong> June 6<sup>th</sup>, 8:30am – 10:00am<br />
<strong>Location: </strong>VEC 404 /405</p>
<p><strong>Speech 1:  Support vector machines for learning optimal individualized treatment rules with multiple treatments</strong><br />
<strong>Speaker: Donglin Zeng (UNC)</strong><br />
<strong>Abstract: </strong>Support vector machine (SVM) methods have been proposed to estimate optimal individualized treatment rules when treatment is binary. Extending SVM to more than two treatment options remains an open and challenging problem. In this work, we propose a novel and efficient algorithm to generalize SVM-based outcome weighted learning to a multi-treatment setting.  The proposed method sequentially solves binary SVM problems. Theoretically, we show that the resulting treatment rule is Fisher consistent and derive the convergence rate for the estimated value function. We conduct extensive simulation studies to demonstrate that the proposed method has superior performance to competing methods.</p>
<p><strong>Speech 2: Individualized Treatment Recommendation (ITR) for SurvivalOutcomes</strong><br />
<strong>Speaker: Haoda Fu (Eli Lilly)</strong><br />
<strong>Abstract: </strong>ITR is a method to recommend treatment based on individual patient characteristics to maximize clinical benefit. During the past a few years, we have developed and published methods on this topic with various applications including comprehensive search algorithms, tree methods , benefit risk algorithm, multiple treatment &amp; multiple ordinal treatment algorithms. In this talk, we propose a new ITR method to handle survival outcomes for multiple treatments. This new model enjoy the following practical and theoretical features</p>
<ol>
<li>Instead of fitting the data, our method directly search the optimal treatment policy which improvesthe efficiency</li>
<li>To adjust censoring, we propose a doubly robust estimator. Our method only requires either censoring model or survival model is correct, but not both.  When both are correct, our method enjoys better efficiency</li>
<li>Our method handles multiple treatments with intuitive geometry explanations</li>
<li>Our method is Fisher’s consistent even under either censoring model or survival model misspecification (but not both).</li>
</ol>
<p>&nbsp;</p>
<p>This method has potential applications in multiple therapeutic areas. One direct impact for Diabetes business unit is that how we can leverage Lilly Diabetes’ broad treatment options to reduce or delay diabetes comorbidities such as CV event, diabetes related retinopathy, nephropathy, or neuropathy.</p>
<p><strong>Speech 3: Estimation and Evaluation of Linear Individualized Treatment Rules to Guarantee Performance</strong><br />
<strong>Speaker: Yuanjia Wang (Columbia)</strong><br />
<strong>Abstract:  </strong>In clinical practice, an informative and practically useful treatment rule should be simple and transparent. However, because simple rules are likely to be far from optimal, effective methods to construct such rules must guarantee performance, in terms of yielding the best clinical outcome (highest reward) among the class of simple rules under consideration. Furthermore, it is important to evaluate the benefit of the derived rules on the whole sample and in pre-specified subgroups (e.g., vulnerable patients). To achieve both goals, we propose a robust machine learning method to estimate a linear treatment rule that is guaranteed to achieve optimal reward among the class of all linear rules. We then develop a diagnostic measure and inference procedure to evaluate the benefit of the obtained rule and compare it with the rules estimated by other methods. We provide theoretical justification for the proposed method and its inference procedure, and we demonstrate via simulations its superior performance when compared to existing methods. Lastly, we apply the method to the Sequenced Treatment Alternatives to Relieve Depression (STAR*D) trial on major depressive disorder and show that the estimated optimal linear rule provides a large benefit for mildly depressed and severely depressed patients but manifests a lack-of-fit for moderately depressed patients.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#0638bf1l64" href='index.html%3Fp=581.html' onclick='event.preventDefault()'>
		Session 36:  Advances in Nonparametric Statistics and their Applications	</a>
	</div>
<div id="0638bf1l64" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Advances in Nonparametric Statistics and their Applications<br />
<strong>Organizer</strong>: Narisetty, Naveen (UIUC)<br />
<strong>Chair:</strong> Fei Xue (UIUC)<br />
<strong>Time:</strong> June 6<sup>th</sup>, 8:30am – 10:00am<br />
<strong>Location: </strong>VEC 902/903</p>
<p><strong>Speech 1: Fly-By-Night Life Insurance and the NPMLE for Weibull Frailty Models</strong><br />
<strong>Speaker:  Roger Koenker (UIUC and UCL) </strong><br />
<strong>Abstract: </strong>Classical parametric statistical models are often quite fragile when confronted with data that fails to conform precisely to their idealized conditions.  In survival analysis more flexible models have been constructed as mixtures, yielding estimators with improved robustness properties.  These frailty models can themselves be parametric, but recent work has stressed non-parametric mixtures.  This talk will describe an application of the latter approach to the study of mortality rates for a large sample of mediterranean fruit flies.  In addition to revealing some rather surprising biological findings, the methods illustrate a general approach to estimation based on the Kiefer Wolfowitz non-parametric MLE for mixture models  and its use in a somewhat fanciful compound decision problem.</p>
<p><strong>Speech 2: Learning from Dr. Martin Luther King Jr: Text analysis and statistical approaches for civil rights</strong><br />
<strong>Speaker: Christopher Kinson (UIUC ) </strong><br />
<strong>Abstract: </strong>The recent shifts in the political landscape have heightened the importance of education and advancement for minorities, especially those underrepresented in STEM disciplines.  Minority students have been battling for equal access and rights to education at institutions of higher learning and the resources therein throughout the history of the US.  We as a scientific community must boldly step up to improve the outcomes of underrepresented students. The objective of this project is to educate and recruit Black and African students into the fields of statistics and data science. We create this research cohort and lab environment for the students to contribute to their own knowledge and to the body of knowledge within data science and the digital humanities. This lab is a creative and interdisciplinary space where the subject is one of the most important figures in Black history, Rev. Dr. Martin Luther King Jr. Specifically, we employ text analysis of King’s writings and speeches as well as articles written about him. Additionally, we educate students about statistics and introduce them to statistical programming in R.  The project tackles several challenges in broadening the participation of underrepresented students in STEM and celebrates the achievements made in that process.</p>
<p><strong>Speech 3: Inference on the dependence structure of time series extremes</strong><br />
<strong>Speaker: Stanislav Volgushev (U Toronto) </strong><br />
<strong>Abstract: </strong>Many natural phenomena such as extreme precipitation or heat waves can be described as maxima over blocks of time series. If the dependence structure of such extremes is of interest, component-wise maxima of vector-valued time series need to be considered. Under suitable assumptions on a vector-valued time series, properly standardized component-wise maxima are known to converge to a multivariate extreme-value distribution. In the present talk, we discuss functional central limit theorems and resulting inference procedures for the dependence structure of this limiting distribution. We propose several improvements over existing approaches which allow to reduce both bias and variance. We also contrast our approach (which is based on taking block maxima) with the peak-over-threshold approach which is a popular tool in the analysis of extremes.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#3b3d6a5r58" href='index.html%3Fp=583.html' onclick='event.preventDefault()'>
		Session 37: Recent advances in spectral methods for complex data	</a>
	</div>
<div id="3b3d6a5r58" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Recent advances in spectral methods for complex data<br />
<strong>Organizer</strong>: Yuekai Sun (UMich)<br />
<strong>Chair:</strong> Edgar Dobriban (Upenn)<br />
<strong>Time:</strong> June 6<sup>th</sup>, 8:30am – 10:00am<br />
<strong>Location: </strong>VEC 1202/1203</p>
<p><strong>Speech 1: Analyzing Developmental Processes with Optimal Transport </strong><br />
<strong>Speaker: Geoff Schiebinger (MIT)</strong><br />
<strong>Abstract: </strong>Understanding the molecular programs that guide cellular differentiation during development is a major goal of modern biology. Here, we introduce an approach, based on the mathematics of optimal transport, for inferring developmental landscapes, probabilistic cellular fates and dynamic trajectories from large-scale single-cell RNA-seq (scRNA-seq) data collected along a time course. Our approach, Waddington-OT is based on a novel framework for analyzing stochastic processes whose instantaneous temporal couplings agree with optimal transport. We demonstrate the power of WADDINGTON-OT by applying the approach to study 65,781 scRNA-seq profiles collected at 10 time points over 16 days during reprogramming of fibroblasts to iPSCs.</p>
<p>We construct a high-resolution map of reprogramming that rediscovers known features; uncovers new alternative cell fates including neural- and placental-like cells; predicts the origin and fate of any cell class; highlights senescent-like cells that may support reprogramming through paracrine signaling; and implicates regulatory models in particular trajectories. Of these findings, we highlight Obox6, which we experimentally show enhances reprogramming efficiency. Our approach provides a general framework for investigating cellular differentiation.</p>
<p><strong>Speech 2: How to select the number of components in PCA and factor analysis? Understanding and improving permutation methods</strong><br />
<strong>Speaker: Edgar Dobriban (Wharton)</strong><br />
<strong>Abstract: </strong>Selecting the number of components in PCA and factor analysis is a key problem facing practitioners of data science. One of the most popular methods is a permutation approach that randomly scrambles the elements of each feature. It selects the components whose singular values are large compared to the permuted data. This method (also known as parallel analysis) is recommended in many textbooks and review papers, and used in genomics by leading applied statisticians including T Hastie, M Stephens, J Storey, R Tibshirani and WH Wong. However, it is poorly understood. In this talk, we develop a theoretical understanding and propose improvements.</p>
<p><strong>Speech 3: Higher-order spectral graph clustering with motifs</strong><br />
<strong>Speaker: Austin Benson (Cornell)</strong><br />
<strong>Abstract: </strong>Networks are typically described by lower-order connectivity patterns that are captured at the level of individual nodes and edges. However, higher-order connectivity patterns captured by small subgraphs, or network motifs, describe the fundamental structures that control and mediate the behavior of many complex systems. In this talk, I will discuss a higher-order spectral graph clustering framework that finds groups of nodes that participate in many instances of a given motif. I will also show applications of this framework in ecology, biology, transportation, and social networks.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#efcaf1et4m" href='index.html%3Fp=585.html' onclick='event.preventDefault()'>
		Session 38: New machine learning methods	</a>
	</div>
<div id="efcaf1et4m" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: New machine learning methods<br />
<strong>Organizer</strong>: Annie Qu (UIUC)<br />
<strong>Chair:</strong> Annie Qu (UIUC)<br />
<strong>Time:</strong> June 6<sup>th</sup>, 8:30am – 10:00am<br />
<strong>Location: </strong>VEC 1302</p>
<p><strong>Speech 1: Generalized self-concordant optimization and its applications in statistical learning</strong><br />
<strong>Speaker: Quoc Tran-Dinh (UNC)</strong><br />
<strong>Abstract: </strong>Many statistics and machine learning applications can be cast into a composite convex minimization problem. Well-known examples include sparse logistic<br />
regression, SVM, and inverse covariance estimation. These problems are well studied and can efficiently be solved by several state-of-the-arts. Recent development in first-order, second-order, and stochastic gradient-type methods has brought a new opportunity to solve many other classes of convex optimization problems in large-scale settings. Unfortunately, so far, such methods require the underlying models to satisfy some structural assumptions such as Lipschitz gradient and restricted strong convexity, which may be failed to hold or may be hard to<br />
check.<br />
In this talk, we demonstrate how to exploit an analytical structure hidden in convex optimization for developing solution methods. Our key idea is to generalize a<br />
powerful concept so-called “self-concordance” introduced by Y. Nesterov and A. Nemirovskii to a broader class of convex functions. We show that this structure<br />
covers many applications in statistics and machine learning. Then, we develop a unified theory for designing numerical methods. We illustrate our theory through<br />
Newton-type and proximal Newton-type methods. We note that the proposed theory can further be applied to develop other methods as long as the underlying<br />
model is involved with a “generalized self-concordant structure”. We provide some numerical examples in different fields to illustrate our theoretical development.<br />
<strong><br />
Speech 2: On Scalable Inference with Stochastic Gradient Descent<br />
Speaker: Yixin Fang (New Jersey Institute of Technology)</strong><br />
<strong>Abstract: </strong>In many applications involving large dataset or online updating, stochastic gradient descent (SGD) provides a scalable way to compute parameter estimates and has gained increasing popularity due to its numerical convenience and memory efficiency. While the asymptotic properties of SGD-based estimators have been established decades ago, statistical inference such as interval estimation remains much unexplored.  The traditional resampling method such as the bootstrap is not computationally feasible since it requires to repeatedly draw independent samples from the entire dataset.  The plug-in method is not applicable when there are no explicit formulas for the covariance matrix of the estimator. In this paper, we propose a scalable inferential procedure for stochastic gradient descent, which, upon the arrival of each observation, updates the SGD estimate as well as a large number of randomly perturbed SGD estimates. The proposed method is easy to implement in practice.  We establish its theoretical properties for a general class of models that includes generalized linear models and quantile regression models as special cases.  The finite-sample performance and numerical utility is evaluated by simulation studies and two real data applications.</p>
<p><strong>Speech 3: Scalable Kernel-based Variable Selection with Sparsistency</strong><br />
<strong>Speaker: </strong>Junhui Wang (City U. of Hong Kong)<br />
Abstract: Variable selection is central to sparse modeling, and many methods have been proposed under various model assumptions. In this talk, we will present a scalable framework for model-free variable selection in reproducing kernel<br />
Hilbert space (RKHS) without specifying any restrictive model. As opposed to most existing model-free variable selection methods requiring fixed dimension, the proposed method allows dimension p to diverge with sample size n. The proposed method is motivated from the classical hard-threshold variable selection for linear models, but allows for general variable effects. It does not require specification of the underlying model for the response, which is appealing in sparse modeling with a large number of variables. The  proposed method can also be adapted to various scenarios with specific model assumptions, including linear models, quadratic models, as well as additive models. The asymptotic estimation and variable selection consistencies of the proposed method are established in all the scenarios. If time<br />
permits, the extension of the proposed method beyond mean regression will also be discussed.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#9abbeb64g9" href='index.html%3Fp=587.html' onclick='event.preventDefault()'>
		Session 39: New directions in functional data analysis.	</a>
	</div>
<div id="9abbeb64g9" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: New directions in functional data analysis.<br />
<strong>Organizer</strong>: Tailen Hsing ( UMich)<br />
<strong>Chair:</strong>  Vincent Joseph Dorie (Columbia)<br />
<strong>Time:</strong> June 6<sup>th</sup>, 8:30am – 10:00am<br />
<strong>Location: </strong>VEC 1402</p>
<p><strong>Speech 1: Nonparametric covariance estimation for mixed longitudinal studies</strong><br />
<strong>Speaker: Kehui Chen (U of Pitt)</strong><br />
<strong>Abstract: </strong>Motivated by applications of mixed longitudinal studies, where a group of subjects entering the study at different ages (cross-sectional) are followed for successive years (longitudinal), we consider nonparametric covariance estimation with samples of noisy and partially observed functional trajectories. In this talk, we will introduce a novel sequential aggregation scheme, which works for both dense regular and sparse irregular observations. We will present numerical experiment results and applications a midlife women’s working memory study. We will also discuss the details of identifiability and estimation consistency.</p>
<p><strong>Speech 2: Functional Data Analysis with Highly Irregular Designs with Applications to Head Circumference Growth</strong><br />
<strong>Speaker: Matthew Reimherr (Penn State)</strong><br />
<strong>Abstract: </strong>Functional Data Analysis often falls into one of two branches, either sparse or dense, depending on the sampling frequency of the underlying curves.  However, methods for sparse FDA often still rely on having a growing number of observations per subject as the sample size grows.  Practically, this means that for very large sample sizes with infrequently or irregularly sampled curves, common methods may still suffer a non-negligible bias.  This becomes especially true for nonlinear models, which are often defined based on complete curves.  In this talk I will discuss how this issue can be fixed to obtain valid statistical inference regardless of the sampling frequency of the curves.  This work is motivated by a study by Dr. Carrie Daymont from Hershey medical school that examines pathologies related to head circumference growth in children.  In her study, tens of thousands of children are sampled, but with widely varying frequency.</p>
<p><strong>Speech 3:  Supervised Learning on the Path Space and its Applications</strong><br />
<strong>Speaker: Hao Ni (UCL)</strong><br />
<strong>Abstract: </strong>Regression analysis aims to use observational data from multiple observations to develop a functional relationship relating explanatory variables to response variables, which is important for much of modern statistics, and econometrics, and also the field of machine learning. In this talk, we consider the special case where the explanatory variable is a data stream. We provide an approach based on identifying carefully chosen features of the stream which allows linear regression to be used to characterise the functional relationship between explanatory variables and the conditional distribution of the response; the methods used to develop and justify this approach, such as the signature of a stream and the shuffle product of tensors, are standard tools in the theory of rough paths and provide a unified and non-parametric approach with potential significant dimension reduction. To further improve the efficiency of the signature method, we can combine the non-linear regression method (e.g. neural network) with the signature feature set. Numerical examples are provided to show the superior performance of the proposed method. Lastly I will show that the signature based method have achieved the state-of-the-art results in online handwritten text recognition and action recognition.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#cd45b531rd" href='index.html%3Fp=589.html' onclick='event.preventDefault()'>
		Session 40: Modern Approaches for Inference and Estimation	</a>
	</div>
<div id="cd45b531rd" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Modern Approaches for Inference and Estimation<br />
<strong>Organizer</strong>: Genevera Allen (Rice)<br />
<strong>Chair:</strong> Genevera Allen (Rice)<br />
<strong>Time:</strong> June 6<sup>th</sup>, 1:15pm &#8211; 2:45pm<br />
<strong>Location: </strong>VEC 404/405</p>
<p><strong>Speech 1: High-Dimensional Propensity Score Estimation via Covariate Balancing</strong><br />
<strong>Speaker: Yang Ning (Cornell)<br />
Abstract: </strong>In this paper, we address the problem of estimating the average treatment effect (ATE) and the average treatment effect for the treated (ATT) in observational studies when the number of potential confounders is possibly much greater than the sample size. In particular, we develop a robust method to estimate the propensity score via covariate balancing in high-dimensional settings. Since it is usually impossible to obtain the exact covariate balance in high dimension, we propose to estimate the propensity score by balancing a carefully selected subset of covariates that are predictive of the outcome under the assumption that the outcome model is linear and sparse. The estimated propensity score is, then, used for the Horvitz-Thompson estimator to infer the ATE and ATT. We prove that the proposed methodology has the desired properties such as sample boundedness, root-$n$ consistency, asymptotic normality, and semiparametric efficiency. We then extend these results to the case where the outcome model is a sparse generalized linear model. More importantly, we show that the proposed estimator is robust to model misspecification. Finally, we conduct simulation studies to evaluate the finite-sample performance of the proposed methodology, and apply it to estimate the causal effects of college attendance on adulthood political participation. Open-source software is available for implementing the proposed methodology. This is the joint work with Peng and Imai.<br />
<strong><br />
Speech 2: Interactive algorithms for graphical model selection<br />
Speaker: Gautam Dasarthy (Rice University)<br />
Abstract: </strong>With rapid progress in our ability to acquire, process, and learn from data, the true democratization of data-driven intelligence has never seemed closer. Unfortunately, there is a catch. Machine learning algorithms have traditionally been designed independently of the systems that acquire data. As a result, there is a fundamental disconnect between their promise and their real-world applicability. An urgent need has therefore emerged for integrating the design of learning and acquisition systems.<br />
In this talk, I will present an approach for addressing this learning-acquisition disconnect using interactive machine learning methods. In particular, I will consider the problem of learning graphical model structure in high dimensions. This will highlight how traditional (open loop) methods do not take into account data acquisition constraints that arise in applications ranging from sensor networks to calcium imaging of the brain. I will then demonstrate how one can close this loop using techniques from interactive machine learning. I will conclude by discussing several connections to post-selection inference in this context.</p>
<p><strong>Speech 3: AdaPT: An interactive procedure for multiple testing with side information<br />
Speaker: Will Fithian (UCB)<br />
Abstract: </strong>We consider the problem of multiple hypothesis testing with generic side information: for each hypothesis we observe both a p-value and some predictor encoding contextual information about the hypothesis. For large-scale problems, adaptively focusing power on the more promising hypotheses (those more likely to yield discoveries) can lead to much more powerful multiple testing procedures. We propose a general iterative framework for this problem, called the Adaptive p-value Thresholding (AdaPT) procedure, which adaptively estimates a Bayes-optimal p-value rejection threshold and controls the false discovery rate (FDR) in finite samples. At each iteration of the procedure, the analyst proposes a rejection threshold and observes partially censored p-values, estimates the false discovery proportion (FDP) below the threshold, and either stops to reject or proposes another threshold, until the estimated FDP is below α. Our procedure is adaptive in an unusually strong sense, permitting the analyst to use any statistical or machine learning method she chooses to estimate the optimal threshold, and to switch between different models at each iteration as information accrues.<br />
This is joint work with Lihua Lei.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#1658742u04" href='index.html%3Fp=591.html' onclick='event.preventDefault()'>
		Session 41: Functional and high dimensional data	</a>
	</div>
<div id="1658742u04" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Functional and high dimensional data<br />
<strong>Organizer</strong>: Aurore Delaigle (U of Melbourne)<br />
<strong>Chair:</strong> Aurore Delaigle (U of Melbourne)<br />
<strong>Time:</strong> June 6<sup>th</sup>, 1:15pm &#8211; 2:45pm<br />
<strong>Location: </strong>VEC 1403</p>
<p><strong>Speech 1: Small Sample Confidence Intervals for the ACL (Abduskhurov, Cheng, and Lin) Estimators Under the Proportional Hazards Model</strong><br />
<strong>Speaker: Emad Abdurasul (James Madison University)</strong><br />
<strong>Abstract: </strong>We develop a saddlepoint- based method for generating small sample confidence bands for the population survival function from the ACL estimators, under the proportional hazards model. In the process, we derive the exact distribution of this estimator and develop mid-population tolerance bands for saddlepoint estimators. Our method depends upon the Mellin transform of the zero-truncated survival estimator. This transform is inverted via saddlepoint approximation to yield a highly accurate approximation to the cumulative distribution function of the respective cumulative hazard function. This distribution function is then inverted to produce our saddlepoint onfidence bands. Then we compare our saddlepoint confidence bands with those obtained from competing large sample methods as well as with those obtained from the exact distribution. In our simulation study, we found that the saddlepoint confidence bands are very close to the confidence bands derived from the exact distribution. In addition being close, it is easier to compute, and it outperforms the large sample methods in terms probability convergence.<br />
<strong>Speech 2: Binary functional linear models in a stratified sampling setting</strong><br />
<strong>Speaker: </strong>Sophie Dabo-Niang (Université Lille 3)<br />
Abstract: A functional binary choice model is explored in a stratified sample design context. In other words, a model is considered in which the response is binary, the explanatory variable is functional, and the sample is stratified with respect to the values of the response variable. A dimension reduction of the space of the explanatory random function based on a Karhunen– Loève expansion is used to define a conditional maximum likelihood estimate of the model. Based on this formulation, several asymptotic properties are given. Numerical studies are used to compare the proposed method with the ordinary maximum likelihood method, which ignores<br />
the nature of the sampling. The proposed model yields encouraging results. The potential of the functional sampling model for integrating special non-random features of the sample, which would have been difficult to see otherwise, is also outlined.</p>
<p><strong>Speech 3: Functional CLT and sharp bounds for some (conditional Poisson) survey sampling plans with applications to big (tall) data</strong><br />
<strong>Speaker: Patrice Bertail (Université Paris Nanterre)</strong><br />
<strong>Abstract: </strong>Subsampling methods as well as general sampling methods appear as natural tools to handle very large database (big data in the indivual dimension) when traditionnal statistical methods or statistical learning algorithms fail to be implemented on too large datasets. The choice of the weights of the survey sampling scheme may reduce the loss implied by the choice of a much more smaller sampling size (according to the problem of interest). We will first recall some asymptotic results for general survey sampling based empirical processes indexed by class of functions (see Bertail and Clémençon, 2017, Scandinavian Journal of Statistics), for Poisson type and conditional Poisson (rejective) survey samplings. These results may be extended to a large class of survey sampling plans via the notion of negative association of most survey sampling plans (Bertail, Rebecq, 2018)  Then in the perspective to generalize some statistical learning tasks to sampled data, we will obtain  exponential bounds for the probabilities of deviation of a Horvitz Thompson sum from its expectation when the variables involved in the summation are obtained by sampling in a finite population according to associated or rejective scheme.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#472d8c65ck" href='index.html%3Fp=593.html' onclick='event.preventDefault()'>
		Session 42: Machine learning, classification and designs	</a>
	</div>
<div id="472d8c65ck" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Machine learning, classification and designs<br />
<strong>Organizer</strong>: Annie Qu (UIUC)<br />
<strong>Chair:</strong> Annie Qu (UIUC)<br />
<strong>Time:</strong> June 6<sup>th</sup>, 1:15pm &#8211; 2:45pm<br />
<strong>Location: </strong>VEC 1202 /1203</p>
<p><strong>Speech 1: Efficient Gaussian Process Modeling using Experimental Design-based Subagging</strong><br />
<strong>Speaker: Ying Hung (Rutgers)</strong><br />
<strong>Abstract: </strong>We address two important issues in Gaussian process (GP) modeling. One is how to reduce the computational complexity in GP modeling and the other is how to simultaneous perform variable selection and estimation for the mean function of GP models. Estimation is computationally intensive for GP models because it heavily involves manipulations of an n-by-n correlation matrix, where n is the sample size. Conventional penalized likelihood approaches are widely used for variable selection. However the computational cost of the penalized likelihood estimation (PMLE) or the corresponding one-step sparse estimation (OSE) can be prohibitively high as the sample size becomes large, especially for GP models. To address both issues, we propose an efficient subsample aggregating (subagging) approach with an experimental design-based subsampling scheme. The proposed method is computationally cheaper, yet it can be shown that the resulting subagging estimators achieve the same efficiency as the original PMLE and OSE asymptotically. The finite-sample performance is examined through simulation studies. Application of the proposed methodology to a data center thermal study reveals some interesting information, including identifying an efficient cooling mechanism.</p>
<p><strong>Speech 2: Structural Learning and Integrative Decomposition of Multi-View Data</strong><br />
<strong>Speaker: Irina Gaynanova (Texas A&amp;M)</strong><br />
<strong>Abstract: </strong>The increased availability of the multi-view data (data on the same samples from multiple sources) has led to strong interest in models based on low-rank matrix factorizations. These models represent each data view via shared and individual components, and have been successfully applied for exploratory dimension reduction, association analysis between the views, and further learning tasks such as consensus clustering. Despite these advances, there remain significant challenges in modeling partially-shared components, and identifying the number of components of each type (shared/partially-shared/individual). In this work, we formulate a novel linked component model that directly incorporates partially-shared structures. We call this model SLIDE for Structural Learning and Integrative DEcomposition of multi-view data. We prove the existence of SLIDE decomposition and explicitly characterize the identifiability conditions. The proposed model fitting and selection techniques allow for joint identification of the number of components of each type, in contrast to existing sequential approaches. In our empirical studies, SLIDE demonstrates excellent performance in both signal estimation and component selection. We further illustrate the methodology on the breast cancer data from The Cancer Genome Atlas repository. This is joint work with Gen Li.</p>
<p><strong>Speech 3: Shrinking characteristics of precision matrix estimators</strong><br />
<strong>Speaker: Adam Rothman (U. of Minnesota)</strong><br />
<strong>Abstract: </strong>We propose a framework to shrink a user-specified characteristic of a precision matrix estimator that is needed to fit a predictive model. Estimators in our framework minimize the Gaussian negative log-likelihood plus an L1 penalty on a linear function evaluated at the optimization variable corresponding to the precision matrix. We establish convergence rate bounds for these estimators and we propose an alternating direction method of multipliers algorithm for their computation. Our simulation studies show that our estimators can perform better than competitors when they are used to fit predictive models. In particular, we illustrate cases where our precision matrix estimators perform worse at estimating the population precision matrix while performing better at prediction.  This is joint work with Aaron Molstad.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#9c4d14e1wy" href='index.html%3Fp=595.html' onclick='event.preventDefault()'>
		Session 43: Statistics in neuroscience and microbiome research at the Flatiron Institute	</a>
	</div>
<div id="9c4d14e1wy" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Statistics in neuroscience and microbiome research at the Flatiron Institute<br />
<strong>Organizer</strong>: Christian L. Müller (Flatiron Institute, Simons Foundation)<br />
<strong>Chair:</strong> Christian L. Müller (Flatiron Institute, Simons Foundation)<br />
<strong>Time:</strong> June 6<sup>th</sup>, 1:15pm &#8211; 2:45pm<br />
<strong>Location: </strong>VEC 1303</p>
<p><strong>Speech 1: Neural representation learning as kernel alignment</strong><br />
<strong>Speaker:  Cengiz Pehlevan (Simons Foundation)</strong><br />
<strong>Abstract: </strong>What are the brain’s learning cost functions? I show that some kernel alignment cost functions can be minimized by biologically plausible neural learning algorithms. Starting from such cost functions, I derive neural networks for various biologically motivated unsupervised learning tasks, such as soft-clustering and manifold disentangling. I discuss applications of these ideas to circuits of the brain.</p>
<p><strong>Speech 2: Robust regression with compositional covariates</strong><br />
<strong>Speaker: Aditya Mishra (Flatiron Institute)</strong><br />
<strong>Abstract: </strong>With the large-scale efforts in 16S ribosomal RNA sequencing in microbiome study related to human gut or marine ecosystem, we have relative abundance/compositional data of the group of microbial taxa at different taxonomic levels. A problem of interest is to model phenotype/response using these compositional covariates. Often we have observed that there is presence of either outlier or leveraged observation in data. Hence, we propose a robust regression model with compositional covariates. Subcompositional coherence of the model estimates are satisfied via linear constraint to the linear logcontrast model. In order for model to be robust, we add a mean shift parameter to each n instance of the data. The estimation is performed via penalized regression approach with regularization enforcing sparsity in mean shift parameter. We have investigated the model with both convex (l1 and adaptive l1) and non-convex (l0) penalty on mean shift. For the purpose of initialization in latter case and weight constriction in adaptive l1 penalty case, we propose an algorithm extending the idea of S-estimation in case of linear model with compositional covariates. Our approach has only one tuning parameter which is selected via a modified BIC selection criterion. Our theoretical analysis focus on non-asymptotic prediction error bound revealing interesting finitesample behaviors of the estimators. We have demonstrated the efficacy of the approach using various simulation studies and an application relating body mass index to human gut microbiome data.</p>
<p><strong>Speech 3: Online deconvolution and demixing of calcium imaging data in real time</strong><br />
<strong>Speaker: Eftychios Pnevmatikakis (Simons Foundation)</strong><br />
<strong>Abstract: </strong>Optical imaging methods using calcium indicators enable monitoring the activity of large neuronal populations in vivo. Imaging experiments typically generate a large amount of data that needs to be processed to extract the activity of the imaged neuronal sources. While deriving such processing algorithms is an active area of research, most existing methods require the processing of large amounts of data rendering them vulnerable to the volume of the recorded data, and preventing real-time experimental interrogation. Here we introduce CaImAn, an open source suite of tools for the online analysis of calcium imaging data, including i) motion artifact correction, ii) neuronal source extraction, and iii) activity denoising and deconvolution. Our approach combines and extends previous work on online dictionary learning and calcium imaging data analysis, to deliver an automated pipeline that can discover and track the activity of hundreds of cells in real time. We benchmark the performance of our algorithm on manually annotated data, and show that it outperforms popular offline approaches.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#13f84152ob" href='index.html%3Fp=597.html' onclick='event.preventDefault()'>
		Session 44: Recent Avances in Statistical Network, Functional and High-dimensional Data Analysis	</a>
	</div>
<div id="13f84152ob" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Recent Avances in Statistical Network, Functional and High-dimensional Data Analysis<br />
<strong>Organizer</strong>: Ji Zhu (Umich)<br />
<strong>Chair:</strong> Yujia Deng (UIUC)<br />
<strong>Time:</strong> June 6<sup>th</sup>, 1:15pm &#8211; 2:45pm<br />
<strong>Location: </strong>VEC 1402</p>
<p><strong>Speech 1: Factor Augmented Vector Autoregressive Models under High Dimensional Scaling</strong><br />
<strong>Speaker: George Michailidis (U of Florida)</strong><br />
<strong>Abstract: </strong>Vector Autoregressive Models (VAR) are widely used in applied economics and finance. In this talk, we consider a VAR model augmented with dynamically evolving factors. The time series modeled as a VAR, together with the dynamic factors relate to a large number of other time series hat aid in the identifiability of the model parameters. We investigate the identifiability of such models, as well as estimation and inference issues under high-dimensional scaling. The performance of the proposed methods is assessed through synthetic data and the methodology is illustrated on a economic data set.</p>
<p><strong>Speech 2: Model-assisted design of experiments on networks and social media platforms</strong><br />
<strong>Speaker: Edoardo Airoldi (Harvard)</strong><br />
<strong>Abstract: </strong>Classical approaches to causal inference largely rely on the assumption of “lack of interference”, according to which the outcome of an individual does not depend on the treatment assigned to others, as well as on many other simplifying assumptions, including the absence of strategic behavior. In many applications, however, such as evaluating the effectiveness of healthcare interventions that leverage social structure, assessing the impact of product innovations and ad campaigns on social media platforms, or experimentation at scale in large IT companies, assuming lack of interference and other simplifying assumptions is untenable. Moreover, the effect of interference itself is often an inferential target of interest, rather than a nuisance. In this talk, we will formalize technical issues that arise in estimating causal effects when interference can be attributed to a network among the units of analysis, within the potential outcomes framework. We will introduce and discuss several strategies for experimental design in this context centered around a judicious use statistical models, which we refer to as “model-assisted” design of experiments. In particular, we wish for certain finite-sample properties of the estimator to hold even if the model catastrophically fails, while we would like to gain efficiency if certain aspects of the model are correct. We will then contrast design-based, model-based and model-assisted approaches to experimental design from a decision theoretic perspective.</p>
<p><strong>Speech 3: Correcting Selection Bias via Functional Empirical Bayes</strong><br />
<strong>Speaker: Gareth James (USC)</strong><br />
<strong>Abstract: </strong>Selection bias results from the selection of extreme observations and is a well recognized issue for standard scalar or multivariate data. Numerous approaches have been proposed to address the issue, dating back at least as far as the James-Stein shrinkage estimator. However, the same potential issue arises, albeit with additional complications, for functional data. Given a set of observed functions, one may wish to select for further analysis those which are most extreme according to some metric such as the average, maximum, or minimum value of the function. However, given the functions are often noisy realizations of some underlying mean process, these outliers are likely to generate biased estimates of the quantity of interest. In this talk I propose an Empirical Bayes approach, using Tweedie’s formula, to adjust such functional data to generate approximately unbiased estimates of the true mean functions. The approach has several advantages. It is non-parametric in nature, but is capable of automatically shrinking back towards a James-Stein type estimator in low signal situations. It is also computationally efficient and possesses desirable theoretical properties. Furthermore, I demonstrate through extensive simulations that the approach can produce significant improvements in prediction accuracy relative to possible competitors. It is joint work with Joshua Derenski and Yingying Fan.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#e20bbf86qx" href='index.html%3Fp=599.html' onclick='event.preventDefault()'>
		Session 45: New insights into classical statistical methods	</a>
	</div>
<div id="e20bbf86qx" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: New insights into classical statistical methods<br />
<strong>Organizer</strong>: Qing Mai (Florida State U)<br />
<strong>Chair:</strong> Qing Mai (Florida State U)<br />
<strong>Time:</strong> June 6<sup>th</sup>, 3:15pm &#8211; 4:45pm<br />
<strong>Location: </strong>VEC 404/405</p>
<p><strong>Speech 1: Rank-constrained inherent clustering paradigm for supervised and unsupervised learning</strong><br />
<strong>Speaker: Yiyuan She (Florida State U)</strong><br />
<strong>Abstract: </strong>Modern clustering applications are often faced with challenges from high dimensionality and/or nonconvex clusters. This paper gives a mathematical formulation of clustering with concurrent dimension reduction and proposes an optimization-based inherent clustering framework. Inherent clustering enjoys a kernel property to work on similarity matrices and can be extended to supervised learning. A simple-to-implement iterative algorithm is developed by use of linearization and block coordinate descent. Non-asymptotic analysis shows the tight error rate of inherent clustering in the supervised setting. Extensive simulations, as well as real-data experiments in network community detection and learning, demonstrate the excellent performance of the proposed approach.</p>
<p><strong>Speech2:  Fast and Optimal Bayesian Inference via Variational Approximations<br />
Speaker: Yun Yang (Florida State U)</strong><br />
<strong>Abstract: </strong>We propose a variational approximation to Bayesian posterior distributions, called $\alpha$-VB, with provable statistical guarantees for models with and without latent variables. The standard variational approximation is a special case of $\alpha$-VB with $\alpha=1$. When $\alpha \in(0,1)$, a novel class of variational inequalities are developed for linking the Bayes risk under the variationalapproximation to the objective function in the variational optimization problem, implying that maximizing the evidence lower bound in variational inference has the effect of minimizing the Bayes risk within the variational density family. Operating in a frequentist setup, the variational inequalities imply that point estimates constructed from the $\alpha$-VB procedure converge at an optimal rate to the true parameter in a wide range of problems. We illustrate our general theory with a number of examples, including the mean-field variational approximation to (low)-high-dimensional Bayesian linear regression with spike and slab priors, mixture of Gaussian models, latent Dirichlet allocation, and (mixture of) Gaussian variational approximation in regular parametric models.<br />
<strong><br />
</strong><br />
<strong>Speech 3: An Iterative Penalized Least Squares Approach to</strong><strong> </strong><strong>Sparse Canonical Correlation Analysis</strong><br />
<strong>Speaker: Xin Zhang (Florida State U)</strong><br />
<strong>Abstract: </strong>It is increasingly interesting to model the relationship between two sets of measurements when both of them are high-dimensional. Canonical correlation analysis (CCA) is a classical tool that explores the dependency of two multivariate random variables and extracts canonical pairs of highly correlated linear combinations. Driven by applications in genomics, text mining and imaging research among others, many recent studies generalize CCA to high-dimensional settings. However, most of them either rely on strong assumptions on the covariance matrices, or does not produce nested solutions. We propose a new sparse CCA (SCCA) method that recasts high-dimensional CCA as an iterative penalized least squares problem. Thanks to the new penalized least squares formulation, our SCCA method directly penalizes and estimates the sparse CCA directions with efficient algorithms. Therefore, in contrast to some existing methods, the new SCCA does not impose any sparsity assumptions on the covariance matrices. The proposed SCCA is also very flexible in the sense that it can be easily combined with properly chosen penalty functions to perform structured variable selection or to incorporate prior information. Moreover, our proposal of SCCA produces nested solutions, which provides great convenient in practice. Theoretical results show that SCCA can consistently estimate the true canonical pairs with an overwhelming probability in ultra-high dimensions. Numerical results also demonstrate the competitive performance of SCCA.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#d081e21wex" href='index.html%3Fp=601.html' onclick='event.preventDefault()'>
		Session 46: New developments for large complex data	</a>
	</div>
<div id="d081e21wex" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: New developments for large complex data<br />
<strong>Organizer</strong>: Annie Qu (UIUC)<br />
<strong>Chair:</strong> Annie Qu (UIUC)<br />
<strong>Time:</strong> June 6<sup>th</sup>, 3:15pm &#8211; 4:45pm<br />
<strong>Location: </strong>VEC 902<br />
<strong><br />
Speech 1: Point and Interval Estimations for Individualized MCID</strong><br />
<strong>Speaker: Jiwei Zhao (SUNY, Buffalo)</strong><br />
<strong>Abstract: </strong>The minimal clinically important difference (MCID) is the smallest change in a treatment outcome that an individual patient would identify as important. In the era of precision medicine, it is of particular interest to study both point and interval estimations for the individualized MCID. The motivating example of this work is the ChAMP trial, which is a randomized controlled trial to compare debridement to observation of chondral lesions encountered during partial meniscectomy. In this trial, the primary outcome is the patient reported pain score one year after the surgery and we are interested in estimating the individualized MCID so that the treatment effect can be further studied. In this paper, we formulize this problem in a classification setting where nonconvex minimization technique is needed for the optimization. Furthermore, we develop the Bahadur representation of the individualized MCID so that its confidence interval can be derived. The proposed method is illustrated via comprehensive simulation studies. We also apply our proposed methodology to the ChAMP trial analysis.</p>
<p><strong>Speech 2: </strong><strong>Robust Probabilistic Classification for Irregularly Sampled Functional Data<br />
Speaker: Doug Simpson (UIUC)</strong><br />
<strong>Abstract:</strong>Motivated by research on diagnostic ultrasound to evaluate tissue regions of interest, we present a robust probabilistic classifier for functional data that predicts the membership for given input and provides reliable probability estimates for class memberships. This method combines Bayes classifier and semi-parametric mixed effects model with robust tuning parameter. We aim to make the method robust to outlying curves especially in providing a robust estimate of certainty in prediction, which is crucial in medical diagnosis. This approach is applicable to various structures, such as samples observed over varying intervals or repeatedly measured curves retaining between-curve correlation, with no parametric assumption on within curve covariance. We conduct simulation studies to investigate the operating characteristics of the probability estimates in the presence of ideal data and data with outlying curves and compare with other functional classification procedures. We illustrate the methodology in classification of quantitative ultrasound data and other applications.</p>
<p><strong>Speech 3: A new method for constructing gene co-expression networks based on samples with tumor purity heterogeneity</strong><br />
<strong>Speaker: Francesca Petralia (Mount Sinai)</strong><br />
<strong>Abstract: </strong>Tumor tissue samples often contain an unknown fraction of stromal cells. This problem well known as tumor purity heterogeneity (TPH) was recently recognized as a severe issue in omics studies. Specifically, if TPH is ignored when inferring co-expression networks, edges are likely to be estimated among genes with mean shift between non-tumor and tumor cells rather than among gene pairs interacting with each other in tumor cells. To address this issue, we propose TSNet a new method which constructs tumor-cell specific gene/protein co-expression networks based on gene/protein expression profiles of tumor tissues. TSNet treats the observed expression profile as a mixture of expressions from different cell types and explicitly models tumor purity percentage in each tumor sample. Using extensive synthetic data experiments, we demonstrate that TSNet outperforms a standard graphical model not accounting for tumor-purity heterogeneity. We then apply TSNet to estimate tumor specific gene co-expression networks based on TCGA ovarian cancer RNAseq data. We identify novel co-expression modules and hub structure specific to tumor cells.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#04d0f5cnuf" href='index.html%3Fp=603.html' onclick='event.preventDefault()'>
		Session 47: Statistical inference and complex data structures	</a>
	</div>
<div id="04d0f5cnuf" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Statistical inference and complex data structures<br />
<strong>Organizer</strong>: Eric Laber (NCST)<br />
<strong>Chair:</strong> Yubai Yuan (UIUC)<br />
<strong>Time:</strong> June 6<sup>th</sup>, 3:15pm &#8211; 4:45pm<br />
<strong>Location: </strong>VEC 1303<br />
<strong><br />
Speech 1: Inter-modal Coupling: A Class of Measurements for Studying Local Covariance Patterns Among Multiple Imaging Modalities</strong><br />
<strong>Speaker: Kristin Linn (UPenn)</strong><br />
<strong>Abstract:</strong>Local cortical coupling was recently introduced as a subject-specific measure for studying localized relationships between cortical thickness and sulcal depth. Although a promising first step towards understanding local covariance patterns that are present between these two specific neuro-anatomical measurements, local cortical coupling suffers from a limited scope of imaging modalities that can be analyzed within the framework. We generalize and improve this local coupling measure by proposing an analogue in volumetric space that can be used to produce subject-level feature images among an arbitrary number of volumetric imaging modalities. Our proposed class of measures, collectively referred to as inter-modal coupling (IMCo), is based on a locally weighted regression framework. In this work, we study IMCo between cerebral blood flow and gray matter density using a sample of youths ages 8-21 from the Philadelphia Neurolodevelopmental Cohort. We describe how these two modalities covary spatially throughout the brain find evidence of significant developmental effects in several notable regions.  We also give an overview of other applications where we are applying IMCo to study relationships between multiple types of images.</p>
<p><strong>Speech 2: Modeling Heterogeneity in Motor Learning using Heteroskedastic Functional Principal Components</strong><br />
<strong>Speaker: Jeff Goldsmith (Columbia University)</strong><br />
<strong>Abstract: </strong>We propose a novel method for estimating population-level and subject-specific effects of covariates on the variability of functional data. We extend the functional principal components analysis framework by modeling the variance of principal component scores as a function of covariates and subject-specific random effects. In a setting where principal components are largely invariant across subjects and covariate values, modeling the variance of these scores provides a flexible and interpretable way to explore factors that affect the variability of functional data. Our work is motivated by a novel dataset from an experiment assessing upper extremity motor control, and quantifies the reduction in motion variance associated with skill learning.</p>
<p><strong>Speech 3: Prior Adaptive Semi-supervised Learning with Application to Electronic Health Records Phenotyping</strong><br />
<strong>Speaker: Yichi Zhang (Harvard) </strong><br />
<strong>Abstract: </strong>Electronic Health Records (EHR) provides large and rich data sources for biomedical researches, and EHR data have been successfully used to gain novel insights into several diseases. However, the usage of EHR data remains quite limited,  because extracting precise phenotype for individual patient requires labor intensive medical chart review and such a manual process is not scalable. To facilitate an automatic procedure for accurate phenotyping, we formulate the problem in a high dimensional setting and propose a semi-supervised method that combine information from chart reviewed records with some data-driven prior knowledge derived from the entire dataset. The proposed estimator, Prior Adaptive Semi-supervised (PASS) estimator, enjoys nice theoretical properties including efficiency and robustness, and applies to a broad class of problems beyond EHR applications. The finite sample performance is evaluated via simulation studies and a real dataset on rheumatoid arthritis phenotyping. Further improvements involving word embedding and selective sampling are discussed.</p>
</div>	</div>
</div></div>
<div class="panel panel-default pt-cv-content-item pt-cv-1-col" >
<div class="panel-heading pt-cv-title">
    <a class="panel-title" data-toggle="cvcollapse" data-parent="#9db5e6bwfs" data-target="#3c23925rwc" href='index.html%3Fp=605.html' onclick='event.preventDefault()'>
		Session 48: Causal inference and statistical learning	</a>
	</div>
<div id="3c23925rwc" class="panel-collapse collapse ">
	<div class="panel-body">
		<div class="pt-cv-content"><p><strong>Session title</strong>: Causal inference and statistical learning<br />
<strong>Organizer</strong>: Cynthia Rudin (Duke)<br />
<strong>Chair:</strong> Cynthia Rudin (Duke)<br />
<strong>Time:</strong> June 6<sup>th</sup>, 3:15pm &#8211; 4:45pm<br />
<strong>Location: </strong>VEC 1402</p>
<p><strong>Speech 1:  Teaching History and Ethics of Data, with Python</strong><br />
<strong>Speaker: Chris Wiggins (Columbia &amp; NY Times)</strong><br />
<strong>Abstract: </strong>Data-empowered algorithms are reshaping our professional, personal, and political realities. However, existing curricula are predominantly designed either for future technologists, focusing on functional capabilities; or for future humanists, focusing on critical and rhetorical context surrounding data. &#8220;Data: Past, Present, and Future&#8221; is a new course at Columbia which seeks to define a curriculum at present taught to neither group, yet of interest and utility to future statisticians, CEOs, and senators alike. The course has been co-developed by Matt Jones, Professor at Columbia&#8217;s History department, and myself. The intellectual arc traces from the 18th century to present day, beginning with examples of contemporary technological advances, disquieting ethical debates, and financial success powered by panoptic persuasion architectures. The weekly cadence of the course pairs primary and secondary readings with Jupyter notebooks in Python, engaging directly with the data and intellectual advances under study. Throughout, these intellectual technical advances are paired with critical inquiry into the forces which encouraged and benefited from these new capabilities, i.e., the political dimension of data and technology. In this talk I will give an overview of lessons learned from teaching the class, and argue that 1) the material can be engaged by students from a wide variety of curricular backgrounds and 2) the structure of the class &#8212; using history to make the present strange, then critiquing the ethics of the technology-enabled future we are building &#8212; can be useful for a variety of subjects.  Syllabus, Jupyter notebooks, and additional info can be found via <a href="https://webmail.illinois.edu/owa/redir.aspx?C=pdfOVLBszAlGFSuBDFI1YQ_YDdqXdmsfVYNb66V90Nkvnazuu5TVCA..&amp;URL=https%3a%2f%2furldefense.proofpoint.com%2fv2%2furl%3fu%3dhttps-3A__data-2Dppf.github.io_%26d%3dDwIGaQ%26c%3dimBPVzF25OnBgGmVOlcsiEgHoG1i6YHLR0Sj_gZ4adc%26r%3dtOZZtjNyrCSrR8o-Z8CHQgSAixSz_BAEnVZS6kAcAqM%26m%3dF8OpY9yVfjfDb4WjIn47RbBGz-fwLvStzUaysCj_eBs%26s%3dnWjj0uj0jEKmFsD0m3J_SPilwvfY3Fs5ZxA0HMUKPao%26e%3d">https://urldefense.proofpoint.com/v2/url?u=https-3A__data-2Dppf.github.io_&amp;d=DwIGaQ&amp;c=imBPVzF25OnBgGmVOlcsiEgHoG1i6YHLR0Sj_gZ4adc&amp;r=tOZZtjNyrCSrR8o-Z8CHQgSAixSz_BAEnVZS6kAcAqM&amp;m=F8OpY9yVfjfDb4WjIn47RbBGz-fwLvStzUaysCj_eBs&amp;s=nWjj0uj0jEKmFsD0m3J_SPilwvfY3Fs5ZxA0HMUKPao&amp;e= </a>&#8220;Data: Past, Present, and Future&#8221; is supported by the Columbia University Collaboratory Fellows Fund. Jointly founded by Columbia University’s Data Science Institute and Columbia Entrepreneurship, The Collaboratory@Columbia is a university-wide program dedicated to supporting collaborative curricula innovations designed to ensure that all Columbia University students receive the education and training that they need to succeed in today’s data rich world.</p>
<p><strong>Speech 2: Bayesian optimization and A/B tests</strong><br />
<strong>Speaker: </strong>Ben Letham (Facebook data science)<br />
<strong>Abstract: </strong>Randomized experiments provide a direct, albeit time-consuming and noisy, measurement of the effect of changes to a system. We often want to optimize the parameters of systems that can only be evaluated via noisy experiments. I will describe how Bayesian optimization is used in this setting at Facebook, such as for optimizing web server compiler flags. I will then discuss our current efforts to expand the scope of optimization via field experiments.</p>
<p><strong>Speech 3: Causal inference from complex observational data</strong><br />
<strong>Speaker: Alex Volfovsky (Duke)</strong><br />
<strong>Abstract: </strong>A classical problem in causal inference is that of matching  treatment units to control units. Some of the main challenges in developing matching methods arise from the tension among (i) inclusion of as many covariates as possible in defining the matched groups, (ii) having matched groups with enough treated and control units for a valid estimate of Average Treatment Effect (ATE) in each group, (iii) computing the matched pairs efficiently for large datasets, and (iv) dealing with complicating factors such as non-independence among units. We propose the Fast Large-scale Almost Matching Exactly (FLAME) framework to tackle these problems. At its core this framework proposes an optimization objective for match quality that captures covariates that are integral for making causal statements while encouraging as many matches as possible. This objective can then be augmented to capture common complicating factors.</p>
</div>	</div>
</div></div></div></div></div></div>
	</div><!-- .entry-content -->
</article><!-- #post-608 -->

		</main><!-- #main -->
	</div><!-- #primary -->
</div><!-- .wrap -->


		</div><!-- #content -->

		<footer id="colophon" class="site-footer">
			<div class="wrap">
				

	<aside class="widget-area" aria-label="Footer">
					<div class="widget-column footer-widget-1">
				<section id="text-5" class="widget widget_text"><h2 class="widget-title">Contact:</h2>			<div class="textwidget"><p>Program Chairs: Annie Qu (anniequ@illinois.edu),<br />
Cynthia Rudin (cynthia@cs.duke.edu)</p>
<p>Registration: <span id="spnR" class="rwRRO" title="Stephanie@amstat.org">Brander, Stephanie ‎(Stephanie@amstat.org)</span></p>
<p>Web Design: Yujia Deng(yujiad2@illinois.edu)<br />
Copyright© SLDS2018 Committee</p>
</div>
		</section>			</div>
				</aside><!-- .widget-area -->

	<div class="site-info">
		<a href="https://wordpress.org/" class="imprint">
		Proudly powered by WordPress	</a>
</div><!-- .site-info -->
			</div><!-- .wrap -->
		</footer><!-- #colophon -->
	</div><!-- .site-content-contain -->
</div><!-- #page -->
<script type="speculationrules">
{"prefetch":[{"source":"document","where":{"and":[{"href_matches":"\/sldsc2018\/*"},{"not":{"href_matches":["\/sldsc2018\/wp-*.php","\/sldsc2018\/wp-admin\/*","\/sldsc2018\/files\/*","\/wp-content\/*","\/wp-content\/plugins\/*","\/wp-content\/themes\/twentyseventeen\/*","\/sldsc2018\/*\\?(.+)"]}},{"not":{"selector_matches":"a[rel~=\"nofollow\"]"}},{"not":{"selector_matches":".no-prefetch, .no-prefetch a"}}]},"eagerness":"conservative"}]}
</script>
<script id="pt-cv-content-views-script-js-extra">
var PT_CV_PUBLIC = {"_prefix":"pt-cv-","page_to_show":"5","_nonce":"38ae751f3f","is_admin":"","is_mobile":"","ajaxurl":"https:\/\/publish.illinois.edu\/sldsc2018\/wp-admin\/admin-ajax.php","lang":"","loading_image_src":"data:image\/gif;base64,R0lGODlhDwAPALMPAMrKygwMDJOTkz09PZWVla+vr3p6euTk5M7OzuXl5TMzMwAAAJmZmWZmZszMzP\/\/\/yH\/C05FVFNDQVBFMi4wAwEAAAAh+QQFCgAPACwAAAAADwAPAAAEQvDJaZaZOIcV8iQK8VRX4iTYoAwZ4iCYoAjZ4RxejhVNoT+mRGP4cyF4Pp0N98sBGIBMEMOotl6YZ3S61Bmbkm4mAgAh+QQFCgAPACwAAAAADQANAAAENPDJSRSZeA418itN8QiK8BiLITVsFiyBBIoYqnoewAD4xPw9iY4XLGYSjkQR4UAUD45DLwIAIfkEBQoADwAsAAAAAA8ACQAABC\/wyVlamTi3nSdgwFNdhEJgTJoNyoB9ISYoQmdjiZPcj7EYCAeCF1gEDo4Dz2eIAAAh+QQFCgAPACwCAAAADQANAAAEM\/DJBxiYeLKdX3IJZT1FU0iIg2RNKx3OkZVnZ98ToRD4MyiDnkAh6BkNC0MvsAj0kMpHBAAh+QQFCgAPACwGAAAACQAPAAAEMDC59KpFDll73HkAA2wVY5KgiK5b0RRoI6MuzG6EQqCDMlSGheEhUAgqgUUAFRySIgAh+QQFCgAPACwCAAIADQANAAAEM\/DJKZNLND\/kkKaHc3xk+QAMYDKsiaqmZCxGVjSFFCxB1vwy2oOgIDxuucxAMTAJFAJNBAAh+QQFCgAPACwAAAYADwAJAAAEMNAs86q1yaWwwv2Ig0jUZx3OYa4XoRAfwADXoAwfo1+CIjyFRuEho60aSNYlOPxEAAAh+QQFCgAPACwAAAIADQANAAAENPA9s4y8+IUVcqaWJ4qEQozSoAzoIyhCK2NFU2SJk0hNnyEOhKR2AzAAj4Pj4GE4W0bkJQIAOw=="};
var PT_CV_PAGINATION = {"first":"\u00ab","prev":"\u2039","next":"\u203a","last":"\u00bb","goto_first":"Go to first page","goto_prev":"Go to previous page","goto_next":"Go to next page","goto_last":"Go to last page","current_page":"Current page is","goto_page":"Go to page"};
</script>
<script src="../wp-content/plugins/content-views-query-and-display-post-page/public/assets/js/cv.js%3Fver=4.2" id="pt-cv-content-views-script-js"></script>
<script src="../wp-content/themes/twentyseventeen/assets/js/jquery.scrollTo.js%3Fver=2.1.3" id="jquery-scrollto-js" defer data-wp-strategy="defer"></script>
<script src="../wp-content/plugins/google-calendar-events/assets/generated/vendor/jquery.qtip.min.js%3Fver=3.5.5" id="simcal-qtip-js"></script>
<script id="simcal-default-calendar-js-extra">
var simcal_default_calendar = {"ajax_url":"https:\/\/publish.illinois.edu\/sldsc2018\/wp-admin\/admin-ajax.php","nonce":"a32c5df192","locale":"en_US","text_dir":"ltr","months":{"full":["January","February","March","April","May","June","July","August","September","October","November","December"],"short":["Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"]},"days":{"full":["Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"],"short":["Sun","Mon","Tue","Wed","Thu","Fri","Sat"]},"meridiem":{"AM":"AM","am":"am","PM":"PM","pm":"pm"}};
</script>
<script src="../wp-content/plugins/google-calendar-events/assets/generated/default-calendar.min.js%3Fver=3.5.5" id="simcal-default-calendar-js"></script>
<script src="../wp-content/plugins/google-calendar-events/assets/generated/vendor/imagesloaded.pkgd.min.js%3Fver=3.5.5" id="simplecalendar-imagesloaded-js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js?config=Accessible&amp;ver=1.3.13" id="mathjax-js"></script>
<svg style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<defs>
<symbol id="icon-behance" viewBox="0 0 37 32">
<path class="path1" d="M33 6.054h-9.125v2.214h9.125v-2.214zM28.5 13.661q-1.607 0-2.607 0.938t-1.107 2.545h7.286q-0.321-3.482-3.571-3.482zM28.786 24.107q1.125 0 2.179-0.571t1.357-1.554h3.946q-1.786 5.482-7.625 5.482-3.821 0-6.080-2.357t-2.259-6.196q0-3.714 2.33-6.17t6.009-2.455q2.464 0 4.295 1.214t2.732 3.196 0.902 4.429q0 0.304-0.036 0.839h-11.75q0 1.982 1.027 3.063t2.973 1.080zM4.946 23.214h5.286q3.661 0 3.661-2.982 0-3.214-3.554-3.214h-5.393v6.196zM4.946 13.625h5.018q1.393 0 2.205-0.652t0.813-2.027q0-2.571-3.393-2.571h-4.643v5.25zM0 4.536h10.607q1.554 0 2.768 0.25t2.259 0.848 1.607 1.723 0.563 2.75q0 3.232-3.071 4.696 2.036 0.571 3.071 2.054t1.036 3.643q0 1.339-0.438 2.438t-1.179 1.848-1.759 1.268-2.161 0.75-2.393 0.232h-10.911v-22.5z"></path>
</symbol>
<symbol id="icon-deviantart" viewBox="0 0 18 32">
<path class="path1" d="M18.286 5.411l-5.411 10.393 0.429 0.554h4.982v7.411h-9.054l-0.786 0.536-2.536 4.875-0.536 0.536h-5.375v-5.411l5.411-10.411-0.429-0.536h-4.982v-7.411h9.054l0.786-0.536 2.536-4.875 0.536-0.536h5.375v5.411z"></path>
</symbol>
<symbol id="icon-medium" viewBox="0 0 32 32">
<path class="path1" d="M10.661 7.518v20.946q0 0.446-0.223 0.759t-0.652 0.313q-0.304 0-0.589-0.143l-8.304-4.161q-0.375-0.179-0.634-0.598t-0.259-0.83v-20.357q0-0.357 0.179-0.607t0.518-0.25q0.25 0 0.786 0.268l9.125 4.571q0.054 0.054 0.054 0.089zM11.804 9.321l9.536 15.464-9.536-4.75v-10.714zM32 9.643v18.821q0 0.446-0.25 0.723t-0.679 0.277-0.839-0.232l-7.875-3.929zM31.946 7.5q0 0.054-4.58 7.491t-5.366 8.705l-6.964-11.321 5.786-9.411q0.304-0.5 0.929-0.5 0.25 0 0.464 0.107l9.661 4.821q0.071 0.036 0.071 0.107z"></path>
</symbol>
<symbol id="icon-slideshare" viewBox="0 0 32 32">
<path class="path1" d="M15.589 13.214q0 1.482-1.134 2.545t-2.723 1.063-2.723-1.063-1.134-2.545q0-1.5 1.134-2.554t2.723-1.054 2.723 1.054 1.134 2.554zM24.554 13.214q0 1.482-1.125 2.545t-2.732 1.063q-1.589 0-2.723-1.063t-1.134-2.545q0-1.5 1.134-2.554t2.723-1.054q1.607 0 2.732 1.054t1.125 2.554zM28.571 16.429v-11.911q0-1.554-0.571-2.205t-1.982-0.652h-19.857q-1.482 0-2.009 0.607t-0.527 2.25v12.018q0.768 0.411 1.58 0.714t1.446 0.5 1.446 0.33 1.268 0.196 1.25 0.071 1.045 0.009 1.009-0.036 0.795-0.036q1.214-0.018 1.696 0.482 0.107 0.107 0.179 0.161 0.464 0.446 1.089 0.911 0.125-1.625 2.107-1.554 0.089 0 0.652 0.027t0.768 0.036 0.813 0.018 0.946-0.018 0.973-0.080 1.089-0.152 1.107-0.241 1.196-0.348 1.205-0.482 1.286-0.616zM31.482 16.339q-2.161 2.661-6.643 4.5 1.5 5.089-0.411 8.304-1.179 2.018-3.268 2.643-1.857 0.571-3.25-0.268-1.536-0.911-1.464-2.929l-0.018-5.821v-0.018q-0.143-0.036-0.438-0.107t-0.42-0.089l-0.018 6.036q0.071 2.036-1.482 2.929-1.411 0.839-3.268 0.268-2.089-0.643-3.25-2.679-1.875-3.214-0.393-8.268-4.482-1.839-6.643-4.5-0.446-0.661-0.071-1.125t1.071 0.018q0.054 0.036 0.196 0.125t0.196 0.143v-12.393q0-1.286 0.839-2.196t2.036-0.911h22.446q1.196 0 2.036 0.911t0.839 2.196v12.393l0.375-0.268q0.696-0.482 1.071-0.018t-0.071 1.125z"></path>
</symbol>
<symbol id="icon-snapchat-ghost" viewBox="0 0 30 32">
<path class="path1" d="M15.143 2.286q2.393-0.018 4.295 1.223t2.92 3.438q0.482 1.036 0.482 3.196 0 0.839-0.161 3.411 0.25 0.125 0.5 0.125 0.321 0 0.911-0.241t0.911-0.241q0.518 0 1 0.321t0.482 0.821q0 0.571-0.563 0.964t-1.232 0.563-1.232 0.518-0.563 0.848q0 0.268 0.214 0.768 0.661 1.464 1.83 2.679t2.58 1.804q0.5 0.214 1.429 0.411 0.5 0.107 0.5 0.625 0 1.25-3.911 1.839-0.125 0.196-0.196 0.696t-0.25 0.83-0.589 0.33q-0.357 0-1.107-0.116t-1.143-0.116q-0.661 0-1.107 0.089-0.571 0.089-1.125 0.402t-1.036 0.679-1.036 0.723-1.357 0.598-1.768 0.241q-0.929 0-1.723-0.241t-1.339-0.598-1.027-0.723-1.036-0.679-1.107-0.402q-0.464-0.089-1.125-0.089-0.429 0-1.17 0.134t-1.045 0.134q-0.446 0-0.625-0.33t-0.25-0.848-0.196-0.714q-3.911-0.589-3.911-1.839 0-0.518 0.5-0.625 0.929-0.196 1.429-0.411 1.393-0.571 2.58-1.804t1.83-2.679q0.214-0.5 0.214-0.768 0-0.5-0.563-0.848t-1.241-0.527-1.241-0.563-0.563-0.938q0-0.482 0.464-0.813t0.982-0.33q0.268 0 0.857 0.232t0.946 0.232q0.321 0 0.571-0.125-0.161-2.536-0.161-3.393 0-2.179 0.482-3.214 1.143-2.446 3.071-3.536t4.714-1.125z"></path>
</symbol>
<symbol id="icon-yelp" viewBox="0 0 27 32">
<path class="path1" d="M13.804 23.554v2.268q-0.018 5.214-0.107 5.446-0.214 0.571-0.911 0.714-0.964 0.161-3.241-0.679t-2.902-1.589q-0.232-0.268-0.304-0.643-0.018-0.214 0.071-0.464 0.071-0.179 0.607-0.839t3.232-3.857q0.018 0 1.071-1.25 0.268-0.339 0.705-0.438t0.884 0.063q0.429 0.179 0.67 0.518t0.223 0.75zM11.143 19.071q-0.054 0.982-0.929 1.25l-2.143 0.696q-4.911 1.571-5.214 1.571-0.625-0.036-0.964-0.643-0.214-0.446-0.304-1.339-0.143-1.357 0.018-2.973t0.536-2.223 1-0.571q0.232 0 3.607 1.375 1.25 0.518 2.054 0.839l1.5 0.607q0.411 0.161 0.634 0.545t0.205 0.866zM25.893 24.375q-0.125 0.964-1.634 2.875t-2.42 2.268q-0.661 0.25-1.125-0.125-0.25-0.179-3.286-5.125l-0.839-1.375q-0.25-0.375-0.205-0.821t0.348-0.821q0.625-0.768 1.482-0.464 0.018 0.018 2.125 0.714 3.625 1.179 4.321 1.42t0.839 0.366q0.5 0.393 0.393 1.089zM13.893 13.089q0.089 1.821-0.964 2.179-1.036 0.304-2.036-1.268l-6.75-10.679q-0.143-0.625 0.339-1.107 0.732-0.768 3.705-1.598t4.009-0.563q0.714 0.179 0.875 0.804 0.054 0.321 0.393 5.455t0.429 6.777zM25.714 15.018q0.054 0.696-0.464 1.054-0.268 0.179-5.875 1.536-1.196 0.268-1.625 0.411l0.018-0.036q-0.411 0.107-0.821-0.071t-0.661-0.571q-0.536-0.839 0-1.554 0.018-0.018 1.339-1.821 2.232-3.054 2.679-3.643t0.607-0.696q0.5-0.339 1.161-0.036 0.857 0.411 2.196 2.384t1.446 2.991v0.054z"></path>
</symbol>
<symbol id="icon-vine" viewBox="0 0 27 32">
<path class="path1" d="M26.732 14.768v3.536q-1.804 0.411-3.536 0.411-1.161 2.429-2.955 4.839t-3.241 3.848-2.286 1.902q-1.429 0.804-2.893-0.054-0.5-0.304-1.080-0.777t-1.518-1.491-1.83-2.295-1.92-3.286-1.884-4.357-1.634-5.616-1.259-6.964h5.054q0.464 3.893 1.25 7.116t1.866 5.661 2.17 4.205 2.5 3.482q3.018-3.018 5.125-7.25-2.536-1.286-3.982-3.929t-1.446-5.946q0-3.429 1.857-5.616t5.071-2.188q3.179 0 4.875 1.884t1.696 5.313q0 2.839-1.036 5.107-0.125 0.018-0.348 0.054t-0.821 0.036-1.125-0.107-1.107-0.455-0.902-0.92q0.554-1.839 0.554-3.286 0-1.554-0.518-2.357t-1.411-0.804q-0.946 0-1.518 0.884t-0.571 2.509q0 3.321 1.875 5.241t4.768 1.92q1.107 0 2.161-0.25z"></path>
</symbol>
<symbol id="icon-vk" viewBox="0 0 35 32">
<path class="path1" d="M34.232 9.286q0.411 1.143-2.679 5.25-0.429 0.571-1.161 1.518-1.393 1.786-1.607 2.339-0.304 0.732 0.25 1.446 0.304 0.375 1.446 1.464h0.018l0.071 0.071q2.518 2.339 3.411 3.946 0.054 0.089 0.116 0.223t0.125 0.473-0.009 0.607-0.446 0.491-1.054 0.223l-4.571 0.071q-0.429 0.089-1-0.089t-0.929-0.393l-0.357-0.214q-0.536-0.375-1.25-1.143t-1.223-1.384-1.089-1.036-1.009-0.277q-0.054 0.018-0.143 0.063t-0.304 0.259-0.384 0.527-0.304 0.929-0.116 1.384q0 0.268-0.063 0.491t-0.134 0.33l-0.071 0.089q-0.321 0.339-0.946 0.393h-2.054q-1.268 0.071-2.607-0.295t-2.348-0.946-1.839-1.179-1.259-1.027l-0.446-0.429q-0.179-0.179-0.491-0.536t-1.277-1.625-1.893-2.696-2.188-3.768-2.33-4.857q-0.107-0.286-0.107-0.482t0.054-0.286l0.071-0.107q0.268-0.339 1.018-0.339l4.893-0.036q0.214 0.036 0.411 0.116t0.286 0.152l0.089 0.054q0.286 0.196 0.429 0.571 0.357 0.893 0.821 1.848t0.732 1.455l0.286 0.518q0.518 1.071 1 1.857t0.866 1.223 0.741 0.688 0.607 0.25 0.482-0.089q0.036-0.018 0.089-0.089t0.214-0.393 0.241-0.839 0.17-1.446 0-2.232q-0.036-0.714-0.161-1.304t-0.25-0.821l-0.107-0.214q-0.446-0.607-1.518-0.768-0.232-0.036 0.089-0.429 0.304-0.339 0.679-0.536 0.946-0.464 4.268-0.429 1.464 0.018 2.411 0.232 0.357 0.089 0.598 0.241t0.366 0.429 0.188 0.571 0.063 0.813-0.018 0.982-0.045 1.259-0.027 1.473q0 0.196-0.018 0.75t-0.009 0.857 0.063 0.723 0.205 0.696 0.402 0.438q0.143 0.036 0.304 0.071t0.464-0.196 0.679-0.616 0.929-1.196 1.214-1.92q1.071-1.857 1.911-4.018 0.071-0.179 0.179-0.313t0.196-0.188l0.071-0.054 0.089-0.045t0.232-0.054 0.357-0.009l5.143-0.036q0.696-0.089 1.143 0.045t0.554 0.295z"></path>
</symbol>
<symbol id="icon-search" viewBox="0 0 30 32">
<path class="path1" d="M20.571 14.857q0-3.304-2.348-5.652t-5.652-2.348-5.652 2.348-2.348 5.652 2.348 5.652 5.652 2.348 5.652-2.348 2.348-5.652zM29.714 29.714q0 0.929-0.679 1.607t-1.607 0.679q-0.964 0-1.607-0.679l-6.125-6.107q-3.196 2.214-7.125 2.214-2.554 0-4.884-0.991t-4.018-2.679-2.679-4.018-0.991-4.884 0.991-4.884 2.679-4.018 4.018-2.679 4.884-0.991 4.884 0.991 4.018 2.679 2.679 4.018 0.991 4.884q0 3.929-2.214 7.125l6.125 6.125q0.661 0.661 0.661 1.607z"></path>
</symbol>
<symbol id="icon-envelope-o" viewBox="0 0 32 32">
<path class="path1" d="M29.714 26.857v-13.714q-0.571 0.643-1.232 1.179-4.786 3.679-7.607 6.036-0.911 0.768-1.482 1.196t-1.545 0.866-1.83 0.438h-0.036q-0.857 0-1.83-0.438t-1.545-0.866-1.482-1.196q-2.821-2.357-7.607-6.036-0.661-0.536-1.232-1.179v13.714q0 0.232 0.17 0.402t0.402 0.17h26.286q0.232 0 0.402-0.17t0.17-0.402zM29.714 8.089v-0.438t-0.009-0.232-0.054-0.223-0.098-0.161-0.161-0.134-0.25-0.045h-26.286q-0.232 0-0.402 0.17t-0.17 0.402q0 3 2.625 5.071 3.446 2.714 7.161 5.661 0.107 0.089 0.625 0.527t0.821 0.67 0.795 0.563 0.902 0.491 0.768 0.161h0.036q0.357 0 0.768-0.161t0.902-0.491 0.795-0.563 0.821-0.67 0.625-0.527q3.714-2.946 7.161-5.661 0.964-0.768 1.795-2.063t0.83-2.348zM32 7.429v19.429q0 1.179-0.839 2.018t-2.018 0.839h-26.286q-1.179 0-2.018-0.839t-0.839-2.018v-19.429q0-1.179 0.839-2.018t2.018-0.839h26.286q1.179 0 2.018 0.839t0.839 2.018z"></path>
</symbol>
<symbol id="icon-close" viewBox="0 0 25 32">
<path class="path1" d="M23.179 23.607q0 0.714-0.5 1.214l-2.429 2.429q-0.5 0.5-1.214 0.5t-1.214-0.5l-5.25-5.25-5.25 5.25q-0.5 0.5-1.214 0.5t-1.214-0.5l-2.429-2.429q-0.5-0.5-0.5-1.214t0.5-1.214l5.25-5.25-5.25-5.25q-0.5-0.5-0.5-1.214t0.5-1.214l2.429-2.429q0.5-0.5 1.214-0.5t1.214 0.5l5.25 5.25 5.25-5.25q0.5-0.5 1.214-0.5t1.214 0.5l2.429 2.429q0.5 0.5 0.5 1.214t-0.5 1.214l-5.25 5.25 5.25 5.25q0.5 0.5 0.5 1.214z"></path>
</symbol>
<symbol id="icon-angle-down" viewBox="0 0 21 32">
<path class="path1" d="M19.196 13.143q0 0.232-0.179 0.411l-8.321 8.321q-0.179 0.179-0.411 0.179t-0.411-0.179l-8.321-8.321q-0.179-0.179-0.179-0.411t0.179-0.411l0.893-0.893q0.179-0.179 0.411-0.179t0.411 0.179l7.018 7.018 7.018-7.018q0.179-0.179 0.411-0.179t0.411 0.179l0.893 0.893q0.179 0.179 0.179 0.411z"></path>
</symbol>
<symbol id="icon-folder-open" viewBox="0 0 34 32">
<path class="path1" d="M33.554 17q0 0.554-0.554 1.179l-6 7.071q-0.768 0.911-2.152 1.545t-2.563 0.634h-19.429q-0.607 0-1.080-0.232t-0.473-0.768q0-0.554 0.554-1.179l6-7.071q0.768-0.911 2.152-1.545t2.563-0.634h19.429q0.607 0 1.080 0.232t0.473 0.768zM27.429 10.857v2.857h-14.857q-1.679 0-3.518 0.848t-2.929 2.134l-6.107 7.179q0-0.071-0.009-0.223t-0.009-0.223v-17.143q0-1.643 1.179-2.821t2.821-1.179h5.714q1.643 0 2.821 1.179t1.179 2.821v0.571h9.714q1.643 0 2.821 1.179t1.179 2.821z"></path>
</symbol>
<symbol id="icon-twitter" viewBox="0 0 30 32">
<path class="path1" d="M28.929 7.286q-1.196 1.75-2.893 2.982 0.018 0.25 0.018 0.75 0 2.321-0.679 4.634t-2.063 4.437-3.295 3.759-4.607 2.607-5.768 0.973q-4.839 0-8.857-2.589 0.625 0.071 1.393 0.071 4.018 0 7.161-2.464-1.875-0.036-3.357-1.152t-2.036-2.848q0.589 0.089 1.089 0.089 0.768 0 1.518-0.196-2-0.411-3.313-1.991t-1.313-3.67v-0.071q1.214 0.679 2.607 0.732-1.179-0.786-1.875-2.054t-0.696-2.75q0-1.571 0.786-2.911 2.161 2.661 5.259 4.259t6.634 1.777q-0.143-0.679-0.143-1.321 0-2.393 1.688-4.080t4.080-1.688q2.5 0 4.214 1.821 1.946-0.375 3.661-1.393-0.661 2.054-2.536 3.179 1.661-0.179 3.321-0.893z"></path>
</symbol>
<symbol id="icon-facebook" viewBox="0 0 19 32">
<path class="path1" d="M17.125 0.214v4.714h-2.804q-1.536 0-2.071 0.643t-0.536 1.929v3.375h5.232l-0.696 5.286h-4.536v13.554h-5.464v-13.554h-4.554v-5.286h4.554v-3.893q0-3.321 1.857-5.152t4.946-1.83q2.625 0 4.071 0.214z"></path>
</symbol>
<symbol id="icon-github" viewBox="0 0 27 32">
<path class="path1" d="M13.714 2.286q3.732 0 6.884 1.839t4.991 4.991 1.839 6.884q0 4.482-2.616 8.063t-6.759 4.955q-0.482 0.089-0.714-0.125t-0.232-0.536q0-0.054 0.009-1.366t0.009-2.402q0-1.732-0.929-2.536 1.018-0.107 1.83-0.321t1.679-0.696 1.446-1.188 0.946-1.875 0.366-2.688q0-2.125-1.411-3.679 0.661-1.625-0.143-3.643-0.5-0.161-1.446 0.196t-1.643 0.786l-0.679 0.429q-1.661-0.464-3.429-0.464t-3.429 0.464q-0.286-0.196-0.759-0.482t-1.491-0.688-1.518-0.241q-0.804 2.018-0.143 3.643-1.411 1.554-1.411 3.679 0 1.518 0.366 2.679t0.938 1.875 1.438 1.196 1.679 0.696 1.83 0.321q-0.696 0.643-0.875 1.839-0.375 0.179-0.804 0.268t-1.018 0.089-1.17-0.384-0.991-1.116q-0.339-0.571-0.866-0.929t-0.884-0.429l-0.357-0.054q-0.375 0-0.518 0.080t-0.089 0.205 0.161 0.25 0.232 0.214l0.125 0.089q0.393 0.179 0.777 0.679t0.563 0.911l0.179 0.411q0.232 0.679 0.786 1.098t1.196 0.536 1.241 0.125 0.991-0.063l0.411-0.071q0 0.679 0.009 1.58t0.009 0.973q0 0.321-0.232 0.536t-0.714 0.125q-4.143-1.375-6.759-4.955t-2.616-8.063q0-3.732 1.839-6.884t4.991-4.991 6.884-1.839zM5.196 21.982q0.054-0.125-0.125-0.214-0.179-0.054-0.232 0.036-0.054 0.125 0.125 0.214 0.161 0.107 0.232-0.036zM5.75 22.589q0.125-0.089-0.036-0.286-0.179-0.161-0.286-0.054-0.125 0.089 0.036 0.286 0.179 0.179 0.286 0.054zM6.286 23.393q0.161-0.125 0-0.339-0.143-0.232-0.304-0.107-0.161 0.089 0 0.321t0.304 0.125zM7.036 24.143q0.143-0.143-0.071-0.339-0.214-0.214-0.357-0.054-0.161 0.143 0.071 0.339 0.214 0.214 0.357 0.054zM8.054 24.589q0.054-0.196-0.232-0.286-0.268-0.071-0.339 0.125t0.232 0.268q0.268 0.107 0.339-0.107zM9.179 24.679q0-0.232-0.304-0.196-0.286 0-0.286 0.196 0 0.232 0.304 0.196 0.286 0 0.286-0.196zM10.214 24.5q-0.036-0.196-0.321-0.161-0.286 0.054-0.25 0.268t0.321 0.143 0.25-0.25z"></path>
</symbol>
<symbol id="icon-bars" viewBox="0 0 27 32">
<path class="path1" d="M27.429 24v2.286q0 0.464-0.339 0.804t-0.804 0.339h-25.143q-0.464 0-0.804-0.339t-0.339-0.804v-2.286q0-0.464 0.339-0.804t0.804-0.339h25.143q0.464 0 0.804 0.339t0.339 0.804zM27.429 14.857v2.286q0 0.464-0.339 0.804t-0.804 0.339h-25.143q-0.464 0-0.804-0.339t-0.339-0.804v-2.286q0-0.464 0.339-0.804t0.804-0.339h25.143q0.464 0 0.804 0.339t0.339 0.804zM27.429 5.714v2.286q0 0.464-0.339 0.804t-0.804 0.339h-25.143q-0.464 0-0.804-0.339t-0.339-0.804v-2.286q0-0.464 0.339-0.804t0.804-0.339h25.143q0.464 0 0.804 0.339t0.339 0.804z"></path>
</symbol>
<symbol id="icon-google-plus" viewBox="0 0 41 32">
<path class="path1" d="M25.661 16.304q0 3.714-1.554 6.616t-4.429 4.536-6.589 1.634q-2.661 0-5.089-1.036t-4.179-2.786-2.786-4.179-1.036-5.089 1.036-5.089 2.786-4.179 4.179-2.786 5.089-1.036q5.107 0 8.768 3.429l-3.554 3.411q-2.089-2.018-5.214-2.018-2.196 0-4.063 1.107t-2.955 3.009-1.089 4.152 1.089 4.152 2.955 3.009 4.063 1.107q1.482 0 2.723-0.411t2.045-1.027 1.402-1.402 0.875-1.482 0.384-1.321h-7.429v-4.5h12.357q0.214 1.125 0.214 2.179zM41.143 14.125v3.75h-3.732v3.732h-3.75v-3.732h-3.732v-3.75h3.732v-3.732h3.75v3.732h3.732z"></path>
</symbol>
<symbol id="icon-linkedin" viewBox="0 0 27 32">
<path class="path1" d="M6.232 11.161v17.696h-5.893v-17.696h5.893zM6.607 5.696q0.018 1.304-0.902 2.179t-2.42 0.875h-0.036q-1.464 0-2.357-0.875t-0.893-2.179q0-1.321 0.92-2.188t2.402-0.866 2.375 0.866 0.911 2.188zM27.429 18.714v10.143h-5.875v-9.464q0-1.875-0.723-2.938t-2.259-1.063q-1.125 0-1.884 0.616t-1.134 1.527q-0.196 0.536-0.196 1.446v9.875h-5.875q0.036-7.125 0.036-11.554t-0.018-5.286l-0.018-0.857h5.875v2.571h-0.036q0.357-0.571 0.732-1t1.009-0.929 1.554-0.777 2.045-0.277q3.054 0 4.911 2.027t1.857 5.938z"></path>
</symbol>
<symbol id="icon-quote-right" viewBox="0 0 30 32">
<path class="path1" d="M13.714 5.714v12.571q0 1.857-0.723 3.545t-1.955 2.92-2.92 1.955-3.545 0.723h-1.143q-0.464 0-0.804-0.339t-0.339-0.804v-2.286q0-0.464 0.339-0.804t0.804-0.339h1.143q1.893 0 3.232-1.339t1.339-3.232v-0.571q0-0.714-0.5-1.214t-1.214-0.5h-4q-1.429 0-2.429-1t-1-2.429v-6.857q0-1.429 1-2.429t2.429-1h6.857q1.429 0 2.429 1t1 2.429zM29.714 5.714v12.571q0 1.857-0.723 3.545t-1.955 2.92-2.92 1.955-3.545 0.723h-1.143q-0.464 0-0.804-0.339t-0.339-0.804v-2.286q0-0.464 0.339-0.804t0.804-0.339h1.143q1.893 0 3.232-1.339t1.339-3.232v-0.571q0-0.714-0.5-1.214t-1.214-0.5h-4q-1.429 0-2.429-1t-1-2.429v-6.857q0-1.429 1-2.429t2.429-1h6.857q1.429 0 2.429 1t1 2.429z"></path>
</symbol>
<symbol id="icon-mail-reply" viewBox="0 0 32 32">
<path class="path1" d="M32 20q0 2.964-2.268 8.054-0.054 0.125-0.188 0.429t-0.241 0.536-0.232 0.393q-0.214 0.304-0.5 0.304-0.268 0-0.42-0.179t-0.152-0.446q0-0.161 0.045-0.473t0.045-0.42q0.089-1.214 0.089-2.196 0-1.804-0.313-3.232t-0.866-2.473-1.429-1.804-1.884-1.241-2.375-0.759-2.75-0.384-3.134-0.107h-4v4.571q0 0.464-0.339 0.804t-0.804 0.339-0.804-0.339l-9.143-9.143q-0.339-0.339-0.339-0.804t0.339-0.804l9.143-9.143q0.339-0.339 0.804-0.339t0.804 0.339 0.339 0.804v4.571h4q12.732 0 15.625 7.196 0.946 2.393 0.946 5.946z"></path>
</symbol>
<symbol id="icon-youtube" viewBox="0 0 27 32">
<path class="path1" d="M17.339 22.214v3.768q0 1.196-0.696 1.196-0.411 0-0.804-0.393v-5.375q0.393-0.393 0.804-0.393 0.696 0 0.696 1.196zM23.375 22.232v0.821h-1.607v-0.821q0-1.214 0.804-1.214t0.804 1.214zM6.125 18.339h1.911v-1.679h-5.571v1.679h1.875v10.161h1.786v-10.161zM11.268 28.5h1.589v-8.821h-1.589v6.75q-0.536 0.75-1.018 0.75-0.321 0-0.375-0.375-0.018-0.054-0.018-0.625v-6.5h-1.589v6.982q0 0.875 0.143 1.304 0.214 0.661 1.036 0.661 0.857 0 1.821-1.089v0.964zM18.929 25.857v-3.518q0-1.304-0.161-1.768-0.304-1-1.268-1-0.893 0-1.661 0.964v-3.875h-1.589v11.839h1.589v-0.857q0.804 0.982 1.661 0.982 0.964 0 1.268-0.982 0.161-0.482 0.161-1.786zM24.964 25.679v-0.232h-1.625q0 0.911-0.036 1.089-0.125 0.643-0.714 0.643-0.821 0-0.821-1.232v-1.554h3.196v-1.839q0-1.411-0.482-2.071-0.696-0.911-1.893-0.911-1.214 0-1.911 0.911-0.5 0.661-0.5 2.071v3.089q0 1.411 0.518 2.071 0.696 0.911 1.929 0.911 1.286 0 1.929-0.946 0.321-0.482 0.375-0.964 0.036-0.161 0.036-1.036zM14.107 9.375v-3.75q0-1.232-0.768-1.232t-0.768 1.232v3.75q0 1.25 0.768 1.25t0.768-1.25zM26.946 22.786q0 4.179-0.464 6.25-0.25 1.054-1.036 1.768t-1.821 0.821q-3.286 0.375-9.911 0.375t-9.911-0.375q-1.036-0.107-1.83-0.821t-1.027-1.768q-0.464-2-0.464-6.25 0-4.179 0.464-6.25 0.25-1.054 1.036-1.768t1.839-0.839q3.268-0.357 9.893-0.357t9.911 0.357q1.036 0.125 1.83 0.839t1.027 1.768q0.464 2 0.464 6.25zM9.125 0h1.821l-2.161 7.125v4.839h-1.786v-4.839q-0.25-1.321-1.089-3.786-0.661-1.839-1.161-3.339h1.893l1.268 4.696zM15.732 5.946v3.125q0 1.446-0.5 2.107-0.661 0.911-1.893 0.911-1.196 0-1.875-0.911-0.5-0.679-0.5-2.107v-3.125q0-1.429 0.5-2.089 0.679-0.911 1.875-0.911 1.232 0 1.893 0.911 0.5 0.661 0.5 2.089zM21.714 3.054v8.911h-1.625v-0.982q-0.946 1.107-1.839 1.107-0.821 0-1.054-0.661-0.143-0.429-0.143-1.339v-7.036h1.625v6.554q0 0.589 0.018 0.625 0.054 0.393 0.375 0.393 0.482 0 1.018-0.768v-6.804h1.625z"></path>
</symbol>
<symbol id="icon-dropbox" viewBox="0 0 32 32">
<path class="path1" d="M7.179 12.625l8.821 5.446-6.107 5.089-8.75-5.696zM24.786 22.536v1.929l-8.75 5.232v0.018l-0.018-0.018-0.018 0.018v-0.018l-8.732-5.232v-1.929l2.625 1.714 6.107-5.071v-0.036l0.018 0.018 0.018-0.018v0.036l6.125 5.071zM9.893 2.107l6.107 5.089-8.821 5.429-6.036-4.821zM24.821 12.625l6.036 4.839-8.732 5.696-6.125-5.089zM22.125 2.107l8.732 5.696-6.036 4.821-8.821-5.429z"></path>
</symbol>
<symbol id="icon-instagram" viewBox="0 0 27 32">
<path class="path1" d="M18.286 16q0-1.893-1.339-3.232t-3.232-1.339-3.232 1.339-1.339 3.232 1.339 3.232 3.232 1.339 3.232-1.339 1.339-3.232zM20.75 16q0 2.929-2.054 4.982t-4.982 2.054-4.982-2.054-2.054-4.982 2.054-4.982 4.982-2.054 4.982 2.054 2.054 4.982zM22.679 8.679q0 0.679-0.482 1.161t-1.161 0.482-1.161-0.482-0.482-1.161 0.482-1.161 1.161-0.482 1.161 0.482 0.482 1.161zM13.714 4.75q-0.125 0-1.366-0.009t-1.884 0-1.723 0.054-1.839 0.179-1.277 0.33q-0.893 0.357-1.571 1.036t-1.036 1.571q-0.196 0.518-0.33 1.277t-0.179 1.839-0.054 1.723 0 1.884 0.009 1.366-0.009 1.366 0 1.884 0.054 1.723 0.179 1.839 0.33 1.277q0.357 0.893 1.036 1.571t1.571 1.036q0.518 0.196 1.277 0.33t1.839 0.179 1.723 0.054 1.884 0 1.366-0.009 1.366 0.009 1.884 0 1.723-0.054 1.839-0.179 1.277-0.33q0.893-0.357 1.571-1.036t1.036-1.571q0.196-0.518 0.33-1.277t0.179-1.839 0.054-1.723 0-1.884-0.009-1.366 0.009-1.366 0-1.884-0.054-1.723-0.179-1.839-0.33-1.277q-0.357-0.893-1.036-1.571t-1.571-1.036q-0.518-0.196-1.277-0.33t-1.839-0.179-1.723-0.054-1.884 0-1.366 0.009zM27.429 16q0 4.089-0.089 5.661-0.179 3.714-2.214 5.75t-5.75 2.214q-1.571 0.089-5.661 0.089t-5.661-0.089q-3.714-0.179-5.75-2.214t-2.214-5.75q-0.089-1.571-0.089-5.661t0.089-5.661q0.179-3.714 2.214-5.75t5.75-2.214q1.571-0.089 5.661-0.089t5.661 0.089q3.714 0.179 5.75 2.214t2.214 5.75q0.089 1.571 0.089 5.661z"></path>
</symbol>
<symbol id="icon-flickr" viewBox="0 0 27 32">
<path class="path1" d="M22.286 2.286q2.125 0 3.634 1.509t1.509 3.634v17.143q0 2.125-1.509 3.634t-3.634 1.509h-17.143q-2.125 0-3.634-1.509t-1.509-3.634v-17.143q0-2.125 1.509-3.634t3.634-1.509h17.143zM12.464 16q0-1.571-1.107-2.679t-2.679-1.107-2.679 1.107-1.107 2.679 1.107 2.679 2.679 1.107 2.679-1.107 1.107-2.679zM22.536 16q0-1.571-1.107-2.679t-2.679-1.107-2.679 1.107-1.107 2.679 1.107 2.679 2.679 1.107 2.679-1.107 1.107-2.679z"></path>
</symbol>
<symbol id="icon-tumblr" viewBox="0 0 19 32">
<path class="path1" d="M16.857 23.732l1.429 4.232q-0.411 0.625-1.982 1.179t-3.161 0.571q-1.857 0.036-3.402-0.464t-2.545-1.321-1.696-1.893-0.991-2.143-0.295-2.107v-9.714h-3v-3.839q1.286-0.464 2.304-1.241t1.625-1.607 1.036-1.821 0.607-1.768 0.268-1.58q0.018-0.089 0.080-0.152t0.134-0.063h4.357v7.571h5.946v4.5h-5.964v9.25q0 0.536 0.116 1t0.402 0.938 0.884 0.741 1.455 0.25q1.393-0.036 2.393-0.518z"></path>
</symbol>
<symbol id="icon-dockerhub" viewBox="0 0 24 28">
<path class="path1" d="M1.597 10.257h2.911v2.83H1.597v-2.83zm3.573 0h2.91v2.83H5.17v-2.83zm0-3.627h2.91v2.829H5.17V6.63zm3.57 3.627h2.912v2.83H8.74v-2.83zm0-3.627h2.912v2.829H8.74V6.63zm3.573 3.627h2.911v2.83h-2.911v-2.83zm0-3.627h2.911v2.829h-2.911V6.63zm3.572 3.627h2.911v2.83h-2.911v-2.83zM12.313 3h2.911v2.83h-2.911V3zm-6.65 14.173c-.449 0-.812.354-.812.788 0 .435.364.788.812.788.447 0 .811-.353.811-.788 0-.434-.363-.788-.811-.788"></path>
<path class="path2" d="M28.172 11.721c-.978-.549-2.278-.624-3.388-.306-.136-1.146-.91-2.149-1.83-2.869l-.366-.286-.307.345c-.618.692-.8 1.845-.718 2.73.063.651.273 1.312.685 1.834-.313.183-.668.328-.985.434-.646.212-1.347.33-2.028.33H.083l-.042.429c-.137 1.432.065 2.866.674 4.173l.262.519.03.048c1.8 2.973 4.963 4.225 8.41 4.225 6.672 0 12.174-2.896 14.702-9.015 1.689.085 3.417-.4 4.243-1.968l.211-.4-.401-.223zM5.664 19.458c-.85 0-1.542-.671-1.542-1.497 0-.825.691-1.498 1.541-1.498.849 0 1.54.672 1.54 1.497s-.69 1.498-1.539 1.498z"></path>
</symbol>
<symbol id="icon-dribbble" viewBox="0 0 27 32">
<path class="path1" d="M18.286 26.786q-0.75-4.304-2.5-8.893h-0.036l-0.036 0.018q-0.286 0.107-0.768 0.295t-1.804 0.875-2.446 1.464-2.339 2.045-1.839 2.643l-0.268-0.196q3.286 2.679 7.464 2.679 2.357 0 4.571-0.929zM14.982 15.946q-0.375-0.875-0.946-1.982-5.554 1.661-12.018 1.661-0.018 0.125-0.018 0.375 0 2.214 0.786 4.223t2.214 3.598q0.893-1.589 2.205-2.973t2.545-2.223 2.33-1.446 1.777-0.857l0.661-0.232q0.071-0.018 0.232-0.063t0.232-0.080zM13.071 12.161q-2.143-3.804-4.357-6.75-2.464 1.161-4.179 3.321t-2.286 4.857q5.393 0 10.821-1.429zM25.286 17.857q-3.75-1.071-7.304-0.518 1.554 4.268 2.286 8.375 1.982-1.339 3.304-3.384t1.714-4.473zM10.911 4.625q-0.018 0-0.036 0.018 0.018-0.018 0.036-0.018zM21.446 7.214q-3.304-2.929-7.732-2.929-1.357 0-2.768 0.339 2.339 3.036 4.393 6.821 1.232-0.464 2.321-1.080t1.723-1.098 1.17-1.018 0.67-0.723zM25.429 15.875q-0.054-4.143-2.661-7.321l-0.018 0.018q-0.161 0.214-0.339 0.438t-0.777 0.795-1.268 1.080-1.786 1.161-2.348 1.152q0.446 0.946 0.786 1.696 0.036 0.107 0.116 0.313t0.134 0.295q0.643-0.089 1.33-0.125t1.313-0.036 1.232 0.027 1.143 0.071 1.009 0.098 0.857 0.116 0.652 0.107 0.446 0.080zM27.429 16q0 3.732-1.839 6.884t-4.991 4.991-6.884 1.839-6.884-1.839-4.991-4.991-1.839-6.884 1.839-6.884 4.991-4.991 6.884-1.839 6.884 1.839 4.991 4.991 1.839 6.884z"></path>
</symbol>
<symbol id="icon-skype" viewBox="0 0 27 32">
<path class="path1" d="M20.946 18.982q0-0.893-0.348-1.634t-0.866-1.223-1.304-0.875-1.473-0.607-1.563-0.411l-1.857-0.429q-0.536-0.125-0.786-0.188t-0.625-0.205-0.536-0.286-0.295-0.375-0.134-0.536q0-1.375 2.571-1.375 0.768 0 1.375 0.214t0.964 0.509 0.679 0.598 0.714 0.518 0.857 0.214q0.839 0 1.348-0.571t0.509-1.375q0-0.982-1-1.777t-2.536-1.205-3.25-0.411q-1.214 0-2.357 0.277t-2.134 0.839-1.589 1.554-0.598 2.295q0 1.089 0.339 1.902t1 1.348 1.429 0.866 1.839 0.58l2.607 0.643q1.607 0.393 2 0.643 0.571 0.357 0.571 1.071 0 0.696-0.714 1.152t-1.875 0.455q-0.911 0-1.634-0.286t-1.161-0.688-0.813-0.804-0.821-0.688-0.964-0.286q-0.893 0-1.348 0.536t-0.455 1.339q0 1.643 2.179 2.813t5.196 1.17q1.304 0 2.5-0.33t2.188-0.955 1.58-1.67 0.589-2.348zM27.429 22.857q0 2.839-2.009 4.848t-4.848 2.009q-2.321 0-4.179-1.429-1.375 0.286-2.679 0.286-2.554 0-4.884-0.991t-4.018-2.679-2.679-4.018-0.991-4.884q0-1.304 0.286-2.679-1.429-1.857-1.429-4.179 0-2.839 2.009-4.848t4.848-2.009q2.321 0 4.179 1.429 1.375-0.286 2.679-0.286 2.554 0 4.884 0.991t4.018 2.679 2.679 4.018 0.991 4.884q0 1.304-0.286 2.679 1.429 1.857 1.429 4.179z"></path>
</symbol>
<symbol id="icon-foursquare" viewBox="0 0 23 32">
<path class="path1" d="M17.857 7.75l0.661-3.464q0.089-0.411-0.161-0.714t-0.625-0.304h-12.714q-0.411 0-0.688 0.304t-0.277 0.661v19.661q0 0.125 0.107 0.018l5.196-6.286q0.411-0.464 0.679-0.598t0.857-0.134h4.268q0.393 0 0.661-0.259t0.321-0.527q0.429-2.321 0.661-3.411 0.071-0.375-0.205-0.714t-0.652-0.339h-5.25q-0.518 0-0.857-0.339t-0.339-0.857v-0.75q0-0.518 0.339-0.848t0.857-0.33h6.179q0.321 0 0.625-0.241t0.357-0.527zM21.911 3.786q-0.268 1.304-0.955 4.759t-1.241 6.25-0.625 3.098q-0.107 0.393-0.161 0.58t-0.25 0.58-0.438 0.589-0.688 0.375-1.036 0.179h-4.839q-0.232 0-0.393 0.179-0.143 0.161-7.607 8.821-0.393 0.446-1.045 0.509t-0.866-0.098q-0.982-0.393-0.982-1.75v-25.179q0-0.982 0.679-1.83t2.143-0.848h15.857q1.696 0 2.268 0.946t0.179 2.839zM21.911 3.786l-2.821 14.107q0.071-0.304 0.625-3.098t1.241-6.25 0.955-4.759z"></path>
</symbol>
<symbol id="icon-wordpress" viewBox="0 0 32 32">
<path class="path1" d="M2.268 16q0-2.911 1.196-5.589l6.554 17.946q-3.5-1.696-5.625-5.018t-2.125-7.339zM25.268 15.304q0 0.339-0.045 0.688t-0.179 0.884-0.205 0.786-0.313 1.054-0.313 1.036l-1.357 4.571-4.964-14.75q0.821-0.054 1.571-0.143 0.339-0.036 0.464-0.33t-0.045-0.554-0.509-0.241l-3.661 0.179q-1.339-0.018-3.607-0.179-0.214-0.018-0.366 0.089t-0.205 0.268-0.027 0.33 0.161 0.295 0.348 0.143l1.429 0.143 2.143 5.857-3 9-5-14.857q0.821-0.054 1.571-0.143 0.339-0.036 0.464-0.33t-0.045-0.554-0.509-0.241l-3.661 0.179q-0.125 0-0.411-0.009t-0.464-0.009q1.875-2.857 4.902-4.527t6.563-1.67q2.625 0 5.009 0.946t4.259 2.661h-0.179q-0.982 0-1.643 0.723t-0.661 1.705q0 0.214 0.036 0.429t0.071 0.384 0.143 0.411 0.161 0.375 0.214 0.402 0.223 0.375 0.259 0.429 0.25 0.411q1.125 1.911 1.125 3.786zM16.232 17.196l4.232 11.554q0.018 0.107 0.089 0.196-2.25 0.786-4.554 0.786-2 0-3.875-0.571zM28.036 9.411q1.696 3.107 1.696 6.589 0 3.732-1.857 6.884t-4.982 4.973l4.196-12.107q1.054-3.018 1.054-4.929 0-0.75-0.107-1.411zM16 0q3.25 0 6.214 1.268t5.107 3.411 3.411 5.107 1.268 6.214-1.268 6.214-3.411 5.107-5.107 3.411-6.214 1.268-6.214-1.268-5.107-3.411-3.411-5.107-1.268-6.214 1.268-6.214 3.411-5.107 5.107-3.411 6.214-1.268zM16 31.268q3.089 0 5.92-1.214t4.875-3.259 3.259-4.875 1.214-5.92-1.214-5.92-3.259-4.875-4.875-3.259-5.92-1.214-5.92 1.214-4.875 3.259-3.259 4.875-1.214 5.92 1.214 5.92 3.259 4.875 4.875 3.259 5.92 1.214z"></path>
</symbol>
<symbol id="icon-stumbleupon" viewBox="0 0 34 32">
<path class="path1" d="M18.964 12.714v-2.107q0-0.75-0.536-1.286t-1.286-0.536-1.286 0.536-0.536 1.286v10.929q0 3.125-2.25 5.339t-5.411 2.214q-3.179 0-5.42-2.241t-2.241-5.42v-4.75h5.857v4.679q0 0.768 0.536 1.295t1.286 0.527 1.286-0.527 0.536-1.295v-11.071q0-3.054 2.259-5.214t5.384-2.161q3.143 0 5.393 2.179t2.25 5.25v2.429l-3.482 1.036zM28.429 16.679h5.857v4.75q0 3.179-2.241 5.42t-5.42 2.241q-3.161 0-5.411-2.223t-2.25-5.366v-4.786l2.339 1.089 3.482-1.036v4.821q0 0.75 0.536 1.277t1.286 0.527 1.286-0.527 0.536-1.277v-4.911z"></path>
</symbol>
<symbol id="icon-digg" viewBox="0 0 37 32">
<path class="path1" d="M5.857 5.036h3.643v17.554h-9.5v-12.446h5.857v-5.107zM5.857 19.661v-6.589h-2.196v6.589h2.196zM10.964 10.143v12.446h3.661v-12.446h-3.661zM10.964 5.036v3.643h3.661v-3.643h-3.661zM16.089 10.143h9.518v16.821h-9.518v-2.911h5.857v-1.464h-5.857v-12.446zM21.946 19.661v-6.589h-2.196v6.589h2.196zM27.071 10.143h9.5v16.821h-9.5v-2.911h5.839v-1.464h-5.839v-12.446zM32.911 19.661v-6.589h-2.196v6.589h2.196z"></path>
</symbol>
<symbol id="icon-spotify" viewBox="0 0 27 32">
<path class="path1" d="M20.125 21.607q0-0.571-0.536-0.911-3.446-2.054-7.982-2.054-2.375 0-5.125 0.607-0.75 0.161-0.75 0.929 0 0.357 0.241 0.616t0.634 0.259q0.089 0 0.661-0.143 2.357-0.482 4.339-0.482 4.036 0 7.089 1.839 0.339 0.196 0.589 0.196 0.339 0 0.589-0.241t0.25-0.616zM21.839 17.768q0-0.714-0.625-1.089-4.232-2.518-9.786-2.518-2.732 0-5.411 0.75-0.857 0.232-0.857 1.143 0 0.446 0.313 0.759t0.759 0.313q0.125 0 0.661-0.143 2.179-0.589 4.482-0.589 4.982 0 8.714 2.214 0.429 0.232 0.679 0.232 0.446 0 0.759-0.313t0.313-0.759zM23.768 13.339q0-0.839-0.714-1.25-2.25-1.304-5.232-1.973t-6.125-0.67q-3.643 0-6.5 0.839-0.411 0.125-0.688 0.455t-0.277 0.866q0 0.554 0.366 0.929t0.92 0.375q0.196 0 0.714-0.143 2.375-0.661 5.482-0.661 2.839 0 5.527 0.607t4.527 1.696q0.375 0.214 0.714 0.214 0.518 0 0.902-0.366t0.384-0.92zM27.429 16q0 3.732-1.839 6.884t-4.991 4.991-6.884 1.839-6.884-1.839-4.991-4.991-1.839-6.884 1.839-6.884 4.991-4.991 6.884-1.839 6.884 1.839 4.991 4.991 1.839 6.884z"></path>
</symbol>
<symbol id="icon-soundcloud" viewBox="0 0 41 32">
<path class="path1" d="M14 24.5l0.286-4.304-0.286-9.339q-0.018-0.179-0.134-0.304t-0.295-0.125q-0.161 0-0.286 0.125t-0.125 0.304l-0.25 9.339 0.25 4.304q0.018 0.179 0.134 0.295t0.277 0.116q0.393 0 0.429-0.411zM19.286 23.982l0.196-3.768-0.214-10.464q0-0.286-0.232-0.429-0.143-0.089-0.286-0.089t-0.286 0.089q-0.232 0.143-0.232 0.429l-0.018 0.107-0.179 10.339q0 0.018 0.196 4.214v0.018q0 0.179 0.107 0.304 0.161 0.196 0.411 0.196 0.196 0 0.357-0.161 0.161-0.125 0.161-0.357zM0.625 17.911l0.357 2.286-0.357 2.25q-0.036 0.161-0.161 0.161t-0.161-0.161l-0.304-2.25 0.304-2.286q0.036-0.161 0.161-0.161t0.161 0.161zM2.161 16.5l0.464 3.696-0.464 3.625q-0.036 0.161-0.179 0.161-0.161 0-0.161-0.179l-0.411-3.607 0.411-3.696q0-0.161 0.161-0.161 0.143 0 0.179 0.161zM3.804 15.821l0.446 4.375-0.446 4.232q0 0.196-0.196 0.196-0.179 0-0.214-0.196l-0.375-4.232 0.375-4.375q0.036-0.214 0.214-0.214 0.196 0 0.196 0.214zM5.482 15.696l0.411 4.5-0.411 4.357q-0.036 0.232-0.25 0.232-0.232 0-0.232-0.232l-0.375-4.357 0.375-4.5q0-0.232 0.232-0.232 0.214 0 0.25 0.232zM7.161 16.018l0.375 4.179-0.375 4.393q-0.036 0.286-0.286 0.286-0.107 0-0.188-0.080t-0.080-0.205l-0.357-4.393 0.357-4.179q0-0.107 0.080-0.188t0.188-0.080q0.25 0 0.286 0.268zM8.839 13.411l0.375 6.786-0.375 4.393q0 0.125-0.089 0.223t-0.214 0.098q-0.286 0-0.321-0.321l-0.321-4.393 0.321-6.786q0.036-0.321 0.321-0.321 0.125 0 0.214 0.098t0.089 0.223zM10.518 11.875l0.339 8.357-0.339 4.357q0 0.143-0.098 0.241t-0.241 0.098q-0.321 0-0.357-0.339l-0.286-4.357 0.286-8.357q0.036-0.339 0.357-0.339 0.143 0 0.241 0.098t0.098 0.241zM12.268 11.161l0.321 9.036-0.321 4.321q-0.036 0.375-0.393 0.375-0.339 0-0.375-0.375l-0.286-4.321 0.286-9.036q0-0.161 0.116-0.277t0.259-0.116q0.161 0 0.268 0.116t0.125 0.277zM19.268 24.411v0 0zM15.732 11.089l0.268 9.107-0.268 4.268q0 0.179-0.134 0.313t-0.313 0.134-0.304-0.125-0.143-0.321l-0.25-4.268 0.25-9.107q0-0.196 0.134-0.321t0.313-0.125 0.313 0.125 0.134 0.321zM17.5 11.429l0.25 8.786-0.25 4.214q0 0.196-0.143 0.339t-0.339 0.143-0.339-0.143-0.161-0.339l-0.214-4.214 0.214-8.786q0.018-0.214 0.161-0.357t0.339-0.143 0.33 0.143 0.152 0.357zM21.286 20.214l-0.25 4.125q0 0.232-0.161 0.393t-0.393 0.161-0.393-0.161-0.179-0.393l-0.107-2.036-0.107-2.089 0.214-11.357v-0.054q0.036-0.268 0.214-0.429 0.161-0.125 0.357-0.125 0.143 0 0.268 0.089 0.25 0.143 0.286 0.464zM41.143 19.875q0 2.089-1.482 3.563t-3.571 1.473h-14.036q-0.232-0.036-0.393-0.196t-0.161-0.393v-16.054q0-0.411 0.5-0.589 1.518-0.607 3.232-0.607 3.482 0 6.036 2.348t2.857 5.777q0.946-0.393 1.964-0.393 2.089 0 3.571 1.482t1.482 3.589z"></path>
</symbol>
<symbol id="icon-codepen" viewBox="0 0 32 32">
<path class="path1" d="M3.857 20.875l10.768 7.179v-6.411l-5.964-3.982zM2.75 18.304l3.446-2.304-3.446-2.304v4.607zM17.375 28.054l10.768-7.179-4.804-3.214-5.964 3.982v6.411zM16 19.25l4.857-3.25-4.857-3.25-4.857 3.25zM8.661 14.339l5.964-3.982v-6.411l-10.768 7.179zM25.804 16l3.446 2.304v-4.607zM23.339 14.339l4.804-3.214-10.768-7.179v6.411zM32 11.125v9.75q0 0.732-0.607 1.143l-14.625 9.75q-0.375 0.232-0.768 0.232t-0.768-0.232l-14.625-9.75q-0.607-0.411-0.607-1.143v-9.75q0-0.732 0.607-1.143l14.625-9.75q0.375-0.232 0.768-0.232t0.768 0.232l14.625 9.75q0.607 0.411 0.607 1.143z"></path>
</symbol>
<symbol id="icon-twitch" viewBox="0 0 32 32">
<path class="path1" d="M16 7.75v7.75h-2.589v-7.75h2.589zM23.107 7.75v7.75h-2.589v-7.75h2.589zM23.107 21.321l4.518-4.536v-14.196h-21.321v18.732h5.821v3.875l3.875-3.875h7.107zM30.214 0v18.089l-7.75 7.75h-5.821l-3.875 3.875h-3.875v-3.875h-7.107v-20.679l1.946-5.161h26.482z"></path>
</symbol>
<symbol id="icon-meanpath" viewBox="0 0 27 32">
<path class="path1" d="M23.411 15.036v2.036q0 0.429-0.241 0.679t-0.67 0.25h-3.607q-0.429 0-0.679-0.25t-0.25-0.679v-2.036q0-0.429 0.25-0.679t0.679-0.25h3.607q0.429 0 0.67 0.25t0.241 0.679zM14.661 19.143v-4.464q0-0.946-0.58-1.527t-1.527-0.58h-2.375q-1.214 0-1.714 0.929-0.5-0.929-1.714-0.929h-2.321q-0.946 0-1.527 0.58t-0.58 1.527v4.464q0 0.393 0.375 0.393h0.982q0.393 0 0.393-0.393v-4.107q0-0.429 0.241-0.679t0.688-0.25h1.679q0.429 0 0.679 0.25t0.25 0.679v4.107q0 0.393 0.375 0.393h0.964q0.393 0 0.393-0.393v-4.107q0-0.429 0.25-0.679t0.679-0.25h1.732q0.429 0 0.67 0.25t0.241 0.679v4.107q0 0.393 0.393 0.393h0.982q0.375 0 0.375-0.393zM25.179 17.429v-2.75q0-0.946-0.589-1.527t-1.536-0.58h-4.714q-0.946 0-1.536 0.58t-0.589 1.527v7.321q0 0.375 0.393 0.375h0.982q0.375 0 0.375-0.375v-3.214q0.554 0.75 1.679 0.75h3.411q0.946 0 1.536-0.58t0.589-1.527zM27.429 6.429v19.143q0 1.714-1.214 2.929t-2.929 1.214h-19.143q-1.714 0-2.929-1.214t-1.214-2.929v-19.143q0-1.714 1.214-2.929t2.929-1.214h19.143q1.714 0 2.929 1.214t1.214 2.929z"></path>
</symbol>
<symbol id="icon-pinterest-p" viewBox="0 0 23 32">
<path class="path1" d="M0 10.661q0-1.929 0.67-3.634t1.848-2.973 2.714-2.196 3.304-1.393 3.607-0.464q2.821 0 5.25 1.188t3.946 3.455 1.518 5.125q0 1.714-0.339 3.357t-1.071 3.161-1.786 2.67-2.589 1.839-3.375 0.688q-1.214 0-2.411-0.571t-1.714-1.571q-0.179 0.696-0.5 2.009t-0.42 1.696-0.366 1.268-0.464 1.268-0.571 1.116-0.821 1.384-1.107 1.545l-0.25 0.089-0.161-0.179q-0.268-2.804-0.268-3.357 0-1.643 0.384-3.688t1.188-5.134 0.929-3.625q-0.571-1.161-0.571-3.018 0-1.482 0.929-2.786t2.357-1.304q1.089 0 1.696 0.723t0.607 1.83q0 1.179-0.786 3.411t-0.786 3.339q0 1.125 0.804 1.866t1.946 0.741q0.982 0 1.821-0.446t1.402-1.214 1-1.696 0.679-1.973 0.357-1.982 0.116-1.777q0-3.089-1.955-4.813t-5.098-1.723q-3.571 0-5.964 2.313t-2.393 5.866q0 0.786 0.223 1.518t0.482 1.161 0.482 0.813 0.223 0.545q0 0.5-0.268 1.304t-0.661 0.804q-0.036 0-0.304-0.054-0.911-0.268-1.616-1t-1.089-1.688-0.58-1.929-0.196-1.902z"></path>
</symbol>
<symbol id="icon-periscope" viewBox="0 0 24 28">
<path class="path1" d="M12.285,1C6.696,1,2.277,5.643,2.277,11.243c0,5.851,7.77,14.578,10.007,14.578c1.959,0,9.729-8.728,9.729-14.578 C22.015,5.643,17.596,1,12.285,1z M12.317,16.551c-3.473,0-6.152-2.611-6.152-5.664c0-1.292,0.39-2.472,1.065-3.438 c0.206,1.084,1.18,1.906,2.352,1.906c1.322,0,2.393-1.043,2.393-2.333c0-0.832-0.447-1.561-1.119-1.975 c0.467-0.105,0.955-0.161,1.46-0.161c3.133,0,5.81,2.611,5.81,5.998C18.126,13.94,15.449,16.551,12.317,16.551z"></path>
</symbol>
<symbol id="icon-get-pocket" viewBox="0 0 31 32">
<path class="path1" d="M27.946 2.286q1.161 0 1.964 0.813t0.804 1.973v9.268q0 3.143-1.214 6t-3.259 4.911-4.893 3.259-5.973 1.205q-3.143 0-5.991-1.205t-4.902-3.259-3.268-4.911-1.214-6v-9.268q0-1.143 0.821-1.964t1.964-0.821h25.161zM15.375 21.286q0.839 0 1.464-0.589l7.214-6.929q0.661-0.625 0.661-1.518 0-0.875-0.616-1.491t-1.491-0.616q-0.839 0-1.464 0.589l-5.768 5.536-5.768-5.536q-0.625-0.589-1.446-0.589-0.875 0-1.491 0.616t-0.616 1.491q0 0.911 0.643 1.518l7.232 6.929q0.589 0.589 1.446 0.589z"></path>
</symbol>
<symbol id="icon-vimeo" viewBox="0 0 32 32">
<path class="path1" d="M30.518 9.25q-0.179 4.214-5.929 11.625-5.946 7.696-10.036 7.696-2.536 0-4.286-4.696-0.786-2.857-2.357-8.607-1.286-4.679-2.804-4.679-0.321 0-2.268 1.357l-1.375-1.75q0.429-0.375 1.929-1.723t2.321-2.063q2.786-2.464 4.304-2.607 1.696-0.161 2.732 0.991t1.446 3.634q0.786 5.125 1.179 6.661 0.982 4.446 2.143 4.446 0.911 0 2.75-2.875 1.804-2.875 1.946-4.393 0.232-2.482-1.946-2.482-1.018 0-2.161 0.464 2.143-7.018 8.196-6.821 4.482 0.143 4.214 5.821z"></path>
</symbol>
<symbol id="icon-reddit-alien" viewBox="0 0 32 32">
<path class="path1" d="M32 15.107q0 1.036-0.527 1.884t-1.42 1.295q0.214 0.821 0.214 1.714 0 2.768-1.902 5.125t-5.188 3.723-7.143 1.366-7.134-1.366-5.179-3.723-1.902-5.125q0-0.839 0.196-1.679-0.911-0.446-1.464-1.313t-0.554-1.902q0-1.464 1.036-2.509t2.518-1.045q1.518 0 2.589 1.125 3.893-2.714 9.196-2.893l2.071-9.304q0.054-0.232 0.268-0.375t0.464-0.089l6.589 1.446q0.321-0.661 0.964-1.063t1.411-0.402q1.107 0 1.893 0.777t0.786 1.884-0.786 1.893-1.893 0.786-1.884-0.777-0.777-1.884l-5.964-1.321-1.857 8.429q5.357 0.161 9.268 2.857 1.036-1.089 2.554-1.089 1.482 0 2.518 1.045t1.036 2.509zM7.464 18.661q0 1.107 0.777 1.893t1.884 0.786 1.893-0.786 0.786-1.893-0.786-1.884-1.893-0.777q-1.089 0-1.875 0.786t-0.786 1.875zM21.929 25q0.196-0.196 0.196-0.464t-0.196-0.464q-0.179-0.179-0.446-0.179t-0.464 0.179q-0.732 0.75-2.161 1.107t-2.857 0.357-2.857-0.357-2.161-1.107q-0.196-0.179-0.464-0.179t-0.446 0.179q-0.196 0.179-0.196 0.455t0.196 0.473q0.768 0.768 2.116 1.214t2.188 0.527 1.625 0.080 1.625-0.080 2.188-0.527 2.116-1.214zM21.875 21.339q1.107 0 1.884-0.786t0.777-1.893q0-1.089-0.786-1.875t-1.875-0.786q-1.107 0-1.893 0.777t-0.786 1.884 0.786 1.893 1.893 0.786z"></path>
</symbol>
<symbol id="icon-whatsapp" viewBox="0 0 32 32">
<path d="M15.968 2.003a14.03 13.978 0 0 0-14.03 13.978 14.03 13.978 0 0 0 2.132 7.391L1.938 29.96l6.745-2.052a14.03 13.978 0 0 0 7.285 2.052 14.03 13.978 0 0 0 14.03-13.978 14.03 13.978 0 0 0-14.03-13.978z" stroke-width=".2000562"/>
<path d="M10.454 8.236a2.57 3.401 51.533 0 0-1.475 3.184v.015c.01 2.04 4.045 10.076 10.017 12.688l.017-.013a2.57 3.401 51.533 0 0 3.454-.706 2.57 3.401 51.533 0 0 1.064-4.129 2.57 3.401 51.533 0 0-4.262.103 2.57 3.401 51.533 0 0-.505.473c-1.346-.639-2.952-1.463-4.168-2.98-.771-.962-1.257-2.732-1.549-4.206a2.57 3.401 51.533 0 0 .605-.403 2.57 3.401 51.533 0 0 1.064-4.129 2.57 3.401 51.533 0 0-4.262.103z" stroke-width=".372"/>
</symbol>
<symbol id="icon-telegram" viewBox="0 0 32 32">
<path d="M30.8,2.2L0.6,13.9c-0.8,0.3-0.7,1.3,0,1.6l7.4,2.8l2.9,9.2c0.2,0.6,0.9,0.8,1.4,0.4l4.1-3.4 c0.4-0.4,1-0.4,1.5,0l7.4,5.4c0.5,0.4,1.2,0.1,1.4-0.5L32,3.2C32.1,2.5,31.4,1.9,30.8,2.2z M25,8.3l-11.9,11 c-0.4,0.4-0.7,0.9-0.8,1.5l-0.4,3c-0.1,0.4-0.6,0.4-0.7,0.1l-1.6-5.5c-0.2-0.6,0.1-1.3,0.6-1.6l14.4-8.9C25,7.7,25.3,8.1,25,8.3z"/>
</symbol>
<symbol id="icon-hashtag" viewBox="0 0 32 32">
<path class="path1" d="M17.696 18.286l1.143-4.571h-4.536l-1.143 4.571h4.536zM31.411 9.286l-1 4q-0.125 0.429-0.554 0.429h-5.839l-1.143 4.571h5.554q0.268 0 0.446 0.214 0.179 0.25 0.107 0.5l-1 4q-0.089 0.429-0.554 0.429h-5.839l-1.446 5.857q-0.125 0.429-0.554 0.429h-4q-0.286 0-0.464-0.214-0.161-0.214-0.107-0.5l1.393-5.571h-4.536l-1.446 5.857q-0.125 0.429-0.554 0.429h-4.018q-0.268 0-0.446-0.214-0.161-0.214-0.107-0.5l1.393-5.571h-5.554q-0.268 0-0.446-0.214-0.161-0.214-0.107-0.5l1-4q0.125-0.429 0.554-0.429h5.839l1.143-4.571h-5.554q-0.268 0-0.446-0.214-0.179-0.25-0.107-0.5l1-4q0.089-0.429 0.554-0.429h5.839l1.446-5.857q0.125-0.429 0.571-0.429h4q0.268 0 0.446 0.214 0.161 0.214 0.107 0.5l-1.393 5.571h4.536l1.446-5.857q0.125-0.429 0.571-0.429h4q0.268 0 0.446 0.214 0.161 0.214 0.107 0.5l-1.393 5.571h5.554q0.268 0 0.446 0.214 0.161 0.214 0.107 0.5z"></path>
</symbol>
<symbol id="icon-chain" viewBox="0 0 30 32">
<path class="path1" d="M26 21.714q0-0.714-0.5-1.214l-3.714-3.714q-0.5-0.5-1.214-0.5-0.75 0-1.286 0.571 0.054 0.054 0.339 0.33t0.384 0.384 0.268 0.339 0.232 0.455 0.063 0.491q0 0.714-0.5 1.214t-1.214 0.5q-0.268 0-0.491-0.063t-0.455-0.232-0.339-0.268-0.384-0.384-0.33-0.339q-0.589 0.554-0.589 1.304 0 0.714 0.5 1.214l3.679 3.696q0.482 0.482 1.214 0.482 0.714 0 1.214-0.464l2.625-2.607q0.5-0.5 0.5-1.196zM13.446 9.125q0-0.714-0.5-1.214l-3.679-3.696q-0.5-0.5-1.214-0.5-0.696 0-1.214 0.482l-2.625 2.607q-0.5 0.5-0.5 1.196 0 0.714 0.5 1.214l3.714 3.714q0.482 0.482 1.214 0.482 0.75 0 1.286-0.554-0.054-0.054-0.339-0.33t-0.384-0.384-0.268-0.339-0.232-0.455-0.063-0.491q0-0.714 0.5-1.214t1.214-0.5q0.268 0 0.491 0.063t0.455 0.232 0.339 0.268 0.384 0.384 0.33 0.339q0.589-0.554 0.589-1.304zM29.429 21.714q0 2.143-1.518 3.625l-2.625 2.607q-1.482 1.482-3.625 1.482-2.161 0-3.643-1.518l-3.679-3.696q-1.482-1.482-1.482-3.625 0-2.196 1.571-3.732l-1.571-1.571q-1.536 1.571-3.714 1.571-2.143 0-3.643-1.5l-3.714-3.714q-1.5-1.5-1.5-3.643t1.518-3.625l2.625-2.607q1.482-1.482 3.625-1.482 2.161 0 3.643 1.518l3.679 3.696q1.482 1.482 1.482 3.625 0 2.196-1.571 3.732l1.571 1.571q1.536-1.571 3.714-1.571 2.143 0 3.643 1.5l3.714 3.714q1.5 1.5 1.5 3.643z"></path>
</symbol>
<symbol id="icon-thumb-tack" viewBox="0 0 21 32">
<path class="path1" d="M8.571 15.429v-8q0-0.25-0.161-0.411t-0.411-0.161-0.411 0.161-0.161 0.411v8q0 0.25 0.161 0.411t0.411 0.161 0.411-0.161 0.161-0.411zM20.571 21.714q0 0.464-0.339 0.804t-0.804 0.339h-7.661l-0.911 8.625q-0.036 0.214-0.188 0.366t-0.366 0.152h-0.018q-0.482 0-0.571-0.482l-1.357-8.661h-7.214q-0.464 0-0.804-0.339t-0.339-0.804q0-2.196 1.402-3.955t3.17-1.759v-9.143q-0.929 0-1.607-0.679t-0.679-1.607 0.679-1.607 1.607-0.679h11.429q0.929 0 1.607 0.679t0.679 1.607-0.679 1.607-1.607 0.679v9.143q1.768 0 3.17 1.759t1.402 3.955z"></path>
</symbol>
<symbol id="icon-arrow-left" viewBox="0 0 43 32">
<path class="path1" d="M42.311 14.044c-0.178-0.178-0.533-0.356-0.711-0.356h-33.778l10.311-10.489c0.178-0.178 0.356-0.533 0.356-0.711 0-0.356-0.178-0.533-0.356-0.711l-1.6-1.422c-0.356-0.178-0.533-0.356-0.889-0.356s-0.533 0.178-0.711 0.356l-14.578 14.933c-0.178 0.178-0.356 0.533-0.356 0.711s0.178 0.533 0.356 0.711l14.756 14.933c0 0.178 0.356 0.356 0.533 0.356s0.533-0.178 0.711-0.356l1.6-1.6c0.178-0.178 0.356-0.533 0.356-0.711s-0.178-0.533-0.356-0.711l-10.311-10.489h33.778c0.178 0 0.533-0.178 0.711-0.356 0.356-0.178 0.533-0.356 0.533-0.711v-2.133c0-0.356-0.178-0.711-0.356-0.889z"></path>
</symbol>
<symbol id="icon-arrow-right" viewBox="0 0 43 32">
<path class="path1" d="M0.356 17.956c0.178 0.178 0.533 0.356 0.711 0.356h33.778l-10.311 10.489c-0.178 0.178-0.356 0.533-0.356 0.711 0 0.356 0.178 0.533 0.356 0.711l1.6 1.6c0.178 0.178 0.533 0.356 0.711 0.356s0.533-0.178 0.711-0.356l14.756-14.933c0.178-0.356 0.356-0.711 0.356-0.889s-0.178-0.533-0.356-0.711l-14.756-14.933c0-0.178-0.356-0.356-0.533-0.356s-0.533 0.178-0.711 0.356l-1.6 1.6c-0.178 0.178-0.356 0.533-0.356 0.711s0.178 0.533 0.356 0.711l10.311 10.489h-33.778c-0.178 0-0.533 0.178-0.711 0.356-0.356 0.178-0.533 0.356-0.533 0.711v2.311c0 0.178 0.178 0.533 0.356 0.711z"></path>
</symbol>
<symbol id="icon-play" viewBox="0 0 22 28">
<path d="M21.625 14.484l-20.75 11.531c-0.484 0.266-0.875 0.031-0.875-0.516v-23c0-0.547 0.391-0.781 0.875-0.516l20.75 11.531c0.484 0.266 0.484 0.703 0 0.969z"></path>
</symbol>
<symbol id="icon-pause" viewBox="0 0 24 28">
<path d="M24 3v22c0 0.547-0.453 1-1 1h-8c-0.547 0-1-0.453-1-1v-22c0-0.547 0.453-1 1-1h8c0.547 0 1 0.453 1 1zM10 3v22c0 0.547-0.453 1-1 1h-8c-0.547 0-1-0.453-1-1v-22c0-0.547 0.453-1 1-1h8c0.547 0 1 0.453 1 1z"></path>
</symbol>
</defs>
</svg>

</body>
</html>
